{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Simple Perplexity AI clone\n",
    "\n",
    "\n",
    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mklarqvist/ai-playground/blob/main/perplexity-clone/perplexity_clone.ipynb)\n",
    "\n",
    "Perplexity AI ([perplexity.ai](perplexity.ai)) is an chat tool that uses foundational language models, such as GPT-4 from OpenAI, along with current information from the internet. It not only provides answers, but also references to the sources that contributed to those answers. This simple, yet powerful approach addresses the limitation of potentially outdated training data used to train the models. By returning the sources used to provide an answer, you can verify its accuracy. This combats the issue of language models generating incorrect answers.\n",
    "\n",
    "This may sound like a major project and a serious undertaking, but modern tools have made it surprisingly easy.\n",
    "\n",
    "The workflow can be described as followed:\n",
    "\n",
    "1. The user poses a question.\n",
    "2. A Google search is performed using the question.\n",
    "3. The top-k search results, or the most relevant webpages, are downloaded.\n",
    "4. Raw HTML data is transformed into a usable format by LangChain.\n",
    "5. All documents are split into 1,000 character chunks.\n",
    "6. Compute embeddings for each document chunk and store them in a vector store (chromadb).\n",
    "7. Build a prompt using the user's question from step 1 and all the scraped web data using LangChain.\n",
    "8. Query an OpenAI model to generate an answer.\n",
    "9. Identify the documents that contributed to the answer and return them as references."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Querying Google and scraping websites"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First we need to install the required dependencies."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip3 install -U readabilipy langchain openai bs4 requests chromadb tiktoken"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests # Required to make HTTP requests\n",
    "from bs4 import BeautifulSoup # Required to parse HTML\n",
    "import numpy as np # Required to dedupe sites\n",
    "from urllib.parse import unquote # Required to unquote URLs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "query = 'history of the human genome project' # The query to search Google for and ask the AI about"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "response = requests.get(f\"https://www.google.com/search?q={query}\") # Make the request\n",
    "soup = BeautifulSoup(response.text, \"html.parser\") # Parse the HTML\n",
    "links = soup.find_all(\"a\") # Find all the links in the HTML"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://ellipse.prbb.org/the-human-genome-20-years-of-history/',\n",
       " 'https://en.wikipedia.org/wiki/Human_Genome_Project',\n",
       " 'https://europepmc.org/article/med/8489331',\n",
       " 'https://genomemedicine.biomedcentral.com/articles/10.1186/gm483',\n",
       " 'https://plato.stanford.edu/entries/human-genome/',\n",
       " 'https://pubmed.ncbi.nlm.nih.gov/1991595/',\n",
       " 'https://web.ornl.gov/sci/techresources/Human_Genome/project/hgp.shtml',\n",
       " 'https://www.britannica.com/event/Human-Genome-Project',\n",
       " 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project',\n",
       " 'https://www.genome.gov/about-nhgri/Brief-History-Timeline',\n",
       " 'https://www.genome.gov/human-genome-project',\n",
       " 'https://www.genome.gov/human-genome-project/20th-anniversary-of-landmark-human-genome-project-publications',\n",
       " 'https://www.genome.gov/leadership-initiatives/History-of-Genomics-Program',\n",
       " 'https://www.jstor.org/stable/3083473',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1592577/']"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# loop over `links` and keep only the one that have the href starting with \"/url?q=\"\n",
    "urls = []\n",
    "for l in [link for link in links if link[\"href\"].startswith(\"/url?q=\")]:\n",
    "    # get the url\n",
    "    url = l[\"href\"]\n",
    "    # remove the \"/url?q=\" part\n",
    "    url = url.replace(\"/url?q=\", \"\")\n",
    "    # remove the part after the \"&sa=...\"\n",
    "    url = unquote(url.split(\"&sa=\")[0])\n",
    "    # special case for google scholar\n",
    "    if url.startswith(\"https://scholar.google.com/scholar_url?url=http\"):\n",
    "        url = url.replace(\"https://scholar.google.com/scholar_url?url=\", \"\").split(\"&\")[0]\n",
    "    elif 'google.com/' in url: # skip google links\n",
    "        continue\n",
    "    if url.endswith('.pdf'): # skip pdf links\n",
    "        continue\n",
    "    if '#' in url: # remove anchors (e.g. wikipedia.com/bob#history and wikipedia.com/bob#genetics are the same page)\n",
    "        url = url.split('#')[0]\n",
    "    # print the url\n",
    "    urls.append(url)\n",
    "\n",
    "# Use numpy to dedupe the list of urls after removing anchors\n",
    "urls = list(np.unique(urls))\n",
    "urls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from readabilipy import simple_json_from_html_string # Required to parse HTML to pure text\n",
    "from langchain.schema import Document # Required to create a Document object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def scrape_and_parse(url: str) -> Document:\n",
    "    \"\"\"Scrape a webpage and parse it into a Document object\"\"\"\n",
    "    req = requests.get(url)\n",
    "    article = simple_json_from_html_string(req.text, use_readability=True)\n",
    "    # The following line seems to work with the package versions on my local machine, but not on Google Colab\n",
    "    # return Document(page_content=article['plain_text'][0]['text'], metadata={'source': url, 'page_title': article['title']})\n",
    "    return Document(page_content='\\n\\n'.join([a['text'] for a in article['plain_text']]), metadata={'source': url, 'page_title': article['title']})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# It's possible to optitimize this by using asyncio, but it's not necessary for this example\n",
    "documents = [scrape_and_parse(f) for f in urls] # Scrape and parse all the urls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(page_content='For humans, the history of DNA began just over 150 years ago when Friedrich Miescher accidentally discovered nuclein, a phosphate-rich molecule that did not behave like a protein and that, over the years, would end up called DNA. But in this short story there are two other great milestones. The first, when almost seventy years ago Rosalind Franklin, and later James Watson and Francis Crick, discovered through X-ray images, that DNA was structured as a double helix. The second milestone came just 20 years ago, when two teams of scientists from different countries published the first sequence of the human genome. The project to sequence the human genome was extremely long and expensive. But the nearly $ 3 billion invested were justified by the possibility to better understand the diseases of genetic origin, which at that time could be counted with the fingers of one hand. In the words of Roderic Guigó, head of the computational biology group of RNA processing, coordinator of the bioinformatics program at the Centre for Genomic Regulation (CRG) and professor at Pompeu Fabra University, “knowledge of the human genome was a crucial contribution to our understanding of human biology“. Computational biology and technology This titanic project would not have been possible without computational biology, the science that applies theoretical models to analyzing biological data. And the advancement of computational biology has been tied to that of technology. “In the first programming courses I took in the calculation lab of the University of Barcelona we used perforated cards. At that time, computers did not have screens, and when you wanted to communicate with the computer, you put the card on it and the result was printed on a sheet”, explains Guigó when he remembers how was his degree in the mid-eighties. Image of the perforated cards and the sheet with the printed result that Roderic Guigó used while studying his degree. Credit: PRBB Roderic recalls that years later at the University, they had the first computer with three screens connected. But computer science was not yet available to everyone. “During the thesis I already had an e-mail address. The problem was that since no one else had it... it was not useful!”. This is why, for Roderic, the actual technological revolution was to have a personal computer and be able to work from home. A revolution that David Comas, leader of the human genome diversity group at the Institute for Evolutionary Biology (IBE: CSIC-UPF) and current director of the Department of Experimental and Health Sciences, Pompeu Fabra University (DCEXS-UPF) lived in a similar way: “With the knowledge and technique we have today, I would have done my PhD thesis in a fortnight instead of three or four years! It would take a week to produce and genotype the data, and the analysis would be done in four days”. Identifying and visualizing genes Roderic has always been motivated by technical challenges. “I wanted to develop programs to find out where the genes were in the DNA sequence. I wanted to understand what the signals and letter combinations were that the cellular machinery recognized and used to make proteins. So he went to Boston to do a postdoctoral fellowship at the Smith Temple Laboratory, one of the first bioinformatics to develop algorithms to compare and align DNA sequences. “I wanted to develop programs to find out where the genes were in the DNA sequence” Roderic Guigó – CRG In 1994, the Catalan bioinformatician returned to Barcelona and established his own group at the Hospital del Mar Medical Research Institute (IMIM), where he continued to identify these tiny regions of the genome (2%) that correspond to the genes. “At the end, we were well-known in the field because of our experience identifying genes and also for our software that allowed us to visualize the different annotated regions of the genome“, says Guigó. The human genome project The public project to sequence the human genome began in 1990, but the early years were not very successful. In 1998, the american biochemist Craig Venter questioned the operation of the public project. It was expensive, costly, and they used an inefficient technology. And his company, Celera Genomics, joined the race by applying ‘Shotgun’ technology to sequence multiple DNA fragments simultaneously. “Celera was a very open-minded company in the sense that they immediately had the participation of experts in the field, and they contacted us to use our software to visualize the genome of the fruit fly“, says Guigó. This first work meant that a few months before publishing the sequence of the human genome, Celera contacted them again to visualize the sequence of the human genome. “When we were developing the software, here in Barcelona, I thought I was working on something that was important, but just as many scientists consider that their research is important. However, when I went to Celera Genomics with my PhD student Josep Abril, to work on the first draft of the human genome sequence, we were aware we were participating in a project that would go down in history”. Press clipping published in El País in February 2001. Courtesy of Roderic Guigó Finally, on February 12, 2001, Nature and Science announced the publication of the human genome sequence, with the results of public and private initiatives. The Genome Encyclopedia Beyond the historical milestone, having the human genome sequence did not cause any revolution. “After 2001 genomics advanced in two fields. On one hand, scientists wanted to unravel the differences in genome sequence between different individuals and populations, and on the other hand, it was necessary to know which regions of the genome were functional“. To meet this latest challenge, the project to generate the encyclopedia of functional DNA elements (ENCODE) was launched in 2003; a project that is still underway and in which Roderic’s group participated from the very beginning. “We thought that much of the genome had no function. That it was junk DNA resulting from viral transposons”, says Guigó. Now we know that genes represent just 2% of the genome sequence, but other functional regions involved in cellular regulation or gene expression have been identified. The biologist David Comas confirms that “the concept junk DNA is highly questioned because we still have a lot to learn about the functions of many DNA regions and their variants.” Studying diversity and evidence of bias Once having the sequence, scientists realized that 99.9% of the genome is common to all humans. But do we know the remaining 0.1%? In 2008, the 1000 Genomes Project began to sequence the genomes of people from different regions of the planet. However, in the opinion of Comas, expert on the diversity of the human genome, “we know the most common variants of our genome, but we have forgotten some small populations that are less represented.” “We know the most common variants of our genome, but we have forgotten some small populations that are less represented”David Comas – UPF-DCEXS and IBE To better discover genome diversity, over the years, “the genome of specific populations that had some associated diseases has been sequenced“. Besides, national initiatives have also emerged, to sequence the genome of the entire population of Iceland or 100,000 genomes of people in the United Kingdom. Projects that have a more local scope but “also consider the phenotypic data of the individuals studied, enabling us to correlate what we see with the genetic variants”, explains Comas. Despite these efforts, we have a biased knowledge of genome diversity. And that has consequences. In Comas’s words, “infrequent variants, which are specific of small, underrepresented populations, are often involved in biomedical aspects“. And not knowing this diversity or having a low representation of it means that when the data is analyzed, “the less frequent variants are often removed from the analyses because the algorithms take them as sequence errors“. And not only that; there are also technical biases as “it is much easier to study variations that affect a single nucleotide than structural variants that generate large changes.” Leading into the future of human genomics The future of genomics lies in understanding the functional implications of changes in sequence. “Historically, we have seen the genome as something very cold and static, as a linear sequence. And we still don’t know how different regions of the same sequence interact, “explains Comas. This future has already begun with genomic data repositories such as the EGA (European Genome-Phenome Archive) – with one of the two nodes in the Barcelona Biomedical Research Park (PRBB), coordinated by the team led by Arcadi Navarro – and with projects such as genotype-tissue expression (GTEx), in which Guigó’s lab participates. The latter project attempts to understand the relationships between genome sequence variation between individuals and gene expression in different tissues. Although we still need to be able to explain “how the genomic variation of people has an impact on their function and results in differences such as, for example, the predisposition to have certain diseases,” concludes Guigó.', metadata={'source': 'https://ellipse.prbb.org/the-human-genome-20-years-of-history/', 'page_title': 'The human genome: 20 years of history - El·lipse'}),\n",
       " Document(page_content='Logo of the Human Genome Project The Human Genome Project (HGP) was an international scientific research project with the goal of determining the base pairs that make up human DNA, and of identifying, mapping and sequencing all of the genes of the human genome from both a physical and a functional standpoint. It started in 1990 and was completed in 2003.[1] It remains the world\\'s largest collaborative biological project.[2] Planning for the project started after it was adopted in 1984 by the US government, and it officially launched in 1990. It was declared complete on April 14, 2003, and included about 92% of the genome.[3] Level \"complete genome\" was achieved in May 2021, with a remaining only 0.3% bases covered by potential issues.[4][5] The final gapless assembly was finished in January 2022.[6]Funding came from the United States government through the National Institutes of Health (NIH) as well as numerous other groups from around the world. A parallel project was conducted outside the government by the Celera Corporation, or Celera Genomics, which was formally launched in 1998. Most of the government-sponsored sequencing was performed in twenty universities and research centres in the United States, the United Kingdom, Japan, France, Germany, and China,[7] working in the International Human Genome Sequencing Consortium (IHGSC).The Human Genome Project originally aimed to map the complete set of nucleotides contained in a human haploid reference genome, of which there are more than three billion. The \"genome\" of any given individual is unique; mapping the \"human genome\" involved sequencing samples collected from a small number of individuals and then assembling the sequenced fragments to get a complete sequence for each of 24 human chromosomes (22 autosomes and 2 sex chromosomes). Therefore, the finished human genome is a mosaic, not representing any one individual. Much of the project\\'s utility comes from the fact that the vast majority of the human genome is the same in all humans. History[edit] .[8][9]In May 1985, Robert Sinsheimer organized a workshop at the University of California, Santa Cruz, to discuss the feasibility of building a systematic reference genome using gene sequencing technologies.[10] In March 1986, the Santa Fe Workshop was organized by Charles DeLisi and David Smith of the Department of Energy\\'s Office of Health and Environmental Research (OHER).[11] At the same time Renato Dulbecco, President of the Salk Institute for Biological Studies, first proposed the concept of whole genome sequencing in an essay in Science.[12] The published work, titled \"A Turning Point in Cancer Research: Sequencing the Human Genome\", was shortened from the original proposal of using the sequence to understand the genetic basis of breast cancer.[13] James Watson, one of the discoverers of the double helix shape of DNA in the 1970s, followed two months later with a workshop held at the Cold Spring Harbor Laboratory. Thus the idea for obtaining a reference sequence had three independent origins: Sinsheimer, Dulbecco and DeLisi. Ultimately it was the actions by DeLisi that launched the project.[14][15][16][17]The fact that the Santa Fe Workshop was motivated and supported by a federal agency opened a path, albeit a difficult and tortuous one,[18] for converting the idea into public policy in the United States. In a memo to the Assistant Secretary for Energy Research Alvin Trivelpiece, then-Director of the OHER Charles DeLisi outlined a broad plan for the project.[19] This started a long and complex chain of events which led to approved reprogramming of funds that enabled the OHER to launch the project in 1986, and to recommend the first line item for the HGP, which was in President Reagan\\'s 1988 budget submission,[18] and ultimately approved by Congress. Of particular importance in congressional approval was the advocacy of New Mexico Senator Pete Domenici, whom DeLisi had befriended.[20] Domenici chaired the Senate Committee on Energy and Natural Resources, as well as the Budget Committee, both of which were key in the DOE budget process. Congress added a comparable amount to the NIH budget, thereby beginning official funding by both agencies.Trivelpiece sought and obtained the approval of DeLisi\\'s proposal by Deputy Secretary William Flynn Martin. This chart[21] was used by Trivelpiece in the spring of 1986 to brief Martin and Under Secretary Joseph Salgado regarding his intention to reprogram $4 million to initiate the project with the approval of John S. Herrington. This reprogramming was followed by a line item budget of $16 million in the Reagan administration\\'s 1987 budget submission to Congress.[11] It subsequently passed both Houses. The project was planned to be completed within 15 years.[22]In 1990, the two major funding agencies, DOE and the National Institutes of Health, developed a memorandum of understanding in order to coordinate plans and set the clock for the initiation of the Project to 1990.[23] At that time, David Galas was Director of the renamed \"Office of Biological and Environmental Research\" in the U.S. Department of Energy\\'s Office of Science and James Watson headed the NIH Genome Program. In 1993, Aristides Patrinos succeeded Galas and Francis Collins succeeded Watson, assuming the role of overall Project Head as Director of the NIH National Center for Human Genome Research (which would later become the National Human Genome Research Institute). A working draft of the genome was announced in 2000 and the papers describing it were published in February 2001. A more complete draft was published in 2003, and genome \"finishing\" work continued for more than a decade after that.The $3 billion project was formally founded in 1990 by the US Department of Energy and the National Institutes of Health, and was expected to take 15 years.[24] In addition to the United States, the international consortium comprised geneticists in the United Kingdom, France, Australia, China, and myriad other spontaneous relationships.[25] The project ended up costing less than expected, at about $2.7 billion (equivalent to about $5 billion in 2021).[7][26][27]Two technologies enabled the project: gene mapping and DNA sequencing. The gene mapping technique of restriction fragment length polymorphism (RFLP) arose from the search for the location of the breast cancer gene by Dr. Mark Skolnick of the University of Utah,[28] which began in 1974.[29] Seeing a linkage marker for the gene, collaboration with David Botstein, Ray White and Ron Davies conceived of a way to construct a genetic linkage map of the human genome. This enabled scientists to launch the larger human genome effort.[30]Because of widespread international cooperation and advances in the field of genomics (especially in sequence analysis), as well as parallel advances in computing technology, a \\'rough draft\\' of the genome was finished in 2000 (announced jointly by U.S. President Bill Clinton and British Prime Minister Tony Blair on June 26, 2000).[31][32] This first available rough draft assembly of the genome was completed by the Genome Bioinformatics Group at the University of California, Santa Cruz, primarily led by then-graduate student Jim Kent and his advisor David Haussler.[33] Ongoing sequencing led to the announcement of the essentially complete genome on April 14, 2003, two years earlier than planned.[34][35] In May 2006, another milestone was passed on the way to completion of the project when the sequence of the very last chromosome was published in Nature.[36]The various institutions, companies, and laboratories which participated in the Human Genome Project are listed below, according to the NIH:[7] Additionally, beginning in 2000 and continuing for three years in Russia, the Russian Foundation for Basic Research (RFFI; Russian: Российский фонд фундаментальных исследований (РФФИ)) provided a grant of about 500 thousand rubles to fund genome mapping of Russians, three groups:[citation needed] * Vologda-Vyatka (Russian: Вологда-Вятка), * Ilmen-Belozersk (Russian: Ильмень-Белозерск), * Valdai (Russian: Валдай), by the Laboratory of Human Population Genetics of the Medical Genetics Center [ru] of the Russian Academy of Medical Sciences (Russian: лаборатории популяционной генетики человека Медико-генетического центра Российской академии медицинских наук).Although the top Russian geneticist in 2004 was Sergei Inge-Vechtomov [ru] (Russian: Сергей Инге-Вечтомов), the research was headed by Doctor of Biological Sciences Elena Balanovskaya [ru] (Russian: Елена Балановская) at the Laboratory of Human Population Genetics in Moscow. Since 2004, Evgeny Ginter [ru] is the scientific supervisor of the Medical Genetics Center in Moscow.[37] State of completion[edit] Notably, the project was not able to sequence all of the DNA found in human cells; rather, the aim was to sequence only euchromatic regions of the nuclear genome, which make up 92.1% of the human genome. The remaining 7.9% exists in scattered heterochromatic regions such as those found in centromeres and telomeres. These regions by their nature are generally more difficult to sequence and so were not included as part of the project\\'s original plans.[38]The Human Genome Project (HGP) was declared complete in April 2003. An initial rough draft of the human genome was available in June 2000 and by February 2001 a working draft had been completed and published followed by the final sequencing mapping of the human genome on April 14, 2003. Although this was reported to cover 99% of the euchromatic human genome with 99.99% accuracy, a major quality assessment of the human genome sequence was published on May 27, 2004, indicating over 92% of sampling exceeded 99.99% accuracy which was within the intended goal.[39]In March 2009, the Genome Reference Consortium (GRC) released a more accurate version of the human genome, but that still left more than 300 gaps,[40] while 160 such gaps remained in 2015.[41]Though in May 2020, the GRC reported 79 \"unresolved\" gaps,[42] accounting for as much as 5% of the human genome,[43] months later, the application of new long-range sequencing techniques and a hydatidiform mole-derived cell line in which both copies of each chromosome are identical led to the first telomere-to-telomere, truly complete sequence of a human chromosome, the X chromosome.[44] Similarly, an end-to-end complete sequence of human autosomal chromosome 8 followed several months later.[45]In 2021, it was reported that the Telomere-to-Telomere (T2T) consortium had filled in all of the gaps except five in repetitive regions of ribosomal DNA.[46] Months later, those gaps had also been closed. The full sequence did not contain the Y chromosome, which causes the embryo to become male, being absent in the cell line that served as the source for the DNA analyzed. About 0.3% of the full sequence proved difficult to check for quality, and thus might have contained errors,[47] which were being targeted for confirmation.[48] In April 2022, the complete non-Y chromosome sequence was formally published, providing a view of much of the 8% of the genome left out by the HGP.[49] In December, 2022, a preprint article claimed that the sequencing of the remaining missing regions of Y chromosome had been performed, thus completing the sequencing of all 24 human chromosomes.[50] Applications and proposed benefits[edit] The sequencing of the human genome holds benefits for many fields, from molecular medicine to human evolution. The Human Genome Project, through its sequencing of the DNA, can help researchers understand diseases including: genotyping of specific viruses to direct appropriate treatment; identification of mutations linked to different forms of cancer; the design of medication and more accurate prediction of their effects; advancement in forensic applied sciences; biofuels and other energy applications; agriculture, animal husbandry, bioprocessing; risk assessment; bioarcheology, anthropology and evolution. Another proposed benefit is the commercial development of genomics research related to DNA-based products, a multibillion-dollar industry.The sequence of the DNA is stored in databases available to anyone on the Internet. The U.S. National Center for Biotechnology Information (and sister organizations in Europe and Japan) house the gene sequence in a database known as GenBank, along with sequences of known and hypothetical genes and proteins. Other organizations, such as the UCSC Genome Browser at the University of California, Santa Cruz,[51] and Ensembl[52] present additional data and annotation and powerful tools for visualizing and searching it. Computer programs have been developed to analyze the data because the data itself is difficult to interpret without such programs. Generally speaking, advances in genome sequencing technology have followed Moore\\'s Law, a concept from computer science which states that integrated circuits can increase in complexity at an exponential rate.[53] This means that the speeds at which whole genomes can be sequenced can increase at a similar rate, as was seen during the development of the Human Genome Project. Techniques and analysis[edit] The process of identifying the boundaries between genes and other features in a raw DNA sequence is called genome annotation and is in the domain of bioinformatics. While expert biologists make the best annotators, their work proceeds slowly, and computer programs are increasingly used to meet the high-throughput demands of genome sequencing projects. Beginning in 2008, a new technology known as RNA-seq was introduced that allowed scientists to directly sequence the messenger RNA in cells. This replaced previous methods of annotation, which relied on the inherent properties of the DNA sequence, with direct measurement, which was much more accurate. Today, annotation of the human genome and other genomes relies primarily on deep sequencing of the transcripts in every human tissue using RNA-seq. These experiments have revealed that over 90% of genes contain at least one and usually several alternative splice variants, in which the exons are combined in different ways to produce 2 or more gene products from the same locus.[54]The genome published by the HGP does not represent the sequence of every individual\\'s genome. It is the combined mosaic of a small number of anonymous donors, of African, European and east Asian ancestry. The HGP genome is a scaffold for future work in identifying differences among individuals.[citation needed] Subsequent projects sequenced the genomes of multiple distinct ethnic groups, though as of 2019 there is still only one \"reference genome\".[55] Findings[edit] Key findings of the draft (2001) and complete (2004) genome sequences include: * There are approximately 22,300[56] protein-coding genes in human beings, the same range as in other mammals., * The human genome has significantly more segmental duplications (nearly identical, repeated sections of DNA) than had been previously suspected.[57][58][59], * At the time when the draft sequence was published, fewer than 7% of protein families appeared to be vertebrate specific.[60], Accomplishments[edit] The first printout of the human genome to be presented as a series of books, displayed at the Wellcome Collection, London The human genome has approximately 3.1 billion base pairs.[61] The Human Genome Project was started in 1990 with the goal of sequencing and identifying all base pairs in the human genetic instruction set, finding the genetic roots of disease and then developing treatments. It is considered a megaproject.The genome was broken into smaller pieces; approximately 150,000 base pairs in length.[62] These pieces were then ligated into a type of vector known as \"bacterial artificial chromosomes\", or BACs, which are derived from bacterial chromosomes which have been genetically engineered. The vectors containing the genes can be inserted into bacteria where they are copied by the bacterial DNA replication machinery. Each of these pieces was then sequenced separately as a small \"shotgun\" project and then assembled. The larger, 150,000 base pairs go together to create chromosomes. This is known as the \"hierarchical shotgun\" approach, because the genome is first broken into relatively large chunks, which are then mapped to chromosomes before being selected for sequencing.[63][64]Funding came from the US government through the National Institutes of Health in the United States, and a UK charity organization, the Wellcome Trust, as well as numerous other groups from around the world. The funding supported a number of large sequencing centers including those at Whitehead Institute, the Wellcome Sanger Institute (then called The Sanger Centre) based at the Wellcome Genome Campus, Washington University in St. Louis, and Baylor College of Medicine.[24][65]The United Nations Educational, Scientific and Cultural Organization (UNESCO) served as an important channel for the involvement of developing countries in the Human Genome Project.[66] Public vis-à-vis private approaches[edit] In 1998, a similar, privately funded quest was launched by the American researcher Craig Venter, and his firm Celera Genomics. Venter was a scientist at the NIH during the early 1990s when the project was initiated. The $300M Celera effort was intended to proceed at a faster pace and at a fraction of the cost of the roughly $3 billion publicly funded project. The Celera approach was able to proceed at a much more rapid rate and at a lower cost, than the public project. While it made use of publicly available maps at GeneBank, those were of low quality and only slowed down the project.[57]Celera used a technique called whole genome shotgun sequencing, employing pairwise end sequencing,[67] which had been used to sequence bacterial genomes of up to six million base pairs in length, but not for anything nearly as large as the three billion base pair human genome.Celera initially announced that it would seek patent protection on \"only 200–300\" genes, but later amended this to seeking \"intellectual property protection\" on \"fully-characterized important structures\" amounting to 100–300 targets. The firm eventually filed preliminary (\"place-holder\") patent applications on 6,500 whole or partial genes. Celera also promised to publish their findings in accordance with the terms of the 1996 \"Bermuda Statement\", by releasing new data annually (the HGP released its new data daily), although, unlike the publicly funded project, they would not permit free redistribution or scientific use of the data. The publicly funded competitors were compelled to release the first draft of the human genome before Celera for this reason. On July 7, 2000, the UCSC Genome Bioinformatics Group released a first working draft on the web. The scientific community downloaded about 500 GB of information from the UCSC genome server in the first 24 hours of free and unrestricted access.[68]In March 2000, President Clinton, along with Prime Minister Tony Blair in a dual statement, urged that all researchers who wished to research the sequence should have \"unencumbered access\" to the genome sequence.[69] The statement sent Celera\\'s stock plummeting and dragged down the biotechnology-heavy Nasdaq. The biotechnology sector lost about $50 billion in market capitalization in two days.Although the working draft was announced in June 2000, it was not until February 2001 that Celera and the HGP scientists published details of their drafts. Special issues of Nature (which published the publicly funded project\\'s scientific paper)[57] described the methods used to produce the draft sequence and offered analysis of the sequence. These drafts covered about 83% of the genome (90% of the euchromatic regions with 150,000 gaps and the order and orientation of many segments not yet established). In February 2001, at the time of the joint publications, press releases announced that the project had been completed by both groups. Improved drafts were announced in 2003 and 2005, filling in to approximately 92% of the sequence currently. Genome donors[edit] In the International Human Genome Sequencing Consortium (IHGSC) public-sector HGP, researchers collected blood (female) or sperm (male) samples from a large number of donors. Only a few of many collected samples were processed as DNA resources. Thus the donor identities were protected so neither donors nor scientists could know whose DNA was sequenced. DNA clones taken from many different libraries were used in the overall project, with most of those libraries being created by Dr. Pieter J. de Jong. Much of the sequence (>70%) of the reference genome produced by the public HGP came from a single anonymous male donor from Buffalo, New York, (code name RP11; the \"RP\" refers to Roswell Park Comprehensive Cancer Center).[70][71] HGP scientists used white blood cells from the blood of two male and two female donors (randomly selected from 20 of each) – each donor yielding a separate DNA library. One of these libraries (RP11) was used considerably more than others, because of quality considerations. One minor technical issue is that male samples contain just over half as much DNA from the sex chromosomes (one X chromosome and one Y chromosome) compared to female samples (which contain two X chromosomes). The other 22 chromosomes (the autosomes) are the same for both sexes.Although the main sequencing phase of the HGP has been completed, studies of DNA variation continued in the International HapMap Project, whose goal was to identify patterns of single-nucleotide polymorphism (SNP) groups (called haplotypes, or \"haps\"). The DNA samples for the HapMap came from a total of 270 individuals; Yoruba people in Ibadan, Nigeria; Japanese people in Tokyo; Han Chinese in Beijing; and the French Centre d\\'Etude du Polymorphisme Humain (CEPH) resource, which consisted of residents of the United States having ancestry from Western and Northern Europe.In the Celera Genomics private-sector project, DNA from five different individuals were used for sequencing. The lead scientist of Celera Genomics at that time, Craig Venter, later acknowledged (in a public letter to the journal Science) that his DNA was one of 21 samples in the pool, five of which were selected for use.[72][73] Developments[edit] With the sequence in hand, the next step was to identify the genetic variants that increase the risk for common diseases like cancer and diabetes.[23][62]It is anticipated that detailed knowledge of the human genome will provide new avenues for advances in medicine and biotechnology. Clear practical results of the project emerged even before the work was finished. For example, a number of companies, such as Myriad Genetics, started offering easy ways to administer genetic tests that can show predisposition to a variety of illnesses, including breast cancer, hemostasis disorders, cystic fibrosis, liver diseases and many others. Also, the etiologies for cancers, Alzheimer\\'s disease and other areas of clinical interest are considered likely to benefit from genome information and possibly may lead in the long term to significant advances in their management.[74][75]There are also many tangible benefits for biologists. For example, a researcher investigating a certain form of cancer may have narrowed down their search to a particular gene. By visiting the human genome database on the World Wide Web, this researcher can examine what other scientists have written about this gene, including (potentially) the three-dimensional structure of its product, its functions, its evolutionary relationships to other human genes, or to genes in mice, yeast, or fruit flies, possible detrimental mutations, interactions with other genes, body tissues in which this gene is activated, and diseases associated with this gene or other datatypes. Further, a deeper understanding of the disease processes at the level of molecular biology may determine new therapeutic procedures. Given the established importance of DNA in molecular biology and its central role in determining the fundamental operation of cellular processes, it is likely that expanded knowledge in this area will facilitate medical advances in numerous areas of clinical interest that may not have been possible without them.[76]The analysis of similarities between DNA sequences from different organisms is also opening new avenues in the study of evolution. In many cases, evolutionary questions can now be framed in terms of molecular biology; indeed, many major evolutionary milestones (the emergence of the ribosome and organelles, the development of embryos with body plans, the vertebrate immune system) can be related to the molecular level. Many questions about the similarities and differences between humans and their closest relatives (the primates, and indeed the other mammals) are expected to be illuminated by the data in this project.[74][77]The project inspired and paved the way for genomic work in other fields, such as agriculture. For example, by studying the genetic composition of Tritium aestivum, the world\\'s most commonly used bread wheat, great insight has been gained into the ways that domestication has impacted the evolution of the plant.[78] It is being investigated which loci are most susceptible to manipulation, and how this plays out in evolutionary terms. Genetic sequencing has allowed these questions to be addressed for the first time, as specific loci can be compared in wild and domesticated strains of the plant. This will allow for advances in the genetic modification in the future which could yield healthier and disease-resistant wheat crops, among other things. Ethical, legal, and social issues[edit] At the onset of the Human Genome Project, several ethical, legal, and social concerns were raised in regard to how increased knowledge of the human genome could be used to discriminate against people. One of the main concerns of most individuals was the fear that both employers and health insurance companies would refuse to hire individuals or refuse to provide insurance to people because of a health concern indicated by someone\\'s genes.[79] In 1996, the United States passed the Health Insurance Portability and Accountability Act (HIPAA), which protects against the unauthorized and non-consensual release of individually identifiable health information to any entity not actively engaged in the provision of healthcare services to a patient.[80]Along with identifying all of the approximately 20,000–25,000 genes in the human genome (estimated at between 80,000 and 140,000 at the start of the project), the Human Genome Project also sought to address the ethical, legal, and social issues that were created by the onset of the project.[81] For that, the Ethical, Legal, and Social Implications (ELSI) program was founded in 1990. Five percent of the annual budget was allocated to address the ELSI arising from the project.[24][82] This budget started at approximately $1.57 million in the year 1990, but increased to approximately $18 million in the year 2014.[83]Whilst the project may offer significant benefits to medicine and scientific research, some authors have emphasized the need to address the potential social consequences of mapping the human genome. Historian of science Hans-Jörg Rheinberger wrote that \"the prospect of \\'molecularizing\\' diseases and their possible cure will have a profound impact on what patients expect from medical help, and on a new generation of doctors\\' perception of illness.\"[84] See also[edit] References[edit] * ^ Robert Krulwich (2003). Cracking the Code of Life (Television Show). PBS., * ^ \"Economic Impact of the Human Genome Project – Battelle\" (PDF). Archived from the original (PDF) on 23 December 2012. Retrieved 1 August 2013., * ^ \"Human Genome Project Completion: Frequently Asked Questions\". National Human Genome Research Institute (NHGRI)., * ^ \"CHM13 T2T v1.1 – Genome – Assembly – NCBI\". www.ncbi.nlm.nih.gov. Retrieved 16 June 2021., * ^ \"Genome List – Genome – NCBI\". www.ncbi.nlm.nih.gov. Retrieved 16 June 2021., * ^ \"T2T-CHM13v2.0 – Genome – Assembly – NCBI\". www.ncbi.nlm.nih.gov. Retrieved 14 June 2022., * ^ a b c \"Human Genome Project Completion: Frequently Asked Questions\". genome.gov., * ^ \"Human Genome Project: Sequencing the Human Genome | Learn Science at Scitable\". www.nature.com. Retrieved 25 January 2016., * ^ \"History of the Human Genome Project\". web.ornl.gov., * ^ Sinsheimer RL (November 1989). \"The Santa Cruz Workshop – May 1985 and\". Genomics. 5 (4): 954–956. doi:10.1016/0888-7543(89)90142-0. PMID 2591974., * ^ a b DeLisi C (October 2008). \"Meetings that changed the world: Santa Fe 1986: Human genome baby-steps\". Nature. 455 (7215): 876–877. Bibcode:2008Natur.455..876D. doi:10.1038/455876a. PMID 18923499. S2CID 41637733., * ^ Dulbecco R (March 1986). \"A turning point in cancer research: sequencing the human genome\". Science. 231 (4742): 1055–1056. Bibcode:1986Sci...231.1055D. doi:10.1126/science.3945817. PMID 3945817., * ^ Cook-Deegan, Robert M. (1994). The Gene Wars: Science, Politics, and the Human Genome. New York: W.W. Norton. p. 108., * ^ \"President Clinton Awards the Presidential Citizens Medals\". clintonwhitehouse5.archives.gov., * ^ \"File:Plaque commemorating the Human Genome Project, outside Charles DeLisi\\'s former office at DOE.png – Wikimedia Commons\"., * ^ Bevatron\\'s Encyclopedia of Inventions: a compendium of technological leaps, ground break discoveries and scientific breakthroughs that changed the world. The Human Genome Project, Charles DeLisi, pp. 360–362., * ^ Origins of the Human Genome Project: A Political History – Bob Cook-Deegan https://www.youtube.com/watch?v=-opMu4Ld21Q&t=3885s, * ^ a b Gene Wars, Op.Cit. p. 102., * ^ \"Search\". georgetown.edu., * ^ \"President Clinton Awards the Presidential Citizens Medals\". nara.gov. Archived from the original on 1 August 2012. Retrieved 6 August 2014., * ^ \"Archived copy\". Archived from the original on 3 March 2016. Retrieved 19 August 2013.{{cite web}}: CS1 maint: archived copy as title (link), * ^ DeLisi C (1988). \"The Human Genome Project\". American Scientist. 76 (5): 488. Bibcode:1988AmSci..76..488D., * ^ a b \"About the Human Genome Project: What is the Human Genome Project\". The Human Genome Management Information System (HGMIS). 18 July 2011. Archived from the original on 2 September 2011. Retrieved 2 September 2011., * ^ a b c Human Genome Information Archive. \"About the Human Genome Project\". U.S. Department of Energy & Human Genome Project program. Archived from the original on 2 September 2011. Retrieved 1 August 2013., * ^ Collins F, Galas D (1 October 1993). \"A New Five-Year Plan for the United States: Human Genome Program\". National Human Genome Research Institute. Retrieved 1 August 2013., * ^ \"Life on Earth to have its DNA analysed in the name of conservation\". Nature. 563 (7730): 155–156. November 2018. Bibcode:2018Natur.563..155.. doi:10.1038/d41586-018-07323-y. PMID 30401859., * ^ Lewin HA, Robinson GE, Kress WJ, Baker WJ, Coddington J, Crandall KA, et al. (April 2018). \"Earth BioGenome Project: Sequencing life for the future of life\". Proceedings of the National Academy of Sciences of the United States of America. 115 (17): 4325–4333. Bibcode:2018PNAS..115.4325L. doi:10.1073/pnas.1720115115. PMC 5924910. PMID 29686065., * ^ Cook-Deegan, Robert M. (1994). The Gene Wars: Science, Politics, and the Human Genome. New York: W.W. Norton. pp. 95–96., * ^ Bishop, Jerry E.; Waldholz, Michael (1990). Genome: The Story of the Most Astonishing Scientific Adventure of Our Time--the Attempt to Map All the Genes in the Human Body. New York: Simon and Schuster. p. 54., * ^ Bishop, Jerry E.; Waldholz, Michael (1990). Genome: The Story of the Most Astonishing Scientific Adventure of Our Time – the Attempt to Map All the Genes in the Human Body. New York: Simon and Schuster. p. 201., * ^ \"White House Press Release\". Retrieved 22 July 2006., * ^ \"Scientists Complete Rough Draft of Human Genome\"., * ^ Gitschier, Jane (31 January 2013). \"Life, the Universe, and Everything: An Interview with David Haussler\". PLOS Genetics. 9 (1): e1003282. doi:10.1371/journal.pgen.1003282. ISSN 1553-7390. PMC 3561096. PMID 23382705., * ^ Noble I (14 April 2003). \"Human genome finally complete\". BBC News. Retrieved 22 July 2006., * ^ Kolata G (15 April 2013). \"Human Genome, Then and Now\". The New York Times. Retrieved 24 April 2014., * ^ \"Guardian Unlimited |UK Latest | Human Genome Project finalised\". The Guardian. London. Retrieved 22 July 2006.[dead link], * ^ Лаане, Дарья (Laane, Daria); Петухов, Сергей (Petukhov, Sergei) (26 September 2005). \"Лицо русской национальности\" [Face of Russian nationality]. Kommersant (in Russian). Archived from the original on 23 February 2007. Retrieved 6 January 2021., * ^ \"The Human Genome Project FAQ\". Genoscope. Centre National de Séquençage. 19 October 2013. Archived from the original on 22 July 2015. Retrieved 12 February 2015., * ^ Schmutz J, Wheeler J, Grimwood J, Dickson M, Yang J, Caoile C, et al. (May 2004). \"Quality assessment of the human genome sequence\". Nature. 429 (6990): 365–368. Bibcode:2004Natur.429..365S. doi:10.1038/nature02390. PMID 15164052., * ^ Dolgin E (December 2009). \"Human genomics: The genome finishers\". Nature. 462 (7275): 843–845. doi:10.1038/462843a. PMID 20016572., * ^ Chaisson MJ, Huddleston J, Dennis MY, Sudmant PH, Malig M, Hormozdiari F, Antonacci F, Surti U, Sandstrom R, Boitano M, Landolin JM, Stamatoyannopoulos JA, Hunkapiller MW, Korlach J, Eichler EE (January 2015). \"Resolving the complexity of the human genome using single-molecule sequencing\". Nature. 517 (7536): 608–611. Bibcode:2015Natur.517..608C. doi:10.1038/nature13907. PMC 4317254. PMID 25383537., * ^ \"Human Genome Issues\". Genome Reference Consortium. Retrieved 29 June 2019., * ^ The (near) complete sequence of a human genome, 6 October 2020, * ^ Miga, Karen H.; Koren, Sergey; Rhie, Arang; Vollger, Mitchell R.; Gershman, Ariel; Bzikadze, Andrey; Brooks, Shelise; Howe, Edmund; Porubsky, David; Logsdon, Glennis A.; Schneider, Valerie A. (September 2020). \"Telomere-to-telomere assembly of a complete human X chromosome\". Nature. 585 (7823): 79–84. Bibcode:2020Natur.585...79M. doi:10.1038/s41586-020-2547-7. ISSN 1476-4687. PMC 7484160. PMID 32663838., * ^ Logsdon, Glennis A.; Vollger, Mitchell R.; Hsieh, PingHsun; Mao, Yafei; Liskovykh, Mikhail A.; Koren, Sergey; Nurk, Sergey; Mercuri, Ludovica; Dishuck, Philip C.; Rhie, Arang; de Lima, Leonardo G. (May 2021). \"The structure, function and evolution of a complete human chromosome 8\". Nature. 593 (7857): 101–107. Bibcode:2021Natur.593..101L. doi:10.1038/s41586-021-03420-7. ISSN 1476-4687. PMC 8099727. PMID 33828295., * ^ Wrighton, Katharine (February 2021). \"Filling in the gaps telomere to telomere\". Nature Milestones: Genomic Sequencing: S21., * ^ Reardon, Sara (4 June 2021). \"A complete human genome sequence is close: how scientists filled in the gaps\". Nature. 594 (7862): 158–159. Bibcode:2021Natur.594..158R. doi:10.1038/d41586-021-01506-w. PMID 34089035. S2CID 235346408., * ^ \"GitHub – marbl/CHM13-issues: CHM13 human reference genome issue tracking\". GitHub. Retrieved 26 July 2021., * ^ Special Issue: Completing the human genome, Science, vol. 376, no. 6588 (1 April 2022)., * ^ Rhie; et al. (1 December 2022). \"The complete sequence of a human Y chromosome\". bioRxiv. doi:10.1101/2022.12.01.518724. S2CID 254181409., * ^ \"UCSC Genome Browser Home\". genome.ucsc.edu., * ^ \"Ensembl Genome Browser\". ensembl.org., * ^ Mardis ER (March 2008). \"The impact of next-generation sequencing technology on genetics\". Trends in Genetics. 24 (3): 133–141. doi:10.1016/j.tig.2007.12.007. PMC 2680276. PMID 18262675., * ^ Liu Y, Gonzàlez-Porta M, Santos S, Brazma A, Marioni JC, Aebersold R, et al. (August 2017). \"Impact of Alternative Splicing on the Human Proteome\". Cell Reports. 20 (5): 1229–1241. doi:10.1016/j.celrep.2017.07.025. PMC 5554779. PMID 28768205., * ^ Ballouz S, Dobin A, Gillis JA (August 2019). \"Is it time to change the reference genome?\". Genome Biology. 20 (1): 159. doi:10.1186/s13059-019-1774-4. PMC 6688217. PMID 31399121., * ^ Pertea M, Salzberg SL (2010). \"Between a chicken and a grape: estimating the number of human genes\". Genome Biology. 11 (5): 206. doi:10.1186/gb-2010-11-5-206. PMC 2898077. PMID 20441615., * ^ a b c Venter JC, Adams MD, Myers EW, Li PW, Mural RJ, Sutton GG, et al. (February 2001). \"The sequence of the human genome\". Science. 291 (5507): 1304–1351. Bibcode:2001Sci...291.1304V. doi:10.1126/science.1058040. PMID 11181995., * ^ Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, et al. (International Human Genome Sequencing Consortium (IHGSC)) (October 2004). \"Finishing the euchromatic sequence of the human genome\". Nature. 431 (7011): 931–945. Bibcode:2004Natur.431..931H. doi:10.1038/nature03001. PMID 15496913., * ^ Spencer G (20 December 2004). \"International Human Genome Sequencing Consortium Describes Finished Human Genome Sequence\". NIH Nes Release. National Institutes of Health., * ^ Bryant JA (2007). Design and information in biology: From molecules to systems. p. 108. ISBN 9781853128530. ...brought to light about 1200 protein families. Only 94 protein families, or 7%, appear to be vertebrate specific, * ^ Piovesan, A.; Pelleri, M. C.; Antonaros, F.; Strippoli, P.; Caracausi, M.; Vitale, L. (2019). \"On the length, weight and GC content of the human genome\". BMC Research Notes. 12 (1): 106. doi:10.1186/s13104-019-4137-z. PMC 6391780. PMID 30813969., * ^ a b Wellcome Sanger Institute. \"The Human Genome Project: a new reality\". Wellcome Trust Sanger Institute, Genome Research Limited. Archived from the original on 1 August 2013. Retrieved 1 August 2013., * ^ \"Celera: A Unique Approach to Genome Sequencing\". ocf.berkeley.edu. Biocomputing. 2006. Retrieved 1 August 2013., * ^ Davidson College (2002). \"Sequencing Whole Genomes: Hierarchical Shotgun Sequencing v. Shotgun Sequencing\". bio.davidson.edu. Department of Biology, Davidson College. Retrieved 1 August 2013., * ^ Human Genome Project Information Archive (2013). \"U.S. & International HGP Research Sites\". U.S. Department of Energy & Human Genome Project. Retrieved 1 August 2013., * ^ Vizzini C (19 March 2015). \"The Human Variome Project: Global Coordination in Data Sharing\". Science & Diplomacy. 4 (1)., * ^ Roach JC, Boysen C, Wang K, Hood L (March 1995). \"Pairwise end sequencing: a unified approach to genomic mapping and sequencing\". Genomics. 26 (2): 345–353. doi:10.1016/0888-7543(95)80219-C. PMID 7601461., * ^ Center for Biomolecular Science & Engineering. \"The Human Genome Project Race\". Center for Biomolecular Science and Engineering. Retrieved 1 August 2013., * ^ Gillis, Justin (15 March 2000). \"Clinton, Blair Urge Open Access to Gene Data\". Washington Post., * ^ Osoegawa K, Mammoser AG, Wu C, Frengen E, Zeng C, Catanese JJ, de Jong PJ (March 2001). \"A bacterial artificial chromosome library for sequencing the complete human genome\". Genome Research. 11 (3): 483–496. doi:10.1101/gr.169601. PMC 311044. PMID 11230172., * ^ Tuzun E, Sharp AJ, Bailey JA, Kaul R, Morrison VA, Pertz LM, et al. (July 2005). \"Fine-scale structural variation of the human genome\". Nature Genetics. 37 (7): 727–732. doi:10.1038/ng1562. PMID 15895083. S2CID 14162962., * ^ Kennedy D (August 2002). \"Not wicked, perhaps, but tacky\". Science. 297 (5585): 1237. doi:10.1126/science.297.5585.1237. PMID 12193755., * ^ Venter JC (February 2003). \"A part of the human genome sequence\". Science. 299 (5610): 1183–1184. doi:10.1126/science.299.5610.1183. PMID 12595674. S2CID 5188811., * ^ a b Naidoo N, Pawitan Y, Soong R, Cooper DN, Ku CS (October 2011). \"Human genetics and genomics a decade after the release of the draft sequence of the human genome\". Human Genomics. 5 (6): 577–622. doi:10.1186/1479-7364-5-6-577. PMC 3525251. PMID 22155605., * ^ Gonzaga-Jauregui C, Lupski JR, Gibbs RA (2012). \"Human genome sequencing in health and disease\". Annual Review of Medicine. 63 (1): 35–61. doi:10.1146/annurev-med-051010-162644. PMC 3656720. PMID 22248320., * ^ Snyder M, Du J, Gerstein M (March 2010). \"Personal genome sequencing: current approaches and challenges\". Genes & Development. 24 (5): 423–31. doi:10.1101/gad.1864110. PMC 2827837. PMID 20194435., * ^ Lander ES (February 2011). \"Initial impact of the sequencing of the human genome\" (PDF). Nature. 470 (7333): 187–97. Bibcode:2011Natur.470..187L. doi:10.1038/nature09792. hdl:1721.1/69154. PMID 21307931. S2CID 4344403., * ^ Peng JH, Sun D, Nevo E (2011). \"Domestication Evolution, Genetics And Genomics In Wheat\". Molecular Breeding. 28 (3): 281–301. doi:10.1007/s11032-011-9608-4. S2CID 24886686., * ^ Greely H (1992). The Code of Codes: Scientific and Social Issues in the Human Genome Project. Cambridge, Massachusetts: Harvard University Press. pp. 264–265. ISBN 978-0-674-13646-5., * ^ US Department of Health and Human Services (26 August 2015). \"Understanding Health Information Privacy\"., * ^ Human Genome Information Archive. \"Insights Learned from the Human DNA Sequence\". U.S. Department of Energy & Human Genome Project program. Archived from the original on 3 September 2011. Retrieved 20 February 2021., * ^ \"What were some of the ethical, legal, and social implications addressed by the Human Genome Project?\". Genetics Home Reference. U.S. National Library of Medicine. 2013. Retrieved 1 August 2013., * ^ \"ELSI Research Program Fact Sheet – National Human Genome Research Institute (NHGRI)\". www.genome.gov. Retrieved 27 September 2016., * ^ Rheinberger HJ (2000). Living and Working with the New Medical Technologies. Cambridge: Cambridge University Press. p. 20., Further reading[edit] External links[edit] Wikinews has related news: * National Human Genome Research Institute (NHGRI). NHGRI led the National Institutes of Health\\'s contribution to the International Human Genome Project. This project, which had as its primary goal the sequencing of the three billion base pairs that make up the human genome, was successfully completed in April 2003., * Human Genome News. Published from 1989 to 2002 by the US Department of Energy, this newsletter was a major communications method for coordination of the Human Genome Project. Complete online archives are available., * The HGP information pages Department of Energy\\'s portal to the international Human Genome Project, Microbial Genome Program, and Genomics:GTL systems biology for energy and environment, * yourgenome.org: The Sanger Institute public information pages has general and detailed primers on DNA, genes, and genomes, the Human Genome Project and science spotlights., * Ensembl project, an automated annotation system and browser for the human genome, * UCSC genome browser, This site contains the reference sequence and working draft assemblies for a large collection of genomes. It also provides a portal to the ENCODE project., * Nature magazine\\'s human genome gateway, including the HGP\\'s paper on the draft genome sequence, * Wellcome Trust Human Genome website A free resource allowing you to explore the human genome, your health and your future., * Learning about the Human Genome. Part 1: Challenge to Science Educators. ERIC Digest., * Learning about the Human Genome. Part 2: Resources for Science Educators. ERIC Digest., * Patenting Life by Merrill Goozner, * Prepared Statement of Craig Venter of Celera Venter discusses Celera\\'s progress in deciphering the human genome sequence and its relationship to healthcare and to the federally funded Human Genome Project., * Cracking the Code of Life Companion website to 2-hour NOVA program documenting the race to decode the genome, including the entire program hosted in 16 parts in either QuickTime or RealPlayer format., * Bioethics Research Library Numerous original documents at Georgetown University., Works by archive * Works by Human Genome Project at Project Gutenberg Project Gutenberg hosts e-texts for Human Genome Project, titled Human Genome Project, Chromosome Number # (# denotes 01–22, X and Y). This information is the raw sequence, released in November 2002; access to entry pages with download links is available through Human Genome Project, Chromosome Number 01 for Chromosome 1 sequentially to Human Genome Project, Y Chromosome for the Y Chromosome. Note that this sequence might not be considered definitive because of ongoing revisions and refinements. In addition to the chromosome files, there is a supplementary information file dated March 2004 which contains additional sequence information., * Works by or about Human Genome Project at Internet Archive,', metadata={'source': 'https://en.wikipedia.org/wiki/Human_Genome_Project', 'page_title': 'Human Genome Project - Wikipedia'}),\n",
       " Document(page_content='Sign in | Create an account https://orcid.org', metadata={'source': 'https://europepmc.org/article/med/8489331', 'page_title': 'Europe PMC'}),\n",
       " Document(page_content='Genome Medicine volume 5, Article number: 79 (2013) Cite this article * 127k Accesses, * 130 Citations, * 113 Altmetric, * Metrics details, AbstractThe Human Genome Project has transformed biology through its integrated big science approach to deciphering a reference human genome sequence along with the complete sequences of key model organisms. The project exemplifies the power, necessity and success of large, integrated, cross-disciplinary efforts - so-called ‘big science’ - directed towards complex major objectives. In this article, we discuss the ways in which this ambitious endeavor led to the development of novel technologies and analytical tools, and how it brought the expertise of engineers, computer scientists and mathematicians together with biologists. It established an open approach to data sharing and open-source software, thereby making the data resulting from the project accessible to all. The genome sequences of microbes, plants and animals have revolutionized many fields of science, including microbiology, virology, infectious disease and plant biology. Moreover, deeper knowledge of human sequence variation has begun to alter the practice of medicine. The Human Genome Project has inspired subsequent large-scale data acquisition initiatives such as the International HapMap Project, 1000 Genomes, and The Cancer Genome Atlas, as well as the recently announced Human Brain Project and the emerging Human Proteome Project. Origins of the human genome projectThe Human Genome Project (HGP) has profoundly changed biology and is rapidly catalyzing a transformation of medicine [1–3]. The idea of the HGP was first publicly advocated by Renato Dulbecco in an article published in 1984, in which he argued that knowing the human genome sequence would facilitate an understanding of cancer [4]. In May 1985 a meeting focused entirely on the HGP was held, with Robert Sinsheimer, the Chancellor of the University of California, Santa Cruz (UCSC), assembling 12 experts to debate the merits of this potential project [5]. The meeting concluded that the project was technically possible, although very challenging. However, there was controversy as to whether it was a good idea, with six of those assembled declaring themselves for the project, six against (and those against felt very strongly). The naysayers argued that big science is bad science because it diverts resources from the ‘real’ small science (such as single investigator science); that the genome is mostly junk that would not be worth sequencing; that we were not ready to undertake such a complex project and should wait until the technology was adequate for the task; and that mapping and sequencing the genome was a routine and monotonous task that would not attract appropriate scientific talent. Throughout the early years of advocacy for the HGP (mid- to late 1980s) perhaps 80% of biologists were against it, as was the National Institutes of Health (NIH) [6]. The US Department of Energy (DOE) initially pushed for the HGP, partly using the argument that knowing the genome sequence would help us understand the radiation effects on the human genome resulting from exposure to atom bombs and other aspects of energy transmission [7]. This DOE advocacy was critical to stimulating the debate and ultimately the acceptance of the HGP. Curiously, there was more support from the US Congress than from most biologists. Those in Congress understood the appeal of international competitiveness in biology and medicine, the potential for industrial spin-offs and economic benefits, and the potential for more effective approaches to dealing with disease. A National Academy of Science committee report endorsed the project in 1988 [8] and the tide of opinion turned: in 1990, the program was initiated, with the finished sequence published in 2004 ahead of schedule and under budget [9].What did the human genome project entail?This 3-billion-dollar, 15-year program evolved considerably as genomics technologies improved. Initially, the HGP set out to determine a human genetic map, then a physical map of the human genome [10], and finally the sequence map. Throughout, the HGP was instrumental in pushing the development of high-throughput technologies for preparing, mapping and sequencing DNA [11]. At the inception of the HGP in the early 1990s, there was optimism that the then-prevailing sequencing technology would be replaced. This technology, now called ‘first-generation sequencing’, relied on gel electrophoresis to create sequencing ladders, and radioactive- or fluorescent-based labeling strategies to perform base calling [12]. It was considered to be too cumbersome and low throughput for efficient genomic sequencing. As it turned out, the initial human genome reference sequence was deciphered using a 96-capillary (highly parallelized) version of first-generation technology. Alternative approaches such as multiplexing [13] and sequencing by hybridization [14] were attempted but not effectively scaled up. Meanwhile, thanks to the efforts of biotech companies, successive incremental improvements in the cost, throughput, speed and accuracy of first-generation automated fluorescent-based sequencing strategies were made throughout the duration of the HGP. Because biologists were clamoring for sequence data, the goal of obtaining a full-fledged physical map of the human genome was abandoned in the later stages of the HGP in favor of generating the sequence earlier than originally planned. This push was accelerated by Craig Venter’s bold plan to create a company (Celera) for the purpose of using a whole-genome shotgun approach [15] to decipher the sequence instead of the piecemeal clone-by-clone approach using bacterial artificial chromosome (BAC) vectors that was being employed by the International Consortium. Venter’s initiative prompted government funding agencies to endorse production of a clone-based draft sequence for each chromosome, with the finishing to come in a subsequent phase. These parallel efforts accelerated the timetable for producing a genome sequence of immense value to biologists [16, 17].As a key component of the HGP, it was wisely decided to sequence the smaller genomes of significant experimental model organisms such as yeast, a small flowering plant (Arabidopsis thaliana), worm and fruit fly before taking on the far more challenging human genome. The efforts of multiple centers were integrated to produce these reference genome sequences, fostering a culture of cooperation. There were originally 20 centers mapping and sequencing the human genome as part of an international consortium [18]; in the end five large centers (the Wellcome Trust Sanger Institute, the Broad Institute of MIT and Harvard, The Genome Institute of Washington University in St Louis, the Joint Genome Institute, and the Whole Genome Laboratory at Baylor College of Medicine) emerged from this effort, with these five centers continuing to provide genome sequence and technology development. The HGP also fostered the development of mathematical, computational and statistical tools for handling all the data it generated.The HGP produced a curated and accurate reference sequence for each human chromosome, with only a small number of gaps, and excluding large heterochromatic regions [9]. In addition to providing a foundation for subsequent studies in human genomic variation, the reference sequence has proven essential for the development and subsequent widespread use of second-generation sequencing technologies, which began in the mid-2000s. Second-generation cyclic array sequencing platforms produce, in a single run, up to hundreds of millions of short reads (originally approximately 30 to 70 bases, now up to several hundred bases), which are typically mapped to a reference genome at highly redundant coverage [19]. A variety of cyclic array sequencing strategies (such as RNA-Seq, ChIP-Seq, bisulfite sequencing) have significantly advanced biological studies of transcription and gene regulation as well as genomics, progress for which the HGP paved the way.Impact of the human genome project on biology and technologyFirst, the human genome sequence initiated the comprehensive discovery and cataloguing of a ‘parts list’ of most human genes [16, 17], and by inference most human proteins, along with other important elements such as non-coding regulatory RNAs. Understanding a complex biological system requires knowing the parts, how they are connected, their dynamics and how all of these relate to function [20]. The parts list has been essential for the emergence of ‘systems biology’, which has transformed our approaches to biology and medicine [21, 22].As an example, the ENCODE (Encyclopedia Of DNA Elements) Project, launched by the NIH in 2003, aims to discover and understand the functional parts of the genome [23]. Using multiple approaches, many based on second-generation sequencing, the ENCODE Project Consortium has produced voluminous and valuable data related to the regulatory networks that govern the expression of genes [24]. Large datasets such as those produced by ENCODE raise challenging questions regarding genome functionality. How can a true biological signal be distinguished from the inevitable biological noise produced by large datasets [25, 26]? To what extent is the functionality of individual genomic elements only observable (used) in specific contexts (for example, regulatory networks and mRNAs that are operative only during embryogenesis)? It is clear that much work remains to be done before the functions of poorly annotated protein-coding genes will be deciphered, let alone those of the large regions of the non-coding portions of the genome that are transcribed. What is signal and what is noise is a critical question.Second, the HGP also led to the emergence of proteomics, a discipline focused on identifying and quantifying the proteins present in discrete biological compartments, such as a cellular organelle, an organ or the blood. Proteins - whether they act as signaling devices, molecular machines or structural components - constitute the cell-specific functionality of the parts list of an organism’s genome. The HGP has facilitated the use of a key analytical tool, mass spectrometry, by providing the reference sequences and therefore the predicted masses of all the tryptic peptides in the human proteome - an essential requirement for the analysis of mass-spectrometry-based proteomics [27]. This mass-spectrometry-based accessibility to proteomes has driven striking new applications such as targeted proteomics [28]. Proteomics requires extremely sophisticated computational techniques, examples of which are PeptideAtlas [29] and the Trans-Proteomic Pipeline [30].Third, our understanding of evolution has been transformed. Since the completion of the HGP, over 4,000 finished or quality draft genome sequences have been produced, mostly from bacterial species but including 183 eukaryotes [31]. These genomes provide insights into how diverse organisms from microbes to human are connected on the genealogical tree of life - clearly demonstrating that all of the species that exist today descended from a single ancestor [32]. Questions of longstanding interest with implications for biology and medicine have become approachable. Where do new genes come from? What might be the role of stretches of sequence highly conserved across all metazoa? How much large-scale gene organization is conserved across species and what drives local and global genome reorganization? Which regions of the genome appear to be resistant (or particularly susceptible) to mutation or highly susceptible to recombination? How do regulatory networks evolve and alter patterns of gene expression [33]? The latter question is of particular interest now that the genomes of several primates and hominids have been or are being sequenced [34, 35] in hopes of shedding light on the evolution of distinctively human characteristics. The sequence of the Neanderthal genome [36] has had fascinating implications for human evolution; namely, that a few percent of Neanderthal DNA and hence the encoded genes are intermixed in the human genome, suggesting that there was some interbreeding while the two species were diverging [36, 37].Fourth, the HGP drove the development of sophisticated computational and mathematical approaches to data and brought computer scientists, mathematicians, engineers and theoretical physicists together with biologists, fostering a more cross-disciplinary culture [1, 21, 38]. It is important to note that the HGP popularized the idea of making data available to the public immediately in user-friendly databases such as GenBank [39] and the UCSC Genome Browser [40]. Moreover, the HGP also promoted the idea of open-source software, in which the source code of programs is made available to and can be edited by those interested in extending their reach and improving them [41, 42]. The open-source operating system of Linux and the community it has spawned have shown the power of this approach. Data accessibility is a critical concept for the culture and success of biology in the future because the ‘democratization of data’ is critical for attracting available talent to focus on the challenging problems of biological systems with their inherent complexity [43]. This will be even more critical in medicine, as scientists need access to the data cloud available from each individual human to mine for the predictive medicine of the future - an effort that could transform the health of our children and grandchildren [44].Fifth, the HGP, as conceived and implemented, was the first example of ‘big science’ in biology, and it clearly demonstrated both the power and the necessity of this approach for dealing with its integrated biological and technological aims. The HGP was characterized by a clear set of ambitious goals and plans for achieving them; a limited number of funded investigators typically organized around centers or consortia; a commitment to public data/resource release; and a need for significant funding to support project infrastructure and new technology development. Big science and smaller-scope individual-investigator-oriented science are powerfully complementary, in that the former generates resources that are foundational for all researchers while the latter adds detailed experimental clarification of specific questions, and analytical depth and detail to the data produced by big science. There are many levels of complexity in biology and medicine; big science projects are essential to tackle this complexity in a comprehensive and integrative manner [45].The HGP benefited biology and medicine by creating a sequence of the human genome; sequencing model organisms; developing high-throughput sequencing technologies; and examining the ethical and social issues implicit in such technologies. It was able to take advantage of economies of scale and the coordinated effort of an international consortium with a limited number of players, which rendered the endeavor vastly more efficient than would have been possible if the genome were sequenced on a gene-by-gene basis in small labs. It is also worth noting that one aspect that attracted governmental support to the HGP was its potential for economic benefits. The Battelle Institute published a report on the economic impact of the HGP [46]. For an initial investment of approximately $3.5 billion, the return, according to the report, has been about $800 billion - a staggering return on investment.Even today, as budgets tighten, there is a cry to withdraw support from big science and focus our resources on small science. This would be a drastic mistake. In the wake of the HGP there are further valuable biological resource-generating projects and analyses of biological complexity that require a big science approach, including the HapMap Project to catalogue human genetic variation [47, 48], the ENCODE project, the Human Proteome Project (described below) and the European Commission’s Human Brain Project, as well as another brain-mapping project recently announced by President Obama [49]. Similarly to the HGP, significant returns on investment will be possible for other big science projects that are now under consideration if they are done properly. It should be stressed that discretion must be employed in choosing big science projects that are fundamentally important. Clearly funding agencies should maintain a mixed portfolio of big and small science - and the two are synergistic [1, 45].Last, the HGP ignited the imaginations of unusually talented scientists - Jim Watson, Eric Lander, John Sulston, Bob Waterston and Sydney Brenner to mention only a few. So virtually every argument initially posed by the opponents of the HGP turned out to be wrong. The HGP is a wonderful example of a fundamental paradigm change in biology: initially fiercely resisted, it was ultimately far more transformational than expected by even the most optimistic of its proponents.Impact of the human genome project on medicineSince the conclusion of the HGP, several big science projects specifically geared towards a better understanding of human genetic variation and its connection to human health have been initiated. These include the HapMap Project aimed at identifying haplotype blocks of common single nucleotide polymorphisms (SNPs) in different human populations [47, 48], and its successor, the 1000 Genomes project, an ongoing endeavor to catalogue common and rare single nucleotide and structural variation in multiple populations [50]. Data produced by both projects have supported smaller-scale clinical genome-wide association studies (GWAS), which correlate specific genetic variants with disease risk of varying statistical significance based on case–control comparisons. Since 2005, over 1,350 GWAS have been published [51]. Although GWAS analyses give hints as to where in the genome to look for disease-causing variants, the results can be difficult to interpret because the actual disease-causing variant might be rare, the sample size of the study might be too small, or the disease phenotype might not be well stratified. Moreover, most of the GWAS hits are outside of coding regions - and we do not have effective methods for easily determining whether these hits reflect the mis-functioning of regulatory elements. The question as to what fraction of the thousands of GWAS hits are signal and what fraction are noise is a concern. Pedigree-based whole-genome sequencing offers a powerful alternative approach to identifying potential disease-causing variants [52].Five years ago, a mere handful of personal genomes had been fully sequenced (for example, [53, 54]). Now there are thousands of exome and whole-genome sequences (soon to be tens of thousands, and eventually millions), which have been determined with the aim of identifying disease-causing variants and, more broadly, establishing well-founded correlations between sequence variation and specific phenotypes. For example, the International Cancer Genome Consortium [55] and The Cancer Genome Atlas [56] are undertaking large-scale genomic data collection and analyses for numerous cancer types (sequencing both the normal and cancer genome for each individual patient), with a commitment to making their resources available to the research community.We predict that individual genome sequences will soon play a larger role in medical practice. In the ideal scenario, patients or consumers will use the information to improve their own healthcare by taking advantage of prevention or therapeutic strategies that are known to be appropriate for real or potential medical conditions suggested by their individual genome sequence. Physicians will need to educate themselves on how best to advise patients who bring consumer genetic data to their appointments, which may well be a common occurrence in a few years [57].In fact, the application of systems approaches to disease has already begun to transform our understanding of human disease and the practice of healthcare and push us towards a medicine that is predictive, preventive, personalized and participatory: P4 medicine. A key assumption of P4 medicine is that in diseased tissues biological networks become perturbed - and change dynamically with the progression of the disease. Hence, knowing how the information encoded by disease-perturbed networks changes provides insights into disease mechanisms, new approaches to diagnosis and new strategies for therapeutics [58, 59].Let us provide some examples. First, pharmacogenomics has identified more than 70 genes for which specific variants cause humans to metabolize drugs ineffectively (too fast or too slow). Second, there are hundreds of ‘actionable gene variants’ - variants that cause disease but whose consequences can be avoided by available medical strategies with knowledge of their presence [60]. Third, in some cases, cancer-driving mutations in tumors, once identified, can be counteracted by treatments with currently available drugs [61]. And last, a systems approach to blood protein diagnostics has generated powerful new diagnostic panels for human diseases such as hepatitis [62] and lung cancer [63].These latter examples portend a revolution in blood diagnostics that will lead to early detection of disease, the ability to follow disease progression and responses to treatment, and the ability to stratify a disease type (for instance, breast cancer) into its different subtypes for proper impedance match against effective drugs [59]. We envision a time in the future when all patients will be surrounded by a virtual cloud of billions of data points, and when we will have the analytical tools to reduce this enormous data dimensionality to simple hypotheses to optimize wellness and minimize disease for each individual [58].Impact of the human genome project on societyThe HGP challenged biologists to consider the social implications of their research. Indeed, it devoted 5% of its budget to considering the social, ethical and legal aspects of acquiring and understanding the human genome sequence [64]. That process continues as different societal issues arise, such as genetic privacy, potential discrimination, justice in apportioning the benefits from genomic sequencing, human subject protections, genetic determinism (or not), identity politics, and the philosophical concept of what it means to be human beings who are intrinsically connected to the natural world.Strikingly, we have learned from the HGP that there are no race-specific genes in humans [65–68]. Rather, an individual’s genome reveals his or her ancestral lineage, which is a function of the migrations and interbreeding among population groups. We are one race and we honor our species’ heritage when we treat each other accordingly, and address issues of concern to us all, such as human rights, education, job opportunities, climate change and global health.What is to come?There remain fundamental challenges for fully understanding the human genome. For example, as yet at least 5% of the human genome has not been successfully sequenced or assembled for technical reasons that relate to eukaryotic islands being embedded in heterochromatic repeats, copy number variations, and unusually high or low GC content [69]. The question of what information these regions contain is a fascinating one. In addition, there are highly conserved regions of the human genome whose functions have not yet been identified; presumably they are regulatory, but why they should be strongly conserved over a half a billion years of evolution remains a mystery.There will continue to be advances in genome analysis. Developing improved analytical techniques to identify biological information in genomes and decipher what this information relates to functionally and evolutionarily will be important. Developing the ability to rapidly analyze complete human genomes with regard to actionable gene variants is essential. It is also essential to develop software that can accurately fold genome-predicted proteins into three dimensions, so that their functions can be predicted from structural homologies. Likewise, it will be fascinating to determine whether we can make predictions about the structures of biological networks directly from the information of their cognate genomes. Indeed, the idea that we can decipher the ‘logic of life’ of an organism solely from its genome sequence is intriguing. While we have become relatively proficient at determining static and stable genome sequences, we are still learning how to measure and interpret the dynamic effects of the genome: gene expression and regulation, as well as the dynamics and functioning of non-coding RNAs, metabolites, proteins and other products of genetically encoded information.The HGP, with its focus on developing the technology to enumerate a parts list, was critical for launching systems biology, with its concomitant focus on high-throughput ‘omics’ data generation and the idea of ‘big data’ in biology [21, 38]. The practice of systems biology begins with a complete parts list of the information elements of living organisms (for example, genes, RNAs, proteins and metabolites). The goals of systems biology are comprehensive yet open ended because, as seen with the HGP, the field is experiencing an infusion of talented scientists applying multidisciplinary approaches to a variety of problems. A core feature of systems biology, as we see it, is to integrate many different types of biological information to create the ‘network of networks’ - recognizing that networks operate at the genomic, the molecular, the cellular, the organ, and the social network levels, and that these are integrated in the individual organism in a seamless manner [58]. Integrating these data allows the creation of models that are predictive and actionable for particular types of organisms and individual patients. These goals require developing new types of high-throughput omic technologies and ever increasingly powerful analytical tools.The HGP infused a technological capacity into biology that has resulted in enormous increases in the range of research, for both big and small science. Experiments that were inconceivable 20 years ago are now routine, thanks to the proliferation of academic and commercial wet lab and bioinformatics resources geared towards facilitating research. In particular, rapid increases in throughput and accuracy of the massively parallel second-generation sequencing platforms with their correlated decreases in cost of sequencing have resulted in a great wealth of accessible genomic and transcriptional sequence data for myriad microbial, plant and animal genomes. These data in turn have enabled large- and small-scale functional studies that catalyze and enhance further research when the results are provided in publicly accessible databases [70].One descendant of the HGP is the Human Proteome Project, which is beginning to gather momentum, although it is still poorly funded. This exciting endeavor has the potential to be enormously beneficial to biology [71–73]. The Human Proteome Project aims to create assays for all human and model organism proteins, including the myriad protein isoforms produced from the RNA splicing and editing of protein-coding genes, chemical modifications of mature proteins, and protein processing. The project also aims to pioneer technologies that will achieve several goals: enable single-cell proteomics; create microfluidic platforms for thousands of protein enzyme-linked immunosorbent assays (ELISAs) for rapid and quantitative analyses of, for example, a fraction of a droplet of blood; develop protein-capture agents that are small, stable, easy to produce and can be targeted to specific protein epitopes and hence avoid extensive cross-reactivity; and develop the software that will enable the ordinary biologist to analyze the massive amounts of proteomics data that are beginning to emerge from human and other organisms.Newer generations of DNA sequencing platforms will be introduced that will transform how we gather genome information. Third-generation sequencing [74] will employ nanopores or nanochannels, utilize electronic signals, and sequence single DNA molecules for read lengths of 10,000 to 100,000 bases. Third-generation sequencing will solve many current problems with human genome sequences. First, contemporary short-read sequencing approaches make it impossible to assemble human genome sequences de novo; hence, they are usually compared against a prototype reference sequence that is itself not fully accurate, especially with respect to variations other than SNPs. This makes it extremely difficult to precisely identify the insertion-deletion and structural variations in the human genome, both for our species as a whole and for any single individual. The long reads of third-generation sequencing will allow for the de novo assembly of human (and other) genomes, and hence delineate all of the individually unique variability: nucleotide substitutions, indels, and structural variations. Second, we do not have global techniques for identifying the 16 different chemical modifications of human DNA (epigenetic marks, reviewed in [75]). It is increasingly clear that these epigenetic modifications play important roles in gene expression [76]. Thus, single-molecule analyses should be able to identify all the epigenetic marks on DNA. Third, single-molecule sequencing will facilitate the full-length sequencing of RNAs; thus, for example, enhancing interpretation of the transcriptome by enabling the identification of RNA editing, alternative splice forms with a given transcript, and different start and termination sites. Last, it is exciting to contemplate that the ability to parallelize this process (for example, by generating millions of nanopores that can be used simultaneously) could enable the sequencing of a human genome in 15 minutes or less [77]. The high-throughput nature of this sequencing may eventually lead to human genome costs of $100 or under. The interesting question is how long it will take to make third-generation sequencing a mature technology.The HGP has thus opened many avenues in biology, medicine, technology and computation that we are just beginning to explore. AbbreviationsBAC: Bacterial artificial chromosome DOE: Department of Energy ELISA: Enzyme-linked immunosorbent assay GWAS: Genome-wide association studies HGP: Human Genome Project NIH: National Institutes of Health SNP: Single nucleotide polymorphism UCSC: University of California, Santa Cruz. References* Hood L: Acceptance remarks for Fritz J. and Delores H. Russ Prize. The Bridge. 2011, 41: 46-49. Google Scholar, * Collins FS, McKusick VA: Implications of the Human Genome Project for medical science. JAMA. 2001, 285: 540-544. 10.1001/jama.285.5.540.Article CAS PubMed Google Scholar, * Green ED, Guyer MS, National Human Genome Research Institute: Charting a course for genomic medicine from base to bedside. Nature. 2011, 470: 204-213. 10.1038/nature09764.Article CAS PubMed Google Scholar, * Dulbecco R: A turning point in cancer research: sequencing the human genome. Science. 1984, 231: 1055-1056.Article Google Scholar, * Sinsheimer RL: The Santa Cruz workshop - May 1985. Genomics. 1989, 5: 954-956. 10.1016/0888-7543(89)90142-0.Article CAS PubMed Google Scholar, * Cooke-Degan RM: The Gene Wars: Science, Politics and the Human Genome. 1994, New York: WW Norton Google Scholar, * Report on the Human Genome Initiative for the Office of Health and Environmental Research. http://www.ornl.gov/sci/techresources/Human_Genome/project/herac2.shtml,, * National Academy of Science: Report of the Committee on Mapping and Sequencing the Human Genome. 1988, Washington DC: National Academy Press Google Scholar, * Human Genome Sequencing Consortium: Finishing the euchromatic sequence of the human genome. Nature. 2004, 431: 931-945. 10.1038/nature03001.Article Google Scholar, * Understanding Our Genetic Inheritance. The United States Human Genome Project, The First Five Years: Fiscal Years. 1991, http://www.genome.gov/10001477, –1995,, * Collins FS, Galas D: A new five-year plan for the U.S. Human Genome Program. Science. 1993, 262: 43-46. 10.1126/science.8211127.Article CAS PubMed Google Scholar, * Smith LM, Sanders JZ, Kaiser RJ, Hughes P, Dodd C, Connell CR, Heiner C, Kent SBH, Hood LE: Fluorescence detection in automated DNA sequence analysis. Nature. 1986, 321: 674-679. 10.1038/321674a0.Article CAS PubMed Google Scholar, * Church G, Kieffer-Higgins S: Multiplex DNA sequencing. Science. 1988, 240: 185-188. 10.1126/science.3353714.Article CAS PubMed Google Scholar, * Strezoska Z, Paunesku T, Radosavljević D, Labat I, Drmanac R, Crkvenjakov R: DNA sequencing by hybridization: 100 bases read by a non-gel-based method. Proc Natl Acad Sci USA. 1991, 88: 10089-10093. 10.1073/pnas.88.22.10089.Article PubMed Central CAS PubMed Google Scholar, * Venter JC, Adams MD, Sutton GG, Kerlavage AR, Smith HO, Hunkapiller M: Shotgun sequencing of the human genome. Science. 1998, 280: 1540-1542. 10.1126/science.280.5369.1540.Article CAS PubMed Google Scholar, * International Human Genome Sequencing Consortium: Initial sequencing and analysis of the human genome. Nature. 2001, 409: 860-921. 10.1038/35057062.Article Google Scholar, * Venter JC, Adams MD, Myers EW, Li PW, Mural RJ, Sutton GG, Smith HO, Yandell M, Evans CA, Holt RA, Gocayne JD, Amanatides P, Ballew RM, Huson DH, Wortman JR, Zhang Q, Kodira CD, Zheng XH, Chen L, Skupski M, Subramanian G, Thomas PD, Zhang J, Miklos GLG, Nelson C, Broder S, Clark AG, Nadeau J, McKusick VA, Zinder N, et al: The sequence of the human genome. Science. 2001, 291: 1304-1351. 10.1126/science.1058040.Article CAS PubMed Google Scholar, * International Human Genome Sequencing Consortium. http://www.genome.gov/11006939,, * Shendure J, Aiden ER: The expanding scope of DNA sequencing. Nat Biotechnol. 2012, 30: 1084-1094. 10.1038/nbt.2421.Article PubMed Central CAS PubMed Google Scholar, * Hood L: A personal journey of discovery: developing technology and changing biology. Annu Rev Anal Chem. 2008, 1: 1-43. 10.1146/annurev.anchem.1.031207.113113.Article CAS Google Scholar, * Committee on a New Biology for the 21st Century: A New Biology for the 21st Century. 2009, Washington DC: The National Academies Press Google Scholar, * Ideker T, Galitski T, Hood L: A new approach to decoding life: systems biology. Annu Rev Genomics Hum Genet. 2001, 2: 343-372. 10.1146/annurev.genom.2.1.343.Article CAS PubMed Google Scholar, * Encyclopedia of DNA Elements. http://encodeproject.org/ENCODE/,, * ENCODE Project Consortium: An integrated encyclopedia of DNA elements in the human genome. Nature. 2012, 489: 57-74. 10.1038/nature11247.Article Google Scholar, * Editorial: Form and function. Nature. 2013, 495: 141-142., * ENCODE Project Consortium: A user’s guide to the Encyclopedia of DNA Elements (ENCODE). PLoS Biol. 2011, 9: e1001046-10.1371/journal.pbio.1001046.Article Google Scholar, * Aebersold R, Mann M: Mass spectrometry-based proteomics. Nature. 2003, 422: 198-207. 10.1038/nature01511.Article CAS PubMed Google Scholar, * Picotti P, Aebersold R: Selected reaction monitoring-based proteomics: workflows, potential, pitfalls and future directions. Nat Methods. 2012, 9: 555-566. 10.1038/nmeth.2015.Article CAS PubMed Google Scholar, * Desiere F, Deutsch EW, King NL, Nesvizhskii AI, Mallick P, Eng J, Chen S, Eddes J, Loevenich SN, Aebersold R: The PeptideAtlas Project. Nucleic Acids Res. 2006, 34: D655-D658. 10.1093/nar/gkj040.Article PubMed Central CAS PubMed Google Scholar, * Deutsch ED, Mendoza L, Shteynberg D, Farrah T, Lam H, Tasman N, Sun Z, Nilsson E, Pratt B, Prazen B, Eng JK, Martin DB, Nesvizhskii A, Aebersold R: A guided tour of the Trans-Proteomic Pipeline. Proteomics. 2010, 10: 1150-1159. 10.1002/pmic.200900375.Article PubMed Central CAS PubMed Google Scholar, * Genomes Online Database: complete genome projects. http://www.genomesonline.org/cgi-bin/GOLD/index.cgi?page_requested=Complete+Genome+Projects,, * Theobald DL: A formal test of the theory of universal common ancestry. Nature. 2010, 465: 219-222. 10.1038/nature09014.Article CAS PubMed Google Scholar, * Wolfe KE, Li W-H: Molecular evolution meets the genomics evolution. Nat Genet. 2003, Suppl 33: 255-265.Article Google Scholar, * Marques-Bonet T, Ryder OA, Eichler EE: Sequencing primate genomes: what have we learned?. Annu Rev Genomics Hum Genet. 2009, 10: 355-386. 10.1146/annurev.genom.9.081307.164420.Article CAS PubMed Google Scholar, * Noonan JP: Neanderthal genomics and the evolution of modern human. Genome Res. 2010, 20: 547-553. 10.1101/gr.076000.108.Article PubMed Central CAS PubMed Google Scholar, * Stoneking M, Krause J: Learning about human population history from ancient and modern genomes. Nat Rev Genet. 2011, 12: 603-614.Article CAS PubMed Google Scholar, * Sankararaman S, Patterson N, Li H, Paabo S, Reich D: The date of interbreeding between Neanderthals and Modern Humans. PLoS Genet. 2012, 8: e1002947-10.1371/journal.pgen.1002947.Article PubMed Central CAS PubMed Google Scholar, * Schatz MC: Computational thinking in the era of big data biology. Genome Biol. 2012, 13: 177-10.1186/gb-2012-13-11-177.Article PubMed Central PubMed Google Scholar, * Mizrachi I: GenBank: the Nucleotide Sequence Database. The NCBI Handbook. Edited by: McEntyre J, Ostell J. 2002, Bethesda: National Center for Biotechnology Information Google Scholar, * Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, Haussler D: The human genome browser at UCSC. Genome Res. 2002, 12: 996-1006.Article PubMed Central CAS PubMed Google Scholar, * SourceForge. http://sourceforge.net/,, * Bioconductor: open source software for bioinformatics. http://www.bioconductor.org/,, * Field D, Sansone S-A, Collina A, Booth T, Dukes P, Gregurick SK, Kennedy K, Kolar P, Kolker E, Maxon M, Millard S, Mugabushaka M, Perrin N, Remacle JE, Remington K, Rocca-Serra P, Taylor CF, Thorley M, Tiwari B, Wilbanks J: Omics data sharing. Science. 2009, 326: 234-236. 10.1126/science.1180598.Article PubMed Central CAS PubMed Google Scholar, * Knoppers BM, Harris JR, Tasse AM, Budin-Ljosne I, Kaye J, Deschenes M, Zawati M: Towards a data-sharing Code of Conduct for international genomic research. Genome Med. 2011, 3: 46-10.1186/gm262.Article PubMed Central PubMed Google Scholar, * Hood L: Biological complexity under attack: a personal view of systems biology and the coming of “big science”. Genet Eng Biotechnol News. 2011, 31: 17-Article Google Scholar, * Tripp S, Grueber M: Economic Impact of the Human Genome Project. 2011, Columbus: Battelle Memorial Institute Google Scholar, * International HapMap Consortium: A haplotype map of the human genome. Nature. 2005, 437: 1299-1320. 10.1038/nature04226.Article Google Scholar, * The International HapMap3 Consortium: Integrating common and rare genetic variation in diverse human populations. Nature. 2010, 467: 52-58. 10.1038/nature09298.Article Google Scholar, * Abbott A: Neuroscience: solving the brain. Nature. 2013, 499: 272-274. 10.1038/499272a.Article CAS PubMed Google Scholar, * The 1000 Genomes Project Consortium: An integrated map of genetic variation from 1,092 human genomes. Nature. 2012, 491: 56-65. 10.1038/nature11632.Article PubMed Central Google Scholar, * A Catalog of Published Genome-wide Association Studies. http://www.genome.gov/gwastudies/,, * Roach JC, Glusman G, Smit AF, Huff CD, Hubley R, Shannon PT, Rowen L, Pant KP, Goodman N, Bamshad M, Shendure J, Drmanac R, Jorde LB, Hood L, Galas DJ: Analysis of genetic inheritance in a family quartet by whole-genome sequencing. Science. 2010, 328: 636-639. 10.1126/science.1186802.Article PubMed Central CAS PubMed Google Scholar, * Levy S, Sutton G, Ng PC, Feuk L, Halpern AL, Walenz BP, Axelrod N, Huang J, Kirkness EF, Denisov G, Lin Y, MacDonald JR, Pang AW, Shago M, Stockwell TB, Tsiamouri A, Bafna V, Bansal V, Kravitz SA, Busam DA, Beeson KY, McIntosh TC, Remington KA, Abril JF, Gill J, Borman J, Rogers YH, Frazier ME, Scherer SW, Strausberg RL, et al: The diploid genome sequence of an individual human. PLoS Biol. 2007, 5: e254-10.1371/journal.pbio.0050254.Article PubMed Central PubMed Google Scholar, * Wheeler DA, Srinivasian M, Egholm M, Shen Y, Chen L, McGuire A, He W, Chen Y-J, Makhijani V, Roth GT, Gomes X, Tartaro K, Niazi F, Turcotte CL, Irzyk GP, Lupski JR, Chinault C, Song X, Liu Y, Yuan Y, Nazareth L, Qin X, Muzny DM, Margulies M, Weinstock GM, Gibbs RA, Rothberg JM: The complete genome of an individual by massively parallel DNA sequencing. Nature. 2008, 452: 872-876. 10.1038/nature06884.Article CAS PubMed Google Scholar, * International Cancer Genome Consortium. http://icgc.org/,, * The Cancer Genome Atlas. http://cancergenome.nih.gov/,, * Pandey A: Preparing for the 21st century patient. JAMA. 2013, 309: 1471-1472. 10.1001/jama.2012.116971.Article CAS PubMed Google Scholar, * Hood L, Flores M: A personal view on systems medicine and the emergence of proactive P4 medicine: predictive, preventive, personalized and participatory. Nat Biotechnol. 2012, 29: 613-624.CAS Google Scholar, * Price ND, Edelman LB, Lee I, Yoo H, Hwang D, Carlson G, Galas DJ, Heath JR, Hood L: Systems biology and the emergence of systems medicine. Genomic and Personalized Medicine: From Principles to Practice. Volume 1. Edited by: Ginsburg G, Willard H. 2009, Philadelphia: Elsevier, 131-141. Google Scholar, * Green RC, Berg JS, Grody WW, Kalia SS, Korf BR, Martin CL, McGuire A, Nussbaum RL, O’Daniel JM, Ormond KE, Rehm HL, Watson MS, Williams MS, Biesecker LG: ACMG Recommendations for Reporting of Incidental Findings in Clinical Exome and Genome Sequencing. 2013, Bethesda: American College of Medical Genetics and Genomics Google Scholar, * Meyerson M, Gabriel S, Getz G: Advances in understanding cancer genomes through second-generation sequencing. Nat Rev Genet. 2010, 11: 685-696. 10.1038/nrg2841.Article CAS PubMed Google Scholar, * Qin S, Zhou Y, Lok AS, Tsodikov A, Yan X, Gray L, Yuan M, Moritz RL, Galas D, Omenn GS, Hood L: SRM targeted proteomics in search for biomarkers of HCV-induced progression of fibrosis to cirrhosis in HALT-C patients. Proteomics. 2012, 12: 1244-1252. 10.1002/pmic.201100601.Article PubMed Central CAS PubMed Google Scholar, * Li X-J, Hayward C, Fong P-Y, Dominguez M, Hunsucker SW, Lee LW, McClean M, Law S, Butler H, Schirm M, Gingras O, Lamontague J, Allard R, Chelsky D, Price ND, Lam S, Massion PP, Pass H, Rom WN, Vachani A, Fang KC, Hood L, Kearney P: A blood-based proteomic classifier for the molecular characterization of pulmonary nodules. Sci Transl Med. in press, * Knoppers BM, Thorogood A, Chadwick R: The Human Genome Organisation: towards next-generation ethics. Genome Med. 2013, 5: 38-10.1186/gm442.Article PubMed Central PubMed Google Scholar, * Hood L: Who we are: the book of life. Commencement Address. Whitman College Magazine. 2002, 4-7. Google Scholar, * Foster MW, Sharp RR: Beyond race: towards a whole-genome perspective on human populations and genetic variation. Nat Rev Genet. 2004, 5: 790-796. 10.1038/nrg1452.Article CAS PubMed Google Scholar, * Royal CDM, Dunston GM: Changing the paradigm from ‘race’ to human genetic variation. Nat Genet. 2004, 36: S5-S7. 10.1038/ng1454.Article CAS PubMed Google Scholar, * Witherspoon DJ, Wooding S, Rogers AR, Marchani EE, Watkins WS, Batzer MA, Jorde LB: Genetic similarities within and between populations. Genetics. 2007, 176: 351-359. 10.1534/genetics.106.067355.Article PubMed Central CAS PubMed Google Scholar, * Genovese G, Handsaker RE, Li H, Altemose N, Lindgren AM, Chambert K, Pasaniuk B, Price AL, Reich D, Morton CC, Pollak MR, Wilson JG, McCarroll SA: Using population admixture to help complete maps of the human genome. Nat Genet. 2013, 45: 406-414. 10.1038/ng.2565.Article PubMed Central CAS PubMed Google Scholar, * Fernandez-Suarez XM, Galperin MY: The, Nucleic Acids Research Database Issue and the online Molecular Biology Database Collection. Nucleic Acids Res. 2013, 2013: D1-D7.Article Google Scholar, * Human Proteome Project. http://www.hupo.org/research/hpp/,, * Hood LE, Omenn GS, Moritz RL, Aebersold R, Yamamoto KR, Amos M, Hunter-Cevera J, Locascio L, Workshop Participants: New and improved proteomics technologies for understanding complex biological systems: addressing a grand challenge in the life sciences. Proteomics. 2012, 12: 2773-2783. 10.1002/pmic.201270086.Article PubMed Central CAS PubMed Google Scholar, * Editorial: The call of the human proteome. Nat Methods. 2010, 7: 661-, * Schadt E, Turner S, Kasarskis A: A window into third-generation sequencing. Hum Mol Genet. 2010, 19: R227-R240. 10.1093/hmg/ddq416.Article CAS PubMed Google Scholar, * Kim JK, Samaranayake M, Pradhan S: Epigenetic mechanisms in mammals. Cell Mol Life Sci. 2009, 66: 596-612. 10.1007/s00018-008-8432-4.Article PubMed Central CAS PubMed Google Scholar, * Hon G, Ren B, Wang W: ChromaSig: a probabilistic approach to finding common chromatin signatures in the human genome. PLoS Comput Biol. 2008, 4: e1000201-10.1371/journal.pcbi.1000201.Article PubMed Central PubMed Google Scholar, * Hayden EC: Nanopore genome sequencer makes its debut. Nature News. 2012, -10.1038/nature.2012.10051. Google Scholar, Download referencesAcknowledgementsThe authors gratefully acknowledge support from the Luxembourg Centre for Systems Biomedicine and the University of Luxembourg; from the NIH, through award 2P50GM076547-06A; and the US Department of Defense (DOD), through award W911SR-09-C-0062. LH receives support from NIH P01 NS041997; 1U54CA151819-01; and DOD awards W911NF-10-2-0111 and W81XWH-09-1-0107.Author informationAuthors and Affiliations* Institute for Systems Biology, 401 Terry Ave N., Seattle, WA, 98109, USALeroy Hood & Lee Rowen, Corresponding authorsCorrespondence to Leroy Hood or Lee Rowen.Additional informationCompeting interestsThe authors declare that they have no competing interests.About this articleCite this articleHood, L., Rowen, L. The Human Genome Project: big science transforms biology and medicine. Genome Med 5, 79 (2013). https://doi.org/10.1186/gm483Download citation* Published: 13 September 2013, * DOI: https://doi.org/10.1186/gm483, Keywords* Human Genome Project, * Human Genome Sequence, * Human Brain Project, * Small Science, * Individual Genome Sequence,', metadata={'source': 'https://genomemedicine.biomedcentral.com/articles/10.1186/gm483', 'page_title': 'The Human Genome Project: big science transforms biology and medicine'}),\n",
       " Document(page_content=\"1. Brief History of the Human Genome Project HGP at the start The HGP began officially in October 1990, but its origins go back earlier. In the mid-1980s, three scientists independently came up with the idea of sequencing the entire human genome: Robert Sinsheimer, then chancellor of University of California at Santa Cruz, as a way to spend $30 million donated to his institution to build a telescope when that project fell through; Salk Institute researcher Rene Dulbecco as a way to understand the genetic origins of cancer and other diseases; and the Department of Energy's (DOE's) Charles DeLisi as a way to detect radiation-induced mutations, an interest of that agency since the atomic bombings of Hiroshima and Nagasaki. Such a project had become technically feasible due to advances made during the previous decade or two: in the early 1970s, recombinant DNA technologies (use of restriction enzymes to splice DNA, reverse transcriptase to make DNA from RNA, viral vectors to carry bits of DNA into cells, bacterial cloning to multiply quantities of DNA); in the late 1970s, DNA sequencing and use of RFLP (restriction fragment length polymorphism) markers for gene mapping; and in the early to mid-1980s, DNA synthesis, pulsed-field gel electrophoresis, polymerase chain reaction (PCR), and automated DNA sequencing. Sinsheimer's, Dulbecco's, and DeLisi's idea found supporters among a number of prominent molecular biologists and human geneticists—for example, Walter Bodmer, Walter Gilbert, Leroy Hood, Victor McKusick, and James D. Watson. However, many molecular biologists expressed misgivings. Especially through 1986 and 1987, there were concerns about the routine nature of sequencing and the amount of “junk DNA” that would be sequenced, that the expense and big science approach would drain resources from smaller and more worthy projects, and that knowledge of gene sequence was inadequate to yield knowledge of gene function.[1] In September 1986, committees were established to study the feasibility of a publicly-funded project to sequence the human genome: one by the National Research Council (NRC) on scientific merit, and one by the Office for Technology Assessment (OTA) as a matter of public policy. Both committees released reports in 1988. The OTA report, Mapping Our Genes: Genome Projects: How Big, How Fast? downplayed the concerns of scientist critics by emphasizing that there was not one but many genome projects, that these were not on the scale of the Manhattan or Apollo projects, that no agency was committed to massive sequencing, and that the study of other organisms was needed to understand human genes. The NRC report, Mapping and Sequencing the Human Genome, sought to accommodate the scientists’ concerns by formulating recommendations that genetic and physical mapping and the development of cheaper, more efficient sequencing technologies precede large-scale sequencing, and that funding be provided for the mapping and sequencing of nonhuman (“model”) organisms as well. It was the DOE that made the first push toward a “Big Science” genome project: DeLisi advanced a five-year plan in 1986, $4.5 million was allocated from the 1987 budget, and recognizing the boost the endeavor would provide to national weapons laboratories, Senator Pete Domenici from New Mexico introduced a bill in Congress. The DOE undertaking produced consternation among biomedical researchers who were traditionally supported by the NIH's intramural and extramural programs—for example, Caltech's David Botstein referred to the initiative as “DOE's program for unemployed bomb-makers” (in Cook-Deegan 1994, p. 98). James Wyngaarden, head of the NIH, was persuaded to lend his agency's support to the project in 1987. Funding was in place in time for fiscal year (FY) 1988 with Congress awarding the DOE $10.7 million and the NIH $17.2 million.[2] The DOE and NIH coordinated their efforts with a Memorandum of Understanding in 1988 that agreed on an official launch of the HGP on October 1, 1990 and an expected date of completion of 2005. Total cost estimated by the NRC report was $3 billion. The project's specific goals at the outset were: (i) to identify all genes of the human genome (initially estimated to be 100,000); (ii) to sequence the approximately 3 billion nucleotides of the human genome; (iii) to develop databases to store this information; (iv) to develop tools for data analysis; (v) to address ethical, legal, and social issues; and (vi) to sequence a number of “model organisms,” including the bacterium Escherichia coli, the yeast Saccharomyces cerevisiae, the roundworm Caenorhabditis elegans, the fruitfly Drosophila melanogaster, and the mouse Mus musculans. The DOE established three genome centers in 1988–89 at Lawrence Berkeley, Lawrence Livermore, and Los Alamos National Laboratories; as Associate Director of the DOE Office of Health and Environmental Research (OHER), David Galas oversaw the DOE's genome project from April 1990 until he left for the private sector in 1993. The NIH instituted a university grant-based program for human genome research and placed Watson, co-discoverer of the structure of DNA and director of Cold Spring Harbor Laboratory, in charge in 1988. In October 1989, the Department of Health and Human Services established the National Center for Human Genome Research (NCHGR) at the NIH with Watson at the helm. During 1990 and 1991, Watson expanded the grants-based program to fund seven genome centers for five-year periods to work on large-scale mapping projects: Washington University, St. Louis; University of California, San Francisco; Massachusetts Institute of Technology; University of Michigan; University of Utah; Baylor College of Medicine; and Children's Hospital of Philadelphia. As the HGP got underway, a number of philosophers weighed in on its scientific merit—in terms of cost, potential impact on other areas of research, ability to lead to medical cures, and the usefulness of sequence data (Kitcher 1995; Rosenberg 1995; Tauber and Sarkar 1992; Vicedo 1992). However, of particular interest to philosophers is goal (v) concerning ethical, legal, and social issues. At an October 1988 news conference called to announce his appointment, Watson, in an apparently off-the-cuff response to a reporter who asked about the social implications of the project, promised that a portion of the funding would be set aside to study such issues (Marshall 1996c). The result was the NIH/DOE Joint Working Group on Ethical, Legal, and Social Implications (ELSI) of Human Genome Research, chaired by Nancy Wexler, which began to meet in September 1989.[3] The Joint Working Group identified four areas of high priority: “quality and access in the use of genetic tests; fair use of genetic information by employers and insurers; privacy and confidentiality of genetic information; and public and professional education” (Wexler in Cooper 1994, p. 321). The NIH and DOE each established ELSI programs: philosopher Eric T. Juengst served as the first director of the NIH-NCHGR ELSI program from 1990 to 1994. ELSI was funded initially to the tune of three percent of the HGP budget for both agencies; this was increased to four and later five percent at the NIH. Map first, sequence later As the NRC report had recommended, priority at the outset of the project was given to mapping rather than sequencing the human genome. HGP scientists sought to construct two kinds of maps. Genetic maps order polymorphic markers linearly on chromosomes; the aim is to have these markers densely enough situated that linkage relations can be used to locate chromosomal regions containing genes of interest to researchers. Physical maps order collections (or “libraries”) of cloned DNA fragments that cover an organism's genome; these fragments can then be replicated in quantity for sequencing. The joint NIH-DOE five-year plan released in 1990 set specific benchmarks: a resolution of 2 to 5 centimorgans (cM) for genetic linkage maps and physical maps with sequence-tagged site (STS) markers (unique DNA sequences 100–200 base pairs long) spaced approximately 100 kilobases (kb) apart and 2-megabase (Mb) contiguous overlapping clones (“contigs”) assembled for large sections of the genome. Sequencing needed to be made more efficient and less costly: aims were to reduce sequencing costs to $.50 per base and to complete 10 million bases of contiguous DNA (0.3 percent of the human genome) but otherwise to focus efforts on the smaller genomes of less complex model organisms (Watson 1990). HGP goals were facilitated by a number of technological developments during this initial period. For physical mapping, yeast artificial chromosomes (YACs) introduced in 1987 (Burke et al. 1987) permitted much larger segments of DNA to be ordered and stored for sequencing than was possible with plasmid or cosmid libraries. A new class of genetic markers, microsatellite repeats, was identified in 1989 (Litt and Luty 1989; Tautz 1989; Weber and May 1989); because these sets of tandem repeats of short (either dinucleotide, trinucleotide, or tetranucleotide) DNA sequences are more highly polymorphic and detectable by PCR, microsatellites quickly replaced RFLPs as markers of choice for genetic linkage mapping and furnished the STS markers which facilitated the integration of genetic and physical maps. Another technological achievement—the combined use of reverse transcription, PCR, and automated sequencing to map expressed genes—led to administrative changes at the NIH when, in April 1992, Watson resigned from his position as director of the NCHGR following a conflict with NIH director Bernadine Healy over gene patenting. In 1991, while working at the NIH, J. Craig Venter sequenced small portions of cDNAs from existing libraries to provide identifying expressed sequence tags (ESTs) of 200–300 bases which he then compared to already identified genes from various species found in existing databases (Adams et al. 1991).[4] Watson disagreed with Healy's decision to approve patent applications for the ESTs despite lack of knowledge of their function.[5] Soon after Watson's departure, Venter left NIH for the private sector.[6] Francis Collins, an MD-PhD whose lab at University of Michigan co-discovered genes associated with cystic fibrosis and neurofibromatosis and contributed to efforts to isolate the gene for Huntington's disease, was appointed by Healy as Watson's replacement, and he began at the NCHGR in April 1993. Collins established an intramural research program at the NCHGR to complement the extramural program of grants for university-based research which already existed; ELSI remained a grant-funded program. The original NIH-DOE five-year plan was updated in 1993. The new five-year plan, in effect through 1998, accommodated progress that had been made in mapping, sequencing, and technological development (Collins and Galas 1993). The goal of a 2–5 cM genetic map was expected to be met by the 1995 target date. The deadline for a physical map with STS markers at intervals of 100 kb was extended to 1998; a map with intervals averaging 300 kb was expected by 1995 or 1996. Although the goal of $.50 per base cost of sequencing was projected to be met by 1996, it was recognized that this would be insufficient to meet the 2005 target date. The updated goal was to build up to a collective sequencing capacity of 50 Mb per year and to have 80 Mb of DNA (from both human and model organism genomes) sequenced by the end of 1998. This would be achieved by increasing the number of groups working on large-scale sequencing and heightening efforts to develop new sequencing technologies. Accordingly, in November 1995, the U.K.'s Wellcome Trust launched a $75 million, seven-year concentrated sequencing effort at the Sanger Centre in Cambridge, and in April 1996, the NCHGR awarded grants totaling $20 million per year for six centers (Houston's Baylor College of Medicine, Stanford University, The Institute for Genomic Research [TIGR], University of Washington-Seattle, Washington University School of Medicine in St. Louis, and Whitehead Institute for Biomedical Research—MIT Genome Center) to pilot high-volume sequencing approaches (Marshall 1996a). Although the HGP's inceptions were in the U.S., it had not taken long for mapping and sequencing the human genome to become an international venture (see Cook-Deegan 1994). France began to fund genome research in 1988 and had developed a more centralized, although not very well-funded, program by 1990. More significant were the contributions of Centre d’Etudes du Polymorphisme Humain (CEPH) and Généthon. CEPH, founded in 1983 by Jean Dausset, maintained a collection of DNA donated by intergenerational families to help in the study of hereditary disease; Jean Weissenbach led an international effort to construct a complete genetic map of the human genome using the CEPH collection; later, with funding from the French muscular dystrophy association (AFM), director Daniel Cohen set out to construct a YAC clone library for physical mapping and oversaw the launching of Généthon in 1991 as an industrial-sized mapping and sequencing operation funded by the AFM. The U.K.'s genome project received its official start in 1989 although Sydney Brenner had commenced genome research at the Medical Research Council (MRC) laboratory several years before this. MRC funding was supplemented with private monies from the Imperial Cancer Research Fund, and later, the Wellcome Trust. The Sanger Centre, led by John Sulston and funded by Wellcome and the MRC, opened in October 1993. A combined four-year, 15-million-euro genome program by the European Community (E.C.) commenced in 1990. Germany, its citizens all too aware of abuses in the name of genetics, lagged behind other European countries: although individual researchers received government funds for genome research in the late-1980s and participated in the E.C. initiative, no actual national genome project was undertaken until 1995 (Kahn 1996). Japan, ahead of the U.S. in having funded the development of automated sequencing technologies since the early 1980s, was the major genome player outside the U.S. and Europe with several government agencies beginning small-scale genome projects in the late-1980s and early- 1990s, but a frequent target of U.S. criticism for the size of its investment relative to GNP.[7] China was the latecomer on the international scene: with 250 million yuan ($30 million) over three years from government and industry, the Chinese National Human Genome Center with branches in Beijing and Shanghai opened in July 1998, and was followed in 1999 by the Beijing Genomics Institute.[8] As 1998, the last year of the revised five-year plan and midpoint of the project's projected 15-year span, approached, many mapping goals had been met. In 1994, Généthon completed a genetic map with more than 2000 microsatellite markers at an average spacing of 2.9 cM and only one gap larger than 20 cM (Gyapay et al. 1994), though the genetic mapping phase of the project did not finally come to a close until March 1996 with publication of comprehensive genetic maps of the mouse and human genomes in Nature: the mouse map produced by scientists at Whitehead-MIT Center for Genome Research contained 7,377 genetic markers (both microsatellites and RFLPs) with an average spacing of 0.2 cM (Dietrich et al. 1996); the human map produced by scientists at Généthon contained 5,264 microsatellite markers located to 2335 positions with an average spacing of 1.6 cM (Dib et al. 1996). Physical mapping was on track: in 1995, a physical map with 94 percent coverage of the genome and STS markers at average intervals of 199 kb was published (T. Hudson et al. 1995), as was CEPH's updated physical map of 225 YAC contigs covering 75 percent of the genome (Chumakov et al. 1995); however, bacterial artificial chromosomes (BACs), developed in DOE-funded research at Caltech in 1992 (Shizuya et al. 1992), soon replaced YACs because of their greater stability in propagating DNA for sequencing. Sequencing itself presented more of a challenge. The genomes of the smallest model organisms had been sequenced. In April 1996, an international consortium of mostly European laboratories published the sequence for S. cerevisiae which was the first eukaryote completed, with 12 million base pairs and 5,885 genes and at a cost of $40 million (Goffeau et al. 1996). In January 1997, University of Wisconsin researchers completed the sequence of E. coli with 4,638,858 base pairs and 4,286 genes (Blattner et al. 1997). However, despite ramped-up sequencing efforts over the past several years at the Sanger Centre and NHGRI-funded centers (the NCHGR had been elevated to the status of a research institute in 1997 and renamed the National Human Genome Research Institute), with only three percent of the human genome sequenced, sequencing costs hovering at $.40/base, and the desired high-output not yet achieved by the sequencing centers, and about $1.8 billion spent, doubts existed about whether the target date of 2005 could be met. Suddenly, the HGP found itself challenged by sequencing plans from the private sector. In May 1998, TIGR's Venter announced he would partner with Michael Hunkapiller's company Applied Biosystems (ABI), a division of Perkin-Elmer Corporation which manufactured sequencing machines, to form a new company which would sequence the entire genome in three short years and for a fraction of the cost. The foreseen profits rested in the construction of a “definitive” database that would outdo Genbank by integrating medical and other information with the basic sequence and polymorphisms. The company, based in Rockville, MD and later named Celera Genomics, planned to use “whole-genome shotgun” (WGS) sequencing, an approach different from the HGP's. The HGP confined the shotgun method to cloned fragments already mapped to specific chromosomal regions: these are broken down into smaller bits then amplified by bacterial clones, sequences are generated randomly by automated machines, and computational resources are used to reassemble sequence using overlapping areas of bits. Shotgunning is followed by painstaking “finishing” to fill in gaps, correct mistakes, and resolve ambiguities. What Celera was proposing for the shotgun method was to break the organism's entire genome into millions of pieces of DNA with high-frequency sound waves, sequence these pieces using hundreds of ABI's new capillary model machines, and reassemble the sequences with one of the world's largest civilian supercomputers without the assistance provided by the preliminary mapping of clones to chromosomes. When WGS sequencing was considered as a possibility by the HGP, it was rejected because of the risk that repeat sequences would yield mistakes in reassembly.[9] But Venter by this time had successfully used the technique to sequence the 1.83 million nucleotide bases of the bacterium Hemophilus influenzae—the first free-living organism to be completely sequenced—in a year's time (Fleischmann et al. 1995).[10] Race to the genome The race to sequence the genome was on. The publicly-funded scientists downplayed the media image of a race often over the next couple of years, but they were certainly propelled by worries that funding would dry up before the sequence was complete given private sector willingness to take over and that the sequence data would become proprietary information—the Bermuda Accord, agreed to in February 1996 by the world's major sequencing laboratories which at the time included Venter's TIGR, required the public release of sequence data every 24 hours. Wellcome more than doubled its funds to the Sanger Centre (to £205 million) and the center changed its goal from sequencing one-sixth of the genome to sequencing one-third, and possibly one-half (Dickson 1998). The NHGRI and DOE published a new five-year plan for 1998-2003 (Collins et al. 1998). The plan moved the final completion date forward from 2005 to 2003 and aimed for a “working draft” of the human genome sequence to be completed by December 2001. This would be achieved by delaying the finishing process, no longer going clone-by-clone to shotgun, reassemble, and finish the sequence of one clone before proceeding to the next. A physical map of 41,664 STS markers was soon published (Deloukas et al. 1998), and so the physical mapping goal was met, but with only six percent of the human genome sequence completed, the plan called for new and improved sequencing technologies which could increase the sequencing capacity from 90 Mb per year at about $.50 per base to 500 Mb per year at no more than $.25 per base. Goals for completing the sequencing of the remaining model organisms were also set: December 1998 for C. elegans which was 80 percent complete, 2002 for D. melanogaster which was nine percent complete, and 2005 for M. musculus which was still at the physical mapping stage. An interim victory for the publicly-funded project followed when, on schedule, the first animal sequence, that of C. elegans with 97 million bases and 19,099 genes, was published in Science in December 1998 (C. elegans Sequencing Consortium 1998). This was the product of a 10-year collaboration between scientists at Washington University (headed by Bob Waterston) and the Sanger Centre (headed by John Sulston), carried out at a semi-industrial scale with more than 200 people employed in each lab working around the clock. In March 1999, the main players—the NHGRI, Sanger Centre, and DOE—advanced the date of completion of the “working draft”: five-fold coverage of at least 90 percent of the genome was to be completed by the following spring (Pennisi 1999; Wadman 1999). This change reflected improved output of the new model of automated sequencing machines, diminished sequencing costs at $.20 to $.30 per base, and the desire to speed up the release of medically relevant data. NHGRI would take responsibility for 60 percent of the sequence, concentrating these efforts at only three centers with Baylor, Washington University, and Whitehead-MIT sharing $81.6 million over the ensuing 10 months; 33 percent of the sequence would be the responsibility of the Sanger Centre whose funds from Wellcome increased from $57 million to $77 million for the year; and the remaining sequence would be supplied by the DOE's Joint Genome Institute (JGI) in Walnut Creek, CA into which its three centers had merged in January 1997. The smaller international centers involved in sequencing were not consulted on this restructuring, but were later brought on board on the condition that they could keep up with the pace. The first chromosomes to be completed (and this was to finished, not working draft, standards) were the two smallest: the sequence for chromosome 22 was published by scientists at the Sanger Centre and partners at University of Oklahoma, Washington University in St. Louis and Keio University in Japan in December 1999 (Dunham et al. 1999); the sequence for chromosome 21 was published by an international consortium of mostly Japanese and German labs—half at RIKEN—in May 2000 (Hattori et al. 2000). The remaining chromosomes lagged behind, though the DOE announced completion of working drafts of chromosomes 5, 16, and 19 with three-fold coverage in April 2000. The progress made by the publicly-funded project could be monitored because sequence data were released at 24-hour intervals, but Celera's progress was more difficult to assess. HGP scientist Maynard Olson charged that Celera was doing “science by press conference” (in Davies 2002, p. 153). Certainly, Celera's press conferences gave the impression it was ahead in the race: on 10 January 2000 the company announced completion of 90 percent of the human genome sequence, and on 6 April 2000 the company announced completion of three-fold coverage of the DNA of one male donor. But there was also evidence that Celera did remain a threat: the validity of the WGS sequencing approach was demonstrated in March 2000 when Celera and the (publicly-funded) Berkeley Drosophila Genome Project published the sequence of D. melanogaster of about 180 Mb (Adams et al. 2000). In June 2000, the contest ended in what appeared to be a tie for the prize, but was more an arranged truce. On 26 June 2000, Collins, Venter, and the DOE's Ari Patrinos joined U.S. President Bill Clinton (and British Prime Minister Tony Blair by satellite link) at a White House press conference to announce that the human genome had been sequenced. That Collins and Venter even shared the limelight on that day was itself a tremendous feat. The negotiated draw at the finish line permitted HGP scientists to save face and their upstart competitor to minimize the risk of alienating university-based researchers and losing their business. The agreement between parties included eventual simultaneous publication of their results. However, not only had results not yet been readied for publication, neither of the two sequence maps was complete (Pennisi 2000). The HGP had not met its previous year's goal of a working draft covering 90 percent of the genome: Collins reported that ordered BACs existed for 97 percent of the genome and that BACs for 85 percent of the genome had been sequenced, with 24 percent of the genome sequence in finished form, 22 percent of the genome sequence in near-finished form, and 38 percent of the genome sequence in provisional form. Assisted by its researchers’ access to HGP data stored on public databases, Celera's efforts were accepted as being further along: the company's press release that day announced completion of the “first assembly” of the human genome with 99 percent coverage. An editorial in Nature described the fanfare of 26 June as an “extravagant” example—one reaching “an all-out zenith or nadir, according to taste”—of scientists making public announcements not linked to peer-reviewed publication, here to bolster share prices (Celera) and for political effect (the HGP) given the “months to go before even a draft sequence will be scientifically useful” (Anonymous 2000, p. 981). The peer-reviewed publications came almost eight months later. Plans for joint publication in Science broke down when terms of agreement over data release could not be negotiated: Science's editors were willing to publish Celera's findings without Venter meeting the standard requirement that the sequence data be submitted to GenBank; Celera would instead make the data available on its own website. Press conferences in London and Washington, D.C. on 12 February preceded publications that week—by HGP scientists in Nature on 15 February 2001 and by Venter's team in Science on 16 February 2001. The HGP draft genome sequence, prepared based on map and sequence data available on 8 October 2000, covered about 94 percent of the genome, with about 25 percent in the finished form already attained for chromosomes 21 and 22. Indeed, the authors themselves described it as “an incomplete, intermediate product” which “contains many gaps and errors” (International Human Genome Sequencing Consortium 2001, p. 871). The results published by Celera, based on assemblies completed on 1 October 2001 using two different computational methods, had 84–90 percent of the genome covered by scaffolds at least 100 kb in length, with the composition of the scaffolds averaging 91–92 percent sequence and 8–9 percent gaps, leaving 93,857–105,264 gaps in total (Venter et al. 2001). In the end, Celera's published genome assembly made significant use of the HGP's publicly available map and sequence data, which left open the question whether WGS sequencing alone would have worked.[11] Since the gaps in the sequence were unlikely to contain genes, and only genes as functional segments of DNA have potential commercial value, Celera was happy to move on and leave these gaps for the HGP scientists to fill in. Celera was faced with deciding what sort of company it would be: sequences from three different mouse strains were added to help attract subscribers to its database, and a brief foray was made into proteomics, but Venter resigned as CEO in January 2002 with the company's decision to focus on drug discovery rather than information (Davies 2002). Despite being timed to coincide with celebrations of the 50th anniversary of the Watson-Crick discovery of the double-helical structure of DNA, there was less fanfare surrounding the official date of completion of the HGP in April 2003, two years earlier than had been anticipated at the time of its official launch in October 1990, and several months earlier than called for in the most recent five-year plan. Americans had terrorism and war on their minds. In the end, sequencing—the third phase of the publicly-funded project—was carried out at 16 centers in six countries by divvying up among them sections of chromosomes for sequencing. 85 percent of the sequencing, however, was done at the five major sequencing centers (Baylor, Washington University, Whitehead-MIT, Sanger Center, and DOE's JGI), with the Sanger Centre responsible for nearly one-third. The cost was lower than anticipated, with $2.7 billion spent by U.S. agencies and £150 million spent by Wellcome Trust. The “finished” reference DNA sequence for Homo sapiens—all 3.1 billion nucleotide bases—is publicly accessible on the Internet (NCBI Human Genome Resources). If the As, Ts, Cs, and Gs of the genome sequence were printed in standard type, they would fill 75,490 pages of the New York Times (Wade 2003).[12] In the project's early years, Norton Zinder, who chaired the NIH's Program Advisory Committee for the Human Genome, characterized it in this way: “This Project is creating an infrastructure for doing science; it's not the doing of the science per se. it will provide the biological community with the basic materials for doing research in human biology” (in Cooper 1994, p. 74). The published human genome reference sequences are part of that infrastructure, serving as tools for investigating human genetic variation. So far gene identification has been successful for the single genes of large effect implicated in rare Mendelian disorders. Difficulties arise for identifying the multiple genes of variable effect that interact with nongenetic factors in more common, complex conditions and for understanding the physiological processes associated with the development of these phenotypes. One approach to overcoming these difficulties focuses on relatively genetically homogeneous populations with members for whom extensive clinical data are available, as in the case of the Icelandic genome project, and basically extends the methods used for linkage mapping for diseases within families. Another approach is to conduct large-scale case-control association studies between phenotypes of interest and genetic markers. For both these approaches, single nucleotide polymorphisms (SNPs) are preferred as markers over the microsatellites used for genetic and physical mapping by the HGP. Worried about the private sector's efforts to patent SNPs, which would make them costly to use for research, the NHGRI-DOE's five-year plan for 1998–2003 included the goal of mapping 100,000 SNPs by 2003 (Collins et al. 1998). The development of a public database of SNPs received a $138 million push from the International HapMap Project, a three-year public-private partnership completed in 2005 that mapped variation in four population groups. The NHGRI's involvement in the HapMap Project was part of the continuing leadership role in genome research it envisioned for itself upon completion of the HGP (Collins et al. 2003). Other projects include ENCODE, which began as a pilot project to study gene function by analyzing one percent of the genome and is now looking at the remaining 99 percent, and more recently, clinENCODE, in which disease risk is being calculated for 400 people based on the corresponding one percent of the genome as a step toward personalized medicine. However, the infrastructure of mapping and sequencing technologies developed as part of the HGP—especially the ability to sequence entire genomes of organisms—has changed the way biology, not just human biology, is done. It is now recognized that genome structure by itself tells us only so much. In functional genomics, the interest is in how genomes—not just individual genes anymore—function. By studying the coordinated expression of the genome's various segments in different tissues at different times, researchers are coming to better understand organismal development. In comparative genomics, the study of genomic structure and function in different species is bringing about similar gains in understanding evolution. And genomics is now complemented by the field of proteomics which studies the structure and function of all of an organism's proteins, called the proteome. 2. Philosophy and the Human Genome Project At the June 2000 White House press conference, President Clinton compared the feat of mapping and sequencing the human genome to the mapping of the Northwest Passage by early-nineteenth century explorers Meriwether Lewis and William Clark: Nearly two centuries ago, in this room, on this floor, Thomas Jefferson and a trusted aide spread out a magnificent map, a map Jefferson had long prayed he would get to see in his lifetime. The aide was Meriwether Lewis and the map was the product of his courageous expedition across the American frontier, all the way to the Pacific. It was a map that defined the contours and forever expanded the frontiers of our continent and our imagination. Today the world is joining us here in the East Room to behold the map of even greater significance. We are here to celebrate the completion of the first survey of the entire human genome. Without a doubt, this is the most important, most wondrous map ever produced by humankind. Clinton continued on to say that he considered this “epoch-making triumph of science and reason” to be merely a starting point. Three “majestic horizons” lay immediately ahead: by 2003, production of a final version of the sequence map that would be complete and accurate; biotechnological development in the private sector based on the identification of all human genes and their functions; and ethical respect for “our oldest and most cherished human values” to ensure that genome science benefits “all citizens of the world,” protects privacy, and prevents discrimination (White House 2000). Clinton's comparison of the human genome sequence map to Lewis and Clark's map of the Northwest Passage is perhaps less gratuitous than it might appear. Some members of the 1804–1806 expedition, the “Corps of Discovery,” sought to obtain natural scientific and anthropological knowledge over the course of their travels. The HGP shares the Enlightenment ideals of this period, especially the faith in scientific progress, the goal of systematic knowledge, and the confidence that universal benefits for humanity would ensue from the scientific pursuit of truth. Scientist Leroy Hood expressed the belief that “we will learn more about human development and pathology in the next twenty-five years than we have in the past two thousand” (1992, p. 163). He predicted that the HGP would facilitate movement from a reactive to preventive mode of medicine which would “enable most individuals to live a normal, healthy, and intellectually alert life without disease” (p. 158). The Lewis and Clark journey was an important symbol for encouraging Americans to move westward (the frontier was declared gone by 1890); similarly, “getting” the genome was represented by HGP proponents as a “frontier” of knowledge that, like the moon landing, needed to be conquered. But most important are the colonialist and economic aims associated with this early nineteenth-century “voyage of discovery.” Jefferson sought to establish a U.S. presence beyond its borders, in lands long inhabited by peoples indigenous to the Americas and to which Spain had already staked its claim. He made clear to Lewis that the principal aim of the journey was commercial: “The Object of your mission is to explore the Missouri river & such principal stream of it as by it's course and communication with the waters of the Pacific ocean, whether the Columbia, Oregon, Colorado or any other river may offer the most direct & practicable water communication across this continent for the purpose of commerce” (Discovering Lewis & Clark). The infrastructure to be developed with the HGP was similarly presented as an opportunity to “secure the leadership of the United States in biotechnology and present U.S. industry with a wealth of opportunities” (Hood 1992, p. 163). Legislative changes were enacted in the 1980s to encourage the commercial development of federally funded research: universities and other nonprofit institutions were allowed to apply for patents on such research and tax incentives were provided to the private sector to encourage investment. Although Lewis and Clark depended extensively throughout their journey on the assistance of Indians and French traders they encountered, they regarded the lands they covered as “virgin territory” that awaited the arrival of “civilized men” to be named and claimed. Similar attitudes are implicated in controversies over the commercialization of genomics research and intellectual property and patent rights: organizations representing indigenous peoples charge that the patenting of human genes and cell lines is a continuation of the “bioprospecting” and “biopiracy” carried out by multinational corporations in securing patents on medicinal and food uses of plants which have been long a part of traditional knowledge (Shiva 1996). In the early years of the HGP, the DOE's David Galas expressed skepticism that ELSI-sorts of concerns were anything new: “there are no new problems. Issues concerning privacy, confidentiality, and discrimination will become much more pressing once the Genome Project generates the tools to diagnose diseases presymptomatically. The basic problems, however, are not new—they will simply be exacerbated” (in Cooper 1994, p. 167). Although legal scholar George Annas agreed there were no new problems, he argued that the combination and degree of problems involved did make the HGP unique: “there are probably no unique issues raised by the Human Genome Initiative. On the other hand, this project raises all of the issues in a much more focused manner (certainly a difference in degree if not in kind), and the fact that all of these issues are implicated in the project may itself make the project societally unique” (1990, p. 640). Many of the issues are of interest to philosophers: these include conceptual questions pertaining to scientific knowledge itself and the ethical ramifications of such knowledge and related technological developments. Philosophers of science, ethicists, political theorists and philosophers working in other areas have benefited from ELSI-related funding. There is now a vast literature on human genome-related topics, and this entry can do no more than provide a synopsis regarding what questions have been asked, what range of responses has been offered, and what remains for philosophical attention and debate. 2.1 Conceptual Foundations of the Human Genome Project 2.1.1. Gene concept Bets placed during the HGP over how many genes would be discovered, as well as surprise expressed when far fewer than the original estimate were found (about 25,000–30,000 rather than 100,000—the rice genome apparently has more genes!) (Normile and Pennisi 2002; Pennisi 2003), suggest that “gene”—a term introduced by Wilhelm Johannsen in 1909—names a well-defined concept. The report that because of alternate splicing each gene is responsible for three or four proteins makes the same assumption. As does drawing distinctions between normal and abnormal genes, or seeking to isolate disease genes. The assumption is not very well substantiated, however. Philosophers of biology recognize that the genes of classical genetics, molecular genetics, evolutionary genetics, and more recently developmental genetics do not necessarily map onto each other.[13] Difficulties arriving at a definitive gene concept arise even when we confine ourselves to contemporary molecular biology. Evelyn Fox Keller (2000) points out an irony which has ensued from the HGP's successes: even though gene-talk is more pervasive than ever in the popular and scientific presses, the concept of the gene, whether defined structurally or functionally, has been “radically undermined” (p. 5). Keller provides this description of current laboratory practices: “As we listen to the ways in which the term is now used by working biologists, we find that the gene has become many things—no longer a single entity but a word with great plasticity, defined only by the specific experimental context in which it is used” (p. 69). Recent philosophical efforts to define genes have sought to capture these practices. C. Kenneth Waters (1994) recognizes that specific research contexts determine whether genes are considered to include introns as well as exons, or regulatory or promoter regions as well as open reading frames (ORFs), but argues that what remains “fundamental” across these contexts is the concept of a gene as a stretch of DNA the linear sequence of which provides a template for a gene product, whether mRNA transcript or polypeptide. Because of problems posed for Waters’ account by mRNA splicing and editing, Eva Neumann-Held (1999) recommends replacing the “classical molecular gene concept” of a stretch of DNA coding for a single polypeptide with a “molecular process gene concept” which includes not just the relevant stretches of DNA but the entire cellular context in which polypeptides are produced. Lenny Moss (2003) identifies two gene concepts: the preformationist gene, Gene-P, defined by its relationship to a phenotype, is of instrumental utility for molecular geneticists—for example, the BRCA1 gene is used to predict breast cancer risk; the epigenesist gene, Gene-D, defined by its molecular sequence, serves as a “developmental resource” in providing a template for RNA and protein synthesis but is indeterminate with respect to phenotype since this depends on other developmental resources and the cellular and extracellular contexts.[14] Paul Griffiths and Karola Stotz (2006) distinguish three gene concepts: “instrumental genes” remain important in molecular genetics when relationships between genotype and phenotype are under investigation; “nominal molecular genes” are specific DNA sequences annotated by researchers as genes for structural reasons such as presence of ORFs; “postgenomic molecular genes” are not defined by structure but “by the way DNA sequences are used in particular cellular and broader contexts” (p. 515). Given this context-dependence in what genes are considered to be and do, it seems that pluralism has become the order of the day, for genes as for species. Along these lines, John Dupré (2004) advocates “an atheoretical pluralism” that abandons any pretence to a “theoretical core to the concept”: simply, “a gene is any bit of DNA that anyone has reason to name and keep track of” (pp. 332–333).[15] Keller (2000) agrees that the theoretical importance of genes has faded; she writes: “it seems evident that the primacy of the gene as the core explanatory concept of biological structure and function is more a feature of the twentieth century than it will be of the twenty-first” (p. 9). She forecasts the emergence of new language; this is a situation for which Philip Kitcher believed molecular biology was ripe even 15 years ago when he wrote: “it is hard to see what would be lost by dropping talk of genes from molecular biology and simply discussing the properties of various interesting regions of nucleic acid” (1992, p. 130). Keller believes that gene-talk has served a purpose though, providing a flexibility which permits communication across those specific experimental practices within which “gene” attains precision. Hans-Jörg Rheinberger (2000) takes this argument one step further: gene concepts are not merely useful in spite of their ambiguity, they are useful in virtue of their ambiguity because, as “tools of research, they must reach out into the realm of what we do not yet know” (p. 223). He reminds us that this is nothing new: “The spectacular rise of molecular biology has come about without a comprehensive, exact, and rigid definition of what a gene is” (p. 222). Keller's and Rheinberger's views present an evident challenge to philosophical intuitions that scientific practice is furthered by arriving at precise definitions of basic concepts. 2.1.2 Identity and difference Early in the debates surrounding plans for the HGP, questions arose concerning what it means to map and sequence the human genome—“get the genome,” as Watson (1992) put it. About these concerns, McKusick (1989) wrote: “The question often asked, especially by journalists, is ‘Whose genome will be sequenced?’ The answer is that it need not, and surely will not, be the genome of any one person. Keeping track of the origin of the DNA that is studied will be important, but the DNA can come from different persons chosen for study for particular parts of the genome” (p. 913). The HGP and Celera reference sequences are indeed composites based on chromosomal segments that originate from different individuals: the sequence in any given region of the genome belongs to a single individual, but sequences in different regions of the genome belong to different individuals. However, in both cases, the majority of the sequence originates from just one person. As HGP sequencing efforts accelerated, concerns arose that only four genomes, a couple of which belonged to known laboratory personnel, were being used for physical mapping and sequencing (Marshall 1996b). The decision was made to construct 10 new clone libraries for sequencing with each library contributing about 10 percent of the total DNA. In the end, 74.3 percent of the total number of bases sequenced was derived from a single clone library—that of a male, presumably from the Buffalo area; seven other clone libraries contributed to an additional 17.3 percent of the sequence (International Human Genome Sequencing Consortium 2001, p. 866). A similar proportion—close to 71 percent—of the Celera sequence belongs to just one male even though five ethnically diverse donors were selected; incredibly enough, rumors have been confirmed that this individual is Venter himself (McKie 2002). The deeper question, of course, is how we might understand a single human genome sequence, a composite that belongs to no actual individual in its entirety and only a handful of individuals in its parts, to be representative of the entire species. This seems to ignore the extensive genetic variability which exists. The functional equivalence of many DNA polymorphisms led two early critics of the HGP to argue that “there simply is no such entity as a ‘representative sequence’ or the human (or any) genome” making it “fallacious and even dangerous to call any one ‘normal’” (Sarkar and Tauber 1991, p. 691). Another critic pointed out that problems with the idea of a representative sequence persist even when consideration is limited to DNA differences that are not functionally equivalent but related to health and disease: the sequence will contain unknown defective genes (since no one, including donors, is free of these), there is a heterogeneity of mutations even in so-called single gene diseases, and it is impossible to identify the genetic basis of a disorder simply by comparing the sequences of sick and well people since there will be many differences between them (Lewontin 2000 [1992]). For Gilbert (1992), these criticisms of representativeness arise from a failure to appreciate the difference between the approaches of molecular biologists who attend to similarities and evolutionary biologists who attend to differences within the species: “The human genome project ... is directed toward a molecular biologist's view of a species rather than a population biologist's view. The latter views a species as the envelope of all possible variants that can breed together; the importance of that envelope is that different aspects of a species population will be drawn forth if you change the environment. Molecular biologists generally view the species as a single entity, sharply defined by a set of genes and a set of functions that makes up that entity” (p. 84). Gilbert held that the two approaches are consistent with each other, but many evolutionist critics of the HGP—both scientists and philosophers—did not, deriding the aims of mapping and sequencing the human genome as a throwback to anti-evolutionary, preDarwinian, typological, and essentialist thinking.[16] The functional approach of molecular biologists alluded to by Gilbert is said to represent genetic variation in improperly normative ways, whereas “in evolutionary biology, variation is not the same as deviation” (Hull 1994, p. 208). When molecular geneticists view mutations as abnormal, not in the sense that they are rare or a change in form, but as “errors” in the genetic code or “damage” to the genome's proper structure, they impose an arbitrary a priori categorization: “it is genetic ‘errors’ that made us as a biological species: we humans are integrated aggregates of such ‘errors.’ Genetic variation is the source of evolution; it is the reason why there could be primates and not just protists or their precursors” (Limoges 1994, p. 124). There are related worries that the human genome reference sequence will arbitrate a standard of genetic normality; for example, the application of concepts like “genetic error” and “damage” to the genome institutes a call for correction or repair (Limoges 1994; also Murphy 1994). McKusick (1989) has defended the HGP's approach as “consistent with that of most biological research which depends on a few, and even on single individuals, to represent the whole, and with the fact, recognized by geneticists, that there is no single normal, ideal, or perfect genome” (p. 913). However, the normal-abnormal distinction is fundamental to the structure-function studies of proximate fields of biology like physiology and molecular genetics, and while McKusick is no doubt correct to say that geneticists accept that there is no single normal, ideal, or perfect genome, this does not mean that individual DNA sequences are not constituted as normal or abnormal based on their functional significance or that entire genomes are not deemed to fall inside or outside of an acceptable range. Indeed, the 1988 OTA report on the HGP recommends the “eugenic use of genetic information ... to ensure ... that each individual has at least a modicum of normal genes” (p. 85). It is little wonder that many worry that as an increasing number of mutations are identified and tested for, the range of what is considered normal may narrow, with diminished tolerance for those people who lie outside this range. And there can be no reassurance that judgments of health and disease, normality and abnormality, manage to escape normativity by being transported to the level of the genome; instead, they carry with them any social values and cultural biases that are implicated at the higher level. Says critic Ruth Hubbard (in Holloway 1995, p. 50): “I have gone out on a limb by saying that most people in our culture are very judgmental about women who terminate a pregnancy because of sex. How different is that from terminating a pregnancy because of Down syndrome?” With the HGP reference sequence available as a basis for comparison, attention has shifted to the genetic variation within the species that evolutionist critics accused the project at the outset of ignoring. Humans have been found to be 99.9 percent alike, with common sequence variants occurring every 1000 bases. There is interest in identifying the sites of the genome where variation occurs, the frequency of these differences, and their significance. The social significance attaching to such research was foreseen during the early years of the project. Scientist David Baltimore predicted that the HGP would reveal that the belief that “we are all equal, all the same” is a myth: “We are going to have to come to terms with the fact that we are all born with different talents and tendencies” (in Cooper 1994, p. 320). Similarly, philosopher Marc Lappé (1994) raised the possibility that the HGP could reveal group differences—with sequences localized to particular groups or varying in frequency among groups—and that any such differences in the genetic lottery would raise significant ethical implications for health care and social policy.[17] But the conceptualization of this variation also presents challenges—for example, in distinguishing between normal and abnormal genetic variation (Gannett 2003a), or drawing population boundaries in the constitution of individual versus group differences (Gannett 2003b). Pharmaceuticals are the most powerful engine driving post-HGP diversity research, and though “personalized medicine” was touted as a benefit of the HGP, en route, a detour via the study of group genetic differences has been taken. For example, the International HapMap Project, in order to compile a map adequately dense with SNP markers to permit the identification of genes implicated in common diseases and drug responses, sampled the DNA of four populations (European-Americans in Utah, Yoruba in Ibadan, Nigeria, Japanese in Tokyo, and Han Chinese in Beijing).[18] Likely due to lessons learned from the difficulties experienced by the Human Genome Diversity Project (see Reardon 2004), attempts were made to involve representatives of these groups in the planning of research through “community engagement” and “community consultation.” These efforts raise conceptual questions not only about the relations between what are ostensibly distinct social and biological groups (Gannett 2003b, Juengst 1998), but what makes a “community” (Davis 2000). Now that “group” genetic differences have become of interest to more than just evolutionary biologists and population geneticists, impetus is provided to longstanding debates about whether race is biologically real or socially constructed and more recent ones concerning the appropriateness of the use of racial categories in biomedical research (Gannett 2005; Root 2003). 2.1.3 Essentialism and genetic reductionism Various HGP proponents told us that we would discover “our human essence” in the genome. According to Dulbecco (1986), “the sequence of the human DNA is the reality of our species” (p. 1056); Gilbert is quoted as saying “sequencing the human genome is like pursuing the holy grail” (in Lee 1991, p. 9); on the topic of his decision to dedicate three percent of HGP funds to ELSI, Watson writes: “The Human Genome Project is much more than a vast roll call of As, Ts, Gs, and Cs: it is as precious a body of knowledge as humankind will ever acquire, with a potential to speak to our most basic philosophical questions about human nature, for purposes of good and mischief alike” (with Berry 2003, p. 172). There are theological worries about a genetic reductionism that suggests that we are no more than our smallest material parts—the bits of DNA that make up the genome. For example, Leon Kass, chairman of the President's Council on Bioethics from 2001 to 2005, decries, with arrival of “the age of genetic technology,” “the erosion, perhaps the final erosion, of the idea of man as noble, dignified, precious or godlike, and its replacement with a view of man, no less than of nature, as mere raw material for manipulation and homogenization” (2002, p. 138). Collins, an evangelical Christian, doesn’t share such worries; he is quoted in the Los Angeles Times as saying: “God is not threatened by all this. I think God thinks it's wonderful that we puny creatures are going about the business of trying to understand how our instruction book works, because it's a very elegant instruction book indeed” (Gosselin 2000). Of course, this particular religious world view is countered by an evolutionist one held by other scientists. Gilbert's “holy grail” is not so holy after all; he believes that the HGP reveals our place amidst the interconnectedness of all life forms: “The data base of the human genome, coupled with our knowledge of the genetic makeup of model organisms, promises to reveal patterns of genes and to show us how we ourselves are embedded in the sweep of evolution that created our world” (1992, p. 97). A more secular philosophical concern about essentialism is tied to longstanding debates in philosophy of biology about species (see Erefshefsky 1992). Gilbert (1992) foresaw from the HGP a DNA-based definition of Homo sapiens: “At the end of the genome project, we will want to be able to identify all the genes that make up a human being. For example, we will compare the sequences of the human and the mouse and be able to determine the genes that define a mammal by this comparison.... So by comparing a human to a primate, we will be able to identify the genes that encode the features of primates and distinguish them from other mammals. Then, by tweaking our computer programs, we will finally identify the regions of DNA that differ between the primate and the human—and understand those genes that make us uniquely human” (p. 94). While it is true that any stretch of DNA that belongs to all and only humans would be among those differences found by comparing a single human genome sequence to a single nonhuman primate or mouse genome sequence, any “uniquely human” differences could not be distinguished from the others without extensive infra- and inter-specific population studies which are not part of the HGP. Even if such population studies were carried out, Gilbert's assumptions about species essentialism—that species can be defined or represented by properties (in this case, certain stretches of DNA) universally shared among, and particular to, their members—have long been challenged by philosophers of biology (Gannett 2003a; Robert and Baylis 2003). Because evolution is a gradual process where species are constantly undergoing change, Aristotelian (essentialist) definitions of species need to be abandoned; from an evolutionary perspective, in David Hull's (1994) words: “The essence of a particular species is to have no essence” (p. 215). Species should instead be defined as cluster concepts (Hull 1965) or recognized to be individuals (i.e. spatio-temporally restricted, historically contingent particulars) to which organisms belong as parts, and not classes, sets, or natural kinds at all (Ghiselin 1974; Hull 1978). Besides these attempts to reduce species to beanbags of genes, genetic reductionism enters in attempts to explain cellular or organismal properties solely in terms of genes, or entire organisms in terms of genomes. Gilbert (1992) endorses an essentialism of this sort as well: “The information carried on the DNA, that genetic information passed down from our parents,” he writes, “is the most fundamental property of the body” (p. 83), so much so, in fact, that “one will be able to pull a CD out of one's pocket and say, ‘Here is a human being; it's me!’” (p. 96). The social prevalence of this representation of the genome as the “most fundamental” aspect of the individual means that genetic information has a particularly acute impact on self-identity and self-understanding (Quaid 1994). Another genome scientist Eric Lander (1996) characterizes the HGP as “the 20th century's version of the discovery and consolidation of the periodic table” with the genes “elements” and gene variants responsible for disease susceptibilities “isotopes” (pp. 536–537). The probable social consequence of this beanbag conception of the organism, combined with a concept of genetic disease that relocates the locus of disease from organism to genome, is the direction of technological fixes at the genome (Keller 1994). When these technological fixes include prenatal genetic screening and the possible modification of IVF embryos, it is suggested that genetic reductionism contributes to the commodification of children by making them an instrument of parental desire (Darnovsky 2001). The relevant notion of “reduction” at play here is the explanation of wholes in terms of parts. As Sahotra Sarkar (1998) notes, it is important to distinguish between genetic reductionism and physical reductionism: “From the point of view of physical reductionism, DNA enters the molecular milieu on par with proteins or, for that matter, lipids or any other molecules that are found in living organisms. Physical reductionism does not require any assumption about the primacy of DNA or of genes in the explanation of biological behavior” (p. 174). The reduction of organisms to their genomes by molecular geneticists takes yet further molecular biology's—and, more generally, proximate biology's—reduction of organisms to their constituent physical parts in a way that effaces the contexts (provided by populations and environments) in which organisms develop (Griesemer 1994). Definitions of health and disease attach to organisms and their physiological processes in particular environments and cannot simply be relocated to the level of the genome (Limoges 1994; Lloyd 1994). It is wrong to presume that diseases become more objectively defined entities once they receive a genetic basis since social and cultural values implicated in designations of health and disease can merely become incorporated at the level of the genome, in what counts as a normal or mutant gene. There is an additional sense in which genetic reductionism is implicated in the HGP, and Gilbert makes reference to this as well. This is the sense, familiar to philosophers of science, of intertheoretic reduction, whereby (usually) higher-level theories are said to be reduced by lower-level ones insofar as these lower-level theories explain/predict the phenomena of the higher level. Gilbert (1992) foresaw that the HGP would furnish the basis for a “theoretical biology” in which from the genome's DNA sequence it would be possible to predict protein sequence, and from protein sequence it would be possible to predict three-dimensional protein structure—either from “first principles” based on energy calculations or from observed structural similarities of the building blocks—and from there make predictions about function, a predictability that Gilbert suggests would extend to individual organisms and their behavior, and might therefore be difficult to accept: “To recognize that we are determined, in a certain sense, by a finite collection of information that is knowable will change our view of ourselves” (p. 96). Gilbert seems to conflate epistemology with ontology, moving from genetic reductionism where genes suffice to predict or explain behavior to genetic determinism where genes are sufficient causes of behavior (see next section), but more importantly, even at the lowest level of organization, his vision faces formidable obstacles. Notwithstanding the protein folding problem and the need to consider gene regulation in order to proceed beyond the level of protein structure, there are significant difficulties in attempting even to predict the linear structure of proteins from sequence data alone: specifically, abilities are limited for recognizing transcription initiation sites and, in the presence of extensive RNA editing, the boundaries between introns and exons and coding and noncoding segments of DNA (Sarkar 1998).[19] Molecular biology's technological capacity to manipulate the human genome brings society into something of an existentialist predicament. Science has tended to conceive human essence as a fixed object discoverable in nature. But a human essence embedded in manipulable genome is not immutable—it is created, not discovered. There is a very real sense in which in making the difficult choices we face—for example, those involved in prenatal genetic testing and germ-line manipulation—we really are choosing ourselves.[20] 2.1.4 Genetic determinism and the nature-nurture controversy Gilbert's reductionist vision of the sequenced human genome as “the grail” upon which a “theoretical biology” can be founded brings to the fore philosophical questions about genetic determinism. One might ask with Richard Lewontin (2000 [1992], p. 139), however rhetorically: “How is it that a mere molecule [DNA] can have the power of both self-reproduction and self-action, being the cause of itself and the cause of all other things?” Getting straight on genetic determinism is important. There is a long, ignoble history of marshalling ideological justification for unjust and oppressive social and political institutions and structures by appealing to the ostensibly scientific assertion that “human nature is fixed by our genes” (Rose et al. 1984; also Lewontin 1993).[21] Critics of the HGP saw it as placing “the seal of approval from mainstream science” on hereditarianism, favoring nature over nurture like the eugenics of the early to mid-20th century, to promote a “technological fix” for social problems (Allen 1994, p. 164).[22] However, with the HGP nearing completion and the availability of entire genome sequences for numerous organisms supporting the movement from structural to functional genomics, Keller—one such early critic—found that the deterministic and reductionistic assumptions underlying the HGP had actually been undermined by the research in molecular biology the HGP made possible: “What is most impressive to me is not so much the ways in which the genome project has fulfilled our expectations but the ways in which it has transformed them.... Contrary to all expectations, instead of lending support to the familiar notions of genetic determinism that have acquired so powerful grip on the popular imagination, these successes pose critical challenges to such notions” (2000, p. 5). Yet, DNA is still portrayed as fundamental: in a public lecture held in celebration of the completion of the HGP, Collins characterized the HGP as “an amazing adventure into ourselves, to understand our own DNA instruction book, the shared inheritance of all humankind.”[23] While virtually all biologists disavow genetic determinism today, it is not always so clear what exactly they are denying. Jonathan Kaplan (2000) identifies three different ways in which claims about genetic determinism might be understood: (i) as “complete information” where everything about us is viewed as predictable based on our genes; (ii) as “intervention is useless” where traits are said to be impervious to environmental changes; and (iii) as traits that are in some sense primarily, even if not wholly, genetic. Kaplan argues that when biologists disavow genetic determinism it is (ii) they have in mind (with phenylketonuria—PKU—frequently used as an example).[24] According to Kaplan, (i) is easily seen to be “trivially false,” and therefore not worth disavowing—yet, this resembles Laplacian determinism's concern with predictability, and as we have seen, Gilbert seems to be making such a claim. Despite their disavowals of genetic determinism, Kaplan finds that biologists often adhere to (iii); however, the basis for the primacy of genes remains to be understood. Questions about genetic determinism and Collins’ representation of the sequenced human genome as “our own DNA instruction book”—which suggests an asymmetry between genetic and nongenetic causes—need to be approached at several different levels: cellular, organismal, and societal.[25] At the cellular level, the book is said to contain “the genetic instructions for the entire repertoire of cellular components” (Collins et al. 2003, p. 3). This genetic determinism at the cellular level is sustained by metaphors of Weismannism and DNA as “code” or “master molecule” (Griesemer 1994; Keller 1994). DNA is accorded causal priority over other cellular components in a couple of ways. One way is to treat DNA as temporally prior. This may be in a physical sense: Weismannism assumes that intergenerational continuity exists only for germ cell nuclei whereas somatic cells and germ cell cytoplasm arise anew in each generation. It may also be in the sense of a point of origin for the transfer of information: the central dogma of molecular biology, which represents a 1950s reformulation of Weismannism in terms of information theory, asserts that information travels unidirectionally from nucleic acids to protein, and never vice versa. The chief difficulty for these claims of temporal priority is of the chicken-and-egg variety: nucleic acids need proteins and other cellular components to make proteins (Smith 1992). Although it is fully accepted that the fertilized ovum contains the cytoplasmic contribution of at least the maternal germ cell, there persists a tendency in developmental genetics to focus on cytoplasmic (mitochondrial) DNA and to ignore the role of cytoplasmic proteins. It is also contentious whether amongst the cell's components only nucleic acids can be said to transmit information: for some philosophers, genetic coding plays a theoretical role at least at this cellular level (Godfrey-Smith 2000); for others, genetic coding is merely (and misleadingly) metaphorical, and all cellular components are potential bearers of information (Griffiths 2001; Griffiths and Gray 1994; Sarkar 1996). An additional way in which DNA is accorded causal priority lies in its treatment as ontologically prior: this is exemplified in Watson's description of DNA as “the most golden of molecules” (in Bodmer and McKie 1994, p. 10). Causal asymmetry provides a possible reason for privileging DNA on an ontological basis: provided all cellular components necessary for protein synthesis are present, modification of the DNA sequence may be followed by a predictable and specifiable change in protein sequence, but the opposite will not occur. This difference could be conceived in terms of the Aristotelian distinction between formal and efficient causation and the accompanying metaphysical preferences for form over matter and mind over body that are deeply embedded in western philosophy. Keller (2000) describes how, in the discourse of “gene action” which arose between the mid-1920s and 1960s and culminated in Francis Crick's “central dogma,” “the gene was bestowed with the properties of materiality, agency, life, and mind” and rendered “[p]art physicist's atom and part Platonic soul” (p. 47).[26] At the level of the organism, talk of genetic coding and the asymmetry between genetic and nongenetic causes such talk conveys, even when countenanced at the cellular level, are deemed less acceptable (Godfrey-Smith 2000). New research in functional genomics may well lead to less deterministic accounts even of so-called single gene disorders. For these, the concepts of penetrance and expressivity operate in ways which accommodate the one-one genetic determinist model where the mutation is necessary and/or sufficient for both the presence of the condition and confounding patterns of phenotypic variability. But the severity of even a fully penetrant condition like Huntington's disease seems to depend on not just genetic factors like the number of DNA repeats in the mutation but epigenetic factors like the sex of the parent who transmitted the mutation (Ridley et al. 1991). At the level of individuals in society, when we consider complex conditions to which both genetic and environmental differences contribute—for example, psychiatric disorders or behavioral differences—gene-centrism persists. The April 1998 cover of Life captures the reader's attention: “WERE YOU BORN THAT WAY? Personality, temperament, even life choices. New studies show it's mostly in your genes.” Leading scientists have said similar things. At the outset of the HGP, Watson told us: “we used to think our fate is in our stars. Now we know, in large measure, our fate is in our genes” (in Jaroff 1989). Post-HGP, Watson seems unaffected by the changes that have so impressed Keller. While he introduces the recent book Behavioral Genetics in the Postgenomic Era by stating confidently that “with the arrival of the human DNA genome sequence and its attendant list of human genes, the experimental procedures will soon be on hand to finally settle the long contentious nature-nurture arguments” (p. xxii), the question seems already settled for him in his assertions that “children come into the world with fixed personalities” and “effective remedies for socially inappropriate behaviors” will best be carried out at the molecular level (in Plomin et al. 2003, p. xxii). But notice the waffle words used by Watson and on the cover of Life: “in large measure” and “mostly in your genes.” Everyone is an interactionist these days, in some sense of “interaction.” Genes and environment, or nature and nurture, are recognized both to be necessary for development: by themselves, genes can’t determine or do anything. Yet, theorists still seem to give the nod to one or the other, suggesting that it is mostly genes or mostly the environment, mostly nature or mostly nurture, that make us what we are. This implies that it is possible to apportion the relative contributions of each. Gilbert (1992) suggests this in his dismissal of a more simplistic version of genetic determinism: “We must see beyond a first reaction that we are the consequences of our genes; that we are guilty of a crime because our genes made us do it; or that we are noble because our genes made us so. This shallow genetic determinism is unwise and untrue. But society will have to wrestle with the questions of how much of our makeup is dictated by the environment, how much is dictated by our genetics, and how much is dictated by our own will and determination” (pp. 96–97). However, the assertion that the relative contributions of genes and environment, nature and nurture, can be apportioned in this way is misleading if not outright false. As Lewontin argued in his classic paper on heritability, it is impossible to infer causal relations from the analysis of variance. The only legitimate exception is where there is “perfect or nearly perfect additivity between genotypic and environmental effects so that the differences among genotypes are the same in all environments and the differences between environments are the same for all genotypes” (1974, p. 408).[27] Contrary to Watson's assertion, the replacement of quantitative with molecular genetic techniques cannot resolve the nature-nurture controversy because of the same problem that affects heritability measures: the context-dependence of genes as causes given the nonadditivity of gene-gene and gene-environment interactions. Recent work in developmental systems theory (DST) which undermines any such attempts to apportion causal responsibility in organismal development makes clear why: traits are jointly determined by multiple causes, each context-sensitive and contingent (Griffiths and Gray 1994; Griffiths and Knight 1998; Oyama 1985; Oyama et al. 2001; Robert 2004).[28] When genetics enter philosophical debates about freedom and determinism, questions about moral and legal responsibility are central: if the genes a person happens to inherit can be said in some sense to determine her actions, is it legitimate to praise, blame, reward, or punish that person? Retributivism pulls in opposing directions: genetic predisposition to violent or criminal acts may suggest “a volitional disability that makes blame inappropriate” or “a permanence that invites blame” (Wasserman 2001, p. 304). The HGP is unlikely to enlighten or complicate these longstanding debates, however (Baron 2001). The thesis of universal causation (no uncaused events) and its implications for freedom are unaffected by genetics: it makes no sense to claim that all events have genetic causes, and were it to turn out that certain events—i.e. human behaviors or actions—have genetic causes, these pose no different a threat for freedom than their nongenetic counterparts. In addition, genetic causes of behavior are likely to be tendencies or predispositions which do not necessitate their effects: for incompatibilists, genetic and nongenetic causes are jointly responsible for behaviors, and genetic determinism adds nothing to the challenge determinism already poses for freedom; for compatibilists, since a person's behavioral genetic tendencies or predispositions do not compel her to act in a certain way, they are no different than nongenetic (biological or environmental) tendencies or predispositions not of a person's own making. Of course, the general public—in voting booths, on juries, etc.—may be swayed more by genetic explanations given beliefs in genetic determinism fuelled by media reports of apparent discoveries of genes for this or that behavior. 2.1.5 Genetic explanations The gene is a “cultural icon”: in popular culture, from movies to cartoons to Dear Abby, quite apart from its biological and medical contexts, the gene has become “a symbol, a metaphor, a convenient way to define personhood, identity, and relationships in socially meaningful ways” (Nelkin and Lindee 1995, p. 16). Hardly a week goes by when we do not hear about a newly discovered gene for one thing or another. “Geneticization” is a term used to describe this phenomenon marked by an increasing tendency to reduce human differences to genetic ones (Lippman 1991).[29] This tendency is accompanied by worries of critics that embracing a reductionist approach to medicine that conceives of human health and disease in wholly molecular or genetic terms individualizes these and detracts attention from our shared social and physical environments and the role of toxins, fast food, poverty, lack of access to health care, etc. (Nelkin and Tancredi 1989; Hubbard and Wald 1993). One of the justifications for spending several billion dollars on human genome research is the belief that genes are key determinants of not only rare Mendelian diseases like Huntington's disease or cystic fibrosis but common multi-factorial conditions like cancer, depression, and heart disease. In Watson's words: “Some call New Jersey the Cancer State because of all the chemical companies there, but in fact, the major factor is probably your genetic constitution” (in Cooper 1994, p. 326). Writes an early critic of the HGP: “Without question, it was the technical prowess that molecular biology had achieved by the early 1980s that made it possible even to imagine a task as formidable as that of sequencing what has come to be called ‘the human genome.’ But it was the concept of genetic disease that created the climate in which such a project could appear both reasonable and desirable” (Keller 1992, p. 293). Given that the development of any trait involves the interaction of both genetic and nongenetic factors, on what bases can genes be privileged as causes in order to claim that a particular disease or nondisease trait is “genetic” or caused by a “genetic susceptibility” or “genetic predisposition”? Does it make sense for HGP proponents like Bodmer to characterize even smoking-induced forms of cancer as genetic? “Cancer, scientists have discovered, is a genetic condition in which cells spread uncontrollably, and cigarette smoke contains chemicals which stimulate those molecular changes” (Bodmer and McKie 1994, p. 89).[30] From the outset, we need to distinguish between genes conceived as causes of a trait's appearance in a given individual (“x is a gene for trait y in organism z” or “My three-pack-a-day Aunt Viv must have the gene that causes cancer”) and genes as causes of differences in traits among individuals (“x is a gene for trait y in population z” or “Lots of people in my family smoke, but only Aunt Viv and Cousin Sal seem to have inherited the gene for cancer”).[31] The logical interrelatedness of cause and effect—that is, whether a condition is necessary and/or sufficient for a given event to occur—is the approach taken to defining what makes a condition “genetic” in individuals. A strong sense of “genetic disease” is recognized when the genetic factor is both necessary and sufficient for the disease to arise “regardless of environment” (Wulff 1984), or when the genetic factor is sufficient for the disease to present “in all known environments” (Kitcher 1996)—this latter definition recognizes that, in some cases, a disease may have nongenetic as well as genetic origins (since the genetic factor is sufficient but not necessary). “Genetic susceptibility” is defined as an increased probability of disease in all known (strong sense) or some (weak sense) environments (Kitcher 1996). Note that ceteris paribus clauses referring to an assumed background of necessary, though not sufficient, genetic and environment factors are required by these definitions. Just as striking a match causes it to ignite only if it is dry and in the presence of oxygen, as we saw in the previous section, genes don’t do anything alone. This is the first of three ways in which genetic explanations are context-dependent. Adopting a population-based approach to genetic causation, where differences in genes are understood to explain differences in traits and not traits themselves, replaces the need for ceteris paribus clauses because they rely on the actual distribution of the necessary genetic and nongenetic background factors in specific populations. The case can be made that the first approach is indebted to the second, and that one never explains a property of an object tout court but only in relation to a reference class of an object or objects that lack the property (but share the necessary background factors). Writes Germund Hesslow (1983), “all explanations of individual facts of the form Fa—that is, where an object a has a certain property F—involve a comparison with other objects which lack the property in question” (p. 91). No trait can be labeled “genetic” in any absolute sense, but only relative to a specific population. For example, lactose intolerance is considered to be a genetic condition in northern European populations where ingestion of milk products is common and lactase deficiency rare, whereas in African populations, where ingestion of milk products is rare and lactase deficiency common, it is considered to be an environmental condition (Hesslow 1984). This is the second way in which genetic explanations are context-dependent.[32] The third, and final, way in which genetic explanations are context-dependent is that they are a function of the present state of knowledge. Huntington's disease is deemed a genetic condition on both the individual and population accounts: a single mutant gene is necessary, and arguably sufficient given necessary (and standard) background conditions, for symptoms to appear in a given person; the presence and absence of disease symptoms in members of the population is accounted for in terms of the presence and absence of the mutation. This is nevertheless an epistemically relative claim. Once the relevant gene is mapped and sequenced, the mechanisms by which genetic and nongenetic factors interact to produce symptoms of the disease remain to be understood. Such causal knowledge is often obtained through the experimental manipulation of conditions beyond “normal” limits, and what conditions are exploited as possible causes in the laboratory and what conditions are kept constant as necessary background, along with pragmatic decisions about how research efforts should be expended more generally, are influenced by clinical and social, as well as scientific, contexts (Gannett 1999).[33] Behind philosophical attempts to seek objective, nonevaluative foundations for designations of diseases as “genetic” or “environmental” lie positivist assumptions that theoretical understanding furnishes the basis for rational action. One concern with geneticization and the trend to label an increasing number of diseases and conditions “genetic” is that this provides normative support for directing future research and therapeutic interventions in particular ways, that is, at the level of the genome (Cranor 1994). Watson's (1992) colorful metaphor makes this normative support explicit: “Ignoring genes is like trying to solve a murder without finding the murderer. All we have are victims” (p. 167). But this is fallacious reasoning, as the context-dependence of genetic explanations shows. We might instead understand geneticization to be the consequence of an increased capacity to manipulate DNA in the laboratory and (potentially) the clinic and not an advancement in theoretical understanding. Genetic explanations, on such a view, are pragmatic: there is a practical context in which genes are singled out as causes not only because they are amenable to technological control but because they are increasingly perceived to be more tractable than their nongenetic counterparts and therefore the best means to a variety of ends (Gannett 1999). 2.1.6 Model organisms Many of the model organisms chosen for the HGP had already enjoyed illustrious careers in the history of genetics: the fruitfly D. melanogaster was the organism that started it all in T. H. Morgan's lab at Columbia University in the 1910s, ushering in the era known today as classical genetics; with discoveries of spontaneous mutation and recombination in the 1940s, the bacterium E. coli helped to take genetics molecular, serving also as host for the phage studied by Max Delbrück's group; the nematode worm C. elegans was Sydney Brenner's choice to model the development of the nervous system in the mid-1960s at Cambridge University.[34] It was, in fact, these histories that recommended them: “the experimental organisms that became ‘model organisms’ were not selected and constructed mainly on the basis of principles of universality or even typicality of their biological characteristics and processes, though it was hoped that many features would prove to be shared or common to other organisms, particularly humans. Instead they were primarily chosen for ease of experimental tractability and due to the availability of some background information on basic genetic composition and relation to phenotype” (Ankeny 2001, pp. S253-S254). Philosophical questions arise about the senses in which these various organisms serve as “models.” Potentially, models may embody a range of characteristics: as typical or representative; as ideal or perfect; as convenient, tractable, or manipulable; as homologous (conserved evolutionarily); as analogous; as exemplars; as abstractions. Models may also be used in a variety of ways: to model disease processes; to model normal processes; as structural models; as type organisms representative of the species or higher phylogenetic level; as heuristic tools; as mathematical devices.[35] In a recent article examining researchers’ use of the flowering plant Arabidopsis thaliana as a model organism, Sabina Leonelli (2008) points out that models can be abstract (vs. concrete) in different ways: absolutely, in terms of their sense perceptibility; or relatively, in terms of their physical meaning with respect to the phenomena represented or the range of phenomena they are taken to represent. She then shifts the philosophical focus from models themselves to modeling practices: abstraction, for example, becomes a component of the activity of producing a model rather than solely an attribute of the model. This approach emphasizes the need to attend not just to the relationship between model and phenomenon modeled but the material, social, and institutional settings and varied commitments of researchers. Rachel Ankeny (2000, 2001) considers several ways in which organisms might plausibly be considered models. She suggests that mapped and sequenced genomes for these various HGP model organisms serve as “descriptive models.” A genome reference sequence is a model because it is, first, “an idealized, abstract entity constructed from the natural organism” and, second, “‘model’ is used to indicate a promissory note about this organism providing a framework for pursuing explanatory questions and ultimately serving as a prototype for understanding more complex organisms” (Ankeny 2000, p. S267). Genome reference sequences are descriptive because they are “constructed largely without motivation by hypotheses to be tested or traditional explanatory questions” but rather as preliminary to such work (Ankeny 2000, p. S267). In this, they are similar to earlier efforts to use wiring diagrams for C. elegans to model neural structure: these diagrams were based on data from several worms but were presented as canonical; the worms were “wild type” and presumed to exhibit species-typical structure; the diagrams served as a tool to investigate abnormalities (Ankeny 2000). Nonhuman genome reference sequences become tools as their corresponding organisms are used as experimental models for understanding basic biological processes common to many species or diseases processes found in humans—for example, by knocking out genes in mice. Here, analogical reasoning is at work; “[m]odels in this sense of the term seem to provide what might be termed strong causal analog models” (Ankeny 2001, p. S255). According to Kenneth Schaffner (1998a), this is quite typical of biological explanation: unlike physicists, biologists frame explanations “around a few exemplar subsystems in specific organisms ... used as (interlevel) prototypes to organize information about other similar (overlapping) models” (p. 278). Monod famously once said that what is true of E. coli is true of the elephant, but just as famously, this proved not at all to be the case in moving from prokaryotic to eukaryotic gene regulation. More recently, biologist Bruce Alberts writes: “we can say with confidence that the fastest and most efficient way of acquiring an understanding of ourselves is to devote an enormous effort trying to understand ... relatively ‘simple’ organisms” (in Schaffner 1998a, p. 277). What inspires such confidence when a simple organism like C. elegans with its 302 neurons and repertoire of behaviors (movement in response to touch and chemical stimuli, egg laying, and mating) is used to model human behavior in all its complexity? Most importantly, the model organism must be representative of the systems being modeled: genomic sequences must be similar in model and modeled organisms; there must be a known cause-effect relationship between the model organism's genotype and phenotype; there cannot be any “causally relevant disanalogies” between model and modeled organisms, for example, due to differences in complexity (Ankeny 2001, p. S257). Schaffner (1998b) examines molecular geneticists’ use of C. elegans as a behavioral model. Even in these simple organisms, relations between genes, neurons, and behaviors are complex (many-many), with one gene-one behavior associations rare exceptions and their intervening causal chains yet to be understood. While such complexity is to be expected in humans with their more complicated nervous systems, Schaffner believes that there may be a small number of single gene effects on behavior where these genes are highly homologous and strongly conserved—hence, the usefulness of simple models like C. elegans combined with others for investigating basic mechanisms and psychiatric disorders. The model organism approach faces challenges, however. As Schaffner recognizes, model organisms are also idealizations: organisms are selected for features not generalizable even to close relatives like rapid development, short generation time, small adult size, and insensitivity to environmental variation (Wimsatt 1998); strains are inbred to remove genetic diversity. Context-sensitivity diminishes expectations that similar mechanisms operate in simple and complex systems; multiple realizeability creates doubts that similar explanations will be found across taxa (Wimsatt 1998). Evolution is a branching process; as Richard Burian (1993) emphasizes: “At (virtually?) all levels of the biological world—including the biochemical—it is an open question how general the findings produced by the use of a particular organism are” (p. 365). Consequently, support for theoretical hypotheses requires the experimental findings to be placed within a comparative and evolutionary framework attentive to how widely the relevant nucleotide sequences and traits are distributed phylogenetically: “detailed knowledge of (historical) biological contingencies constrains—and ought to constrain—the evaluation of experimental work in biology and the knowledge claims based on that work” (Burian 1993, p. 366).[36] 2.2 Ethical Implications of the Human Genome Project 2.2.1 Gene patents The policy of the U.S. Patent and Trademark Office (PTO) used to be that life forms, as products of nature, were unpatentable. Only products and processes invented by humans could be patented. But what about genetically modified life forms: are they invented or discovered, the product of nature or humans? In 1980, the U.S. Supreme Court issued its 5–4 decision in Diamond v Chakrabarty that a bacterial strain that had been genetically modified to clean up oil spills could be patented since it was “man-made” and not naturally occurring. Since this ruling, “the PTO has awarded thousands of patents on biological products, including patents on genes, SNPs, ESTs, cell lines, mice, plants, rhesus monkeys, and human stem cells” (Resnik 2004, p. 54).[37] Following Diamond v Chakrabarty, genes and gene products are considered patentable in their non-naturally-occurring forms, even if, unlike genetically modified life forms, the sequence information they contain is unaltered from what occurs naturally.[38] Patents are granted on cloned genes because they are isolated and purified versions of their naturally occurring state; patents are granted on cDNAs because they are not found in nature (without intervening introns) but produced using RNA molecules and the enzyme reverse transcriptase; and patents are granted on proteins, even if they occur naturally, when genetically engineered on the basis of DNA or RNA sequence data. For such cases, DNA sequences are treated by the PTO and U.S. Court of Appeals for the Federal Circuit as “compositions of matter” (the wording contained in the patent statute) analogous to patentable chemical compounds (Eisenberg 2002).[39] Controversy about patenting genes was provoked over the course of the HGP when researchers at the NIH and elsewhere applied for patents on ESTs (expressed sequence tags) for cDNAs without having mapped and sequenced the relevant genes, and without any knowledge of gene function. Granting such patents means that subsequent researchers who discover functions conducive to the development of diagnostic tests or treatments are faced with paying licensing fees or royalties. For example, in 1996, researchers discovered that the protein CCR5 plays a role in HIV infection, but a patent on the gene for CCR5 had already been applied for by the biotech company Human Genome Sciences having postulated a possible anti-inflammatory role for the gene using computational methods (Smaglik 2000). Many human genome scientists opposed the policy of granting gene patents without knowledge of function, whether for ESTs and cDNAs or complete gene sequences. Human Genome Organization (HUGO) issued this 1995 statement: “HUGO is worried that the patenting of partial and uncharacterized cDNA sequences will reward those who make routine discoveries but penalize those who determine biological function or application. Such an outcome would impede that development of diagnostics and therapeutics, which is clearly not in the public interest.”[40] In 2000, the NIH and scientific bodies like the National Academy of Sciences and Royal Society of London called for tighter rules to discourage patent applications on genes identified merely by computational analysis of sequence data, the public release of which the 1996 Bermuda Accord required of sequencing centers every 24 hours (Alberts and Klug 2000; Dickson 2000). That year, the PTO revised its guidelines to require that researchers applying for patents demonstrate utility (U.S. Department of Commerce 2001).[41] Controversy has also arisen over whether the purposes of the intellectual property system are being fulfilled in the case of gene patents. Patents are supposed to work to stimulate scientific research and technological development by removing the need for secrecy, and this in turn is supposed to benefit society as a whole. Many believe that secrecy has increased, that there is less sharing of data and techniques between laboratories, and that research and development efforts that build on initial discoveries are being stymied. When researchers are issued a patent on a gene they have mapped and sequenced, they may license exclusive rights to a biotech or pharmaceutical company to develop and market applications—perhaps a drug or diagnostic test. Research costs are driven up when scientists who must test individuals in their studies are beset by royalty payments. Without a competitive market, the costs of administering diagnostic tests in the clinic also rise. For example, Myriad Genetics patented BRCA1 and BRCA2 breast and ovarian cancer genes and granted Eli Lilly exclusive rights to market applications based on the BRCA1 sequence. The cost for a woman at risk of developing cancer to be tested for BRCA mutations was $2400-$3500, and researchers working to understand how these mutations are implicated in the development of cancer also had to pay the fee (Reynolds 2000). Of particular ethical concern is that much of the research which results in patented discoveries is funded by government or nonprofits. For example, the BRCA research was federally funded: it was conducted by researchers at the University of Utah (which assigned its rights to Myriad) who were supported by NIH and at the National Institute of Environmental Health Sciences (Reynolds 2000). This is an outcome of a U.S. governmental policy that encourages federally-funded researchers to seek partnerships with the private sector. So, as taxpayers and consumers, the public ends up paying twice. These discoveries also could not have been made without the cooperation of patients and their families, and yet, while researchers and institutions profit, they are faced with more expensive tests and treatments (see Merz et al. 2002 on patenting's effects on the cost and availability of genetic tests for hereditary haemochromatosis). Intellectual property rights also raise ethical questions about conflict of interest and exploitation. Such potential exists in research that studies relatively small, isolated populations to see if there are any rare genes present which may be of value for pharmaceutical development (Dickenson 2004; Salopek 1997). In one well-known episode, organizations representing indigenous peoples mounted opposition to a 1995 patent granted to the NIH on a cell line obtained from a Hagahai man from Papua New Guinea, a claim later withdrawn by the NIH due to the controversy. The patenting of human genes and cell lines is seen as a continuation of the “bioprospecting” and “biopiracy” that have taken place over the past several decades with western corporations securing patents on medicinal and food uses of plants which have been long a part of traditional knowledge (Shiva 1996). Problems also arise in clinical research. With multiple sources of conflict of interest operating, there was the potential for exploitation when teenager Jesse Gelsinger died in a gene therapy trial in 1999. Researcher James Wilson held patents on several of the procedures used; Genovo, the private firm sponsoring the study, was founded by Wilson; Wilson and his employer, the University of Pennsylvania, held equity in Genovo; and Genovo was providing $4 million per year to the university's Human Gene Therapy Institute (Resnik 2004, p. 162). Philosophical arguments in support of gene patents include: patents are just rewards for researchers’ efforts and costs; by mixing their labor with what occurs naturally, researchers acquire property rights in the Lockean sense; patents contribute to scientific and technological progress by diminishing secrecy and providing incentives; society ultimately benefits with better and more economical medical treatments (Resnik 1997a). Philosophical arguments in opposition to gene patents include: patents exert negative effects on scientific research by promoting secrecy and impeding openness and sharing of data and technologies among scientists, leading to publication delays, inhibiting further research, and encouraging researchers to pursue projects of short term, commercial benefit (Nelkin 2002); patents operate as monopolies which compromise medical care by driving up the costs of diagnostic tests and pharmaceuticals and creating financial conflicts of interests which diminish trust between physicians and patients (Nelkin 2002); patenting is an unacceptable and dehumanizing commodification of life (Dickenson 2004); genes even if isolated and purified remain products of nature and are therefore not patentable (Sagoff 2002); the human genome is the “common heritage of humanity” not private property (HUGO 2000).[42] 2.2.2 Genetic testing Early on the in the HGP, it was recognized that the development of genetic tests would precede, perhaps by decades, the capacity to treat the corresponding conditions successfully: this emphasized the importance of confronting the ethical implications of genetic testing. Genetic testing is carried out for a range of purposes: diagnostic, predictive, and reproductive. Diagnostic genetic testing is performed on individuals already experiencing signs and symptoms of disease. Predictive genetic testing is performed on individuals who are at risk for inheriting a familial condition but do not yet show any signs or symptoms. Predictive testing is presently offered for a number of conditions: these include Huntington's disease (HD), cystic fibrosis (CF), sickle cell anemia, breast cancer, and colon cancer. Although the HGP was defended by its proponents on the basis that the knowledge of individual genome sequences would facilitate the movement from a reactive to predictive medicine, there are no cures yet for most diseases identified as having a genetic basis. For some conditions, like familial hypercholesterolemia or polycystic kidney disease, early intervention may lessen the severity of symptoms. Reproductive genetic testing is carried out in several ways: through carrier screening, prenatal testing of the fetus in utero, and preimplantation genetic diagnosis (PGD) of embryos created by in vitro fertilization (IVF). In carrier screening, prospective parents find out whether they are at risk for passing on disease-related genes to their offspring. Prenatal genetic testing of fetuses in utero is conducted through the use of blood tests early in a woman's pregnancy, chorionic villus sampling (CVS) at 10–12 weeks, and amniocentesis at 15–18 weeks. Testing is generally offered to women for whom risk is elevated because of age or family history; based on the results, women can elect to continue the pregnancy or abort the fetus.[43] In PGD, a single cell is removed from the 8-cell embryo for testing; based on the results, a decision is made about which embryo(s) to implant in the woman's uterus. In the U.S., PGD is restricted to private clinics because of the government's ban on the use of federal funds for embryo research.[44] Genetic testing carried out at the population level—for any of these purposes—is referred to as genetic screening. Newborn screening programs to diagnose conditions like PKU, hemoglobinopathies, etc. on the basis of blood components and circulating metabolites have been carried out for several decades in many jurisdictions; while such screening could be expanded to include genetic tests for many more diseases, without effective treatment measures, this is of limited use. Ethical debate exists over whether newborn screening should be mandatory (see Andrews 1994). While successful population-based carrier screening has been carried out in Sardinia for beta-thalassemia and for Tay-Sachs among U.S. Ashkenazi Jews, the negative impacts of testing for sickle-cell anemia among African Americans provide caution for launching further such programs—outcome is likely to depend on the social status of the targeted group (Duster 1990). The pathway from gene discovery to population-level screening is not straightforward, however. Although the gene implicated in CF was identified in 1989, statements issued by the American Society of Human Genetics in 1990 and 1992 and by a NIH workshop in 1990 recommended against instituting population-based carrier screening because adequate detection rates were compromised by the heterogeneity of mutations found; in addition, the clinical course of the disease remained difficult to predict, and resources for education and counseling were lacking (ASHG 1990; ASHG 1992; NIH Workshop 1990).[45] The 1997 report of the ELSI Working Group's Task Force on Genetic Testing focuses on ensuring the safety and effectiveness of genetic tests (Holtzman and Watson 1997)—an important concern given the social pressures likely to encourage the expansion of genetic testing and screening as more tests become available: commercial interests of the biotech industry; clinicians’ (and their malpractice insurers’) fears of wrongful birth or wrongful life lawsuits; and expectations of governments and other institutions (e.g. private foundations, HMOs) that, by preventing the births of individuals with costly conditions at sufficient rates, carrier and prenatal screening programs will prove cost-effective (Holtzman 1989; Paul 1994a). At the opposite end of the spectrum, the expanding availability of direct-to-consumer genetic tests, which the report recommends against, has led to ethical controversy (Beckman 2004). Like the 1994 report published by the Committee on Assessing Genetic Risks (Andrews et al. 1994), the Task Force report also stresses the need for voluntariness, informed consent, confidentiality, legislative protection against discrimination, improved education of clinicians and the public, nondirectiveness in reproductive decisions, and availability of genetic counseling. Professional codes of ethics for genetic counseling (e.g. the National Society of Genetic Counselors in the U.S.) emphasize the importance of value-neutral, nondirective counseling in all of these situations: the goal is for clients to make their own decisions based on their own values once they are fully informed about their options. Ethical questions are raised about the ideals of value-neutrality and nondirectiveness: choosing which information to present and how to do so inescapably makes evaluative choices—for example, emphasizing medical over social information about disability in prenatal testing (Vehmas 2001); preference for certain outcomes is built into the system—for example, abortion of affected fetuses as society moves to routine prenatal screening (Bennett 2001); and nondirectiveness can be taken to an extreme where counselors are hesitant to challenge clients’ decisions no matter how inconsistent with their expressed values or irrational they appear (Singer 1996). Many difficult ethical questions confront genetic counselors and other clinicians involved in genetic testing in particular cases. Increased knowledge is not an unmitigated good: denial may be a coping mechanism; individuals may feel guilty for passing on harmful mutations to their offspring or stigmatized as having the potential to do so; survivor guilt may arise in a person who finds out she is not at risk for HD after all, or she may become at a loss about how to live her life differently; another person who finds out he is destined to develop HD or early-onset Alzheimer's may become depressed or even suicidal; paternity may not be what it is assumed to be. One person's decision to be tested may also have ramifications for others: for tests that rely on linkage analysis, there may be pressure on relatives to be tested; test results may have implications for the risk status of yet-untested family members; protecting confidentiality may deprive a spouse or child of important information. These issues will be tackled in the next two sections (“Genetic Discrimination” and “Genetic Privacy”). Prenatal genetic testing raises serious ethical questions about reproductive rights and eugenics. Reproductive rights are no longer just about the right not to have a child (to use contraception, to have an abortion) or the right to bear a child (to refuse population control measures). Reproductive rights have come to encompass the right to access technological assistance to procreate and to have a certain kind of child (Callahan 1998).[46] The specter of eugenics—and its images of involuntary sterilizations, immigration quotas, “fitter family” contests, and Nazi death camps—reappears once choices are being made about what sort of people are worth bringing into the world. Many authors have appealed to the history of eugenics to provide warnings about the dangers of genetic testing or to urge caution as we move forward.[47] Historian Diane Paul (1994b) characterizes eugenics as the “‘approved’ project anxiety”: given the lapse of time, we inevitably look good compared to earlier generations (p. 143). Paul notes that attempts to draw lessons from the history of eugenics are confounded by disagreements about how to define “eugenics”—whether to characterize eugenics according to a program's intentions or effects, its use of coercive rather than voluntary means, or its appeals to social and political aims that extend beyond the immediate concerns of individual families. For some commentators, even exhortations to choose the “best child” are not eugenic because they occur in the realm of “private enterprise” and are not directed at improving the population (Savulescu 2001). Others worry that the private reproductive choices of individuals will have the “backdoor” effect of narrowing society's acceptance of diversity (Duster 1990). We are also reminded that valuing reproductive autonomy means supporting a woman's decision not just to abort but to bear a child likely to have a disease or disability: while she may be free of state interference, many other factors make this choice difficult—pressure from clinicians, refusal of health insurers to cover the child when born due to the “pre-existing condition,” disapproval from other health insurance plan members, ableist attitudes in society, lack of financial resources, lack of social support (Hubbard 1990; Hubbard and Wald 1993). Arguably, if intentions matter, prenatal diagnosis means that eugenics is already with us: “prenatal diagnosis necessarily involves systematic and systemic selection of fetuses, mostly frequently on genetic grounds” (Lipmann 1991, p. 24). Indeed, Kitcher (1996) seeks to establish a sound theoretical basis for eugenic practice, envisioning a “laissez-faire” eugenics where people are educated to make “responsible” procreative decisions based on objective evaluative judgments of quality of life, not just for the prospective individual but all others involved—parents, siblings, and since funds for social programs are limited, all members of society.[48] Insofar as “prenatal diagnosis presupposes that certain fetal conditions are intrinsically not bearable” (Lipmann 1991, p. 25), we operate within the realm of “negative” eugenics—the selection against unwelcome traits associated with disease and disability. But the genetic testing of fetuses and embryos has already moved beyond this. Selective abortion and implantation decisions are made to obtain a child who is the correct gender or a prospective tissue donor (Kahn and Mastroianni 2004). A brief internet search reveals that selection of donor sperm or ova goes even further along “positive” eugenic lines (eye and hair color, race and ethnicity, SAT scores, height, musical ability, etc.); this demonstrates the central role consumer demand will play in any new eugenics, calling to mind Robert Nozick's (1974) famous image of a “genetic supermarket” (p. 315n). There is extensive ethical debate about of what “responsible” procreative decision-making consists in the context of genetic testing and selective abortion/implantation. It is argued that parents should prevent children being born with conditions that will cause severe pain and suffering (Brody 2002), that parental decisions should not restrict the autonomy of children-to-be by compromising their right to an open future with as few limitations as possible (Davis 2001), that parents should implant embryos without genes associated with disease and disability to avoid doing harm (Harris 2001), and even that parents should select the “best child”—ostensibly the “most intelligent”—of those they could possibly have (Savulescu 2001). It is also argued that responsible parenting involves an unconditional acceptance inconsistent with the willingness to commit to having only a certain kind of child (Vehmas 2001), and that it should not be assumed that being born with a disability need be detrimental “either to an individual's prospects of leading a worthwhile life, or to the families in which they grow up, or to society at large” (Parens and Asch 1999). Often embedded in these debates is the conundrum that complicates wrongful birth and wrongful life lawsuits: under what conditions can we say an individual has been wronged in the context of genetic testing and selective abortion/implantation when that individual would not otherwise have existed (Brock 1995)? A condition would have to be severe indeed for nonexistence to be preferable to existence with the condition. This provides leeway to argue that parents do no wrong when they refuse prenatal testing/selective abortion or PGD/selective implantation, or even make use of these technologies to choose children who, like themselves, will have congenital deafness or dwarfism.[49] 2.2.3 Genetic discrimination People frequently express fears about genetic discrimination resulting from genetic testing made possible by the HGP. Most concerns have focused on insurance companies and employers, but as the use of genetic information proliferates, one can readily imagine other institutions in society developing an interest in discriminating among individuals on the basis of such information: schools, departments of motor vehicles, immigration authorities, creditors, adoption agencies (Nelkin 1992; Nelkin and Tancredi 1989). A number of general arguments have been made against institutional forms of genetic discrimination: we don’t choose our genes and ought not be punished for what is outside our control (Gostin 1991); the social costs of creating a “biologic” or “genetic underclass” of people who lack health care and are unemployed or stuck in low-wage jobs are too great (Lee 1993; Nelkin and Tancredi 1989); people's fears of genetic discrimination, whether realistic or not, may lead them to forego genetic testing that might benefit their lives and be less inclined to participate in genetic research (Kass 1997); people have the right not to know their genetic risk status (Kass 1997). Genetic discrimination may also occur in less formal circumstances. Mate choice could increasingly proceed on the basis of genetic information, with certain people being labeled as undesirable. As more and more fetuses are aborted on genetic grounds, families of children born with similar conditions, and people with disabilities and their advocates more broadly, worry that increased stigmatization will result. One recommendation to address this concern is that society commit funds toward maintaining social support services for those with disabilities even as the numbers of abortions of genetically abnormal fetuses climb (Kitcher 1996). In addition, group-based genetic research into diseases or behavioral differences risks stigmatizing people based on racial, ethnic, and gender differences. It has been recommended that society invest in public education to combat any racial prejudice behavioral genetic research might unfurl (Kitcher 1996); arguments have also been advanced that such research either should not be done or should be held to more demanding standards than is frequently the case (Kitcher 2001). Focused ethical debate has taken place regarding both insurer- and employer-based genetic discrimination. Insurance companies discriminate against applicants for health, disability, life, or even mortgage policies when genetic testing reveals them or a family member to be at risk of disease or disability. Discrimination takes the form of refusing coverage on the basis that the genetic susceptibility counts as a “preexisting condition,” charging high premiums for the policy, limiting benefits, or excluding certain conditions. The insurance industry argues that there is no principled reason to treat genetic information any differently from other medical information used in underwriting. They point to the problem of “adverse selection”: people who know themselves to be at high risk are more likely to seek insurance than people who know themselves to be at low risk, which threatens the market when insurers are deprived of the same information.[50] If insurers are prohibited from soliciting genetic information altogether, they may decide to offer high-cost policies for those diseases for which testing is available under the assumption that all applicants are high risk. After all, insurers are in the very business of discriminating among individuals: arguably, fair and equitable underwriting requires that those belonging to the same risk class be treated the same (Meyer 2004; Pokorski 1994). In the U.S., where unlike other industrialized countries there is no publicly-funded system of universal health care, genetic discrimination is a particularly serious worry. The 1993 report of the ELSI Working Group's Task Force on Genetic Information and Insurance recommends against the use of genetic information as a basis for denying health care coverage or care and determining the cost of basic coverage, and calls for a voluntary moratorium on medical underwriting by insurance companies until a system of universal health care is established.[51] Kitcher's major policy recommendation in his 1996 The Lives to Come is for a two-tiered health insurance system that provides universal basic coverage regardless of genetic profile or economic status, whether provided by a public health care system funded through progressive taxation or a private system committed to affordable premiums based on ability to pay. Norman Daniels (1994, 2004) responds to the contention of insurers that it is their business to discriminate by drawing a distinction between “actuarial fairness” and moral fairness and arguing that underwriting health insurance is unjust given health care's social function of protecting normal functioning to ensure equality of opportunity. The case for not permitting the use of genetic information in underwriting life insurance has been found morally less compelling by Daniels (2004), but access is actually more threatened given that the individual market makes up 10–15 percent of private health insurance policies but 71 percent of life insurance policies (Kass 1997). In fact, outside the U.S., where universal health care is available, it is life and other forms of insurance which have been at the forefront of debate, with philosophical arguments often appealing to the value of social solidarity in support of pooling risk (Launis 2003). One proposal is to prohibit underwriting for life insurance policies below a certain amount (Kass 1997). Some genetic discrimination by employers is connected to health care coverage, particularly in the U.S. given the lack of universal access: to lower insurance costs, existing or prospective employees found to be at genetic risk may be fired or not hired. Other reasons employers may be interested in genetic information include reducing replacement and retraining costs, avoiding interruptions in production, and learning which workers are genetically susceptible or resistant to toxins in the work environment (MacDonald and Williams-Jones 2002). Already, in the 1960s-70s, companies like DuPont and Dow Chemical were engaged in genetic testing, both screening for susceptibilities and monitoring for damage (Draper 1991). Proponents of genetic testing by employers contend that this is an effective form of preventive medicine and that critics are technophobic or politically motivated. Opponents are concerned about lack of empirical validation of tests and inferences, capacity for informed consent, confidentiality, that focusing on “high-risk workers” instead of workplaces diverts attention away from environmental harms and may lead to the relaxation of precautions, and that the institution of genetic testing programs disproportionately affects groups (women, minorities) already marginalized in the workplace (Draper 1991; Hubbard and Wald 1993; Nelkin and Tancredi 1989). A familiar example is the use of sickle-cell carrier screening to exclude African Americans from certain jobs (Duster 1990). Although there is controversy about how extensive genetic discrimination is (Wertz [2002], for example, questions whether fears are “overblown”), a number of documented cases involving insurers and employers give cause for concern (Billings et al. 1992). The HMO for a family who already had a child with CF was willing to pay for the woman's amniocentesis and, if necessary, abortion but refused health care coverage for the prospective child if born with the condition (Thompson 1989). An air transportation company asked a genetic screening program in Canada to test their employee for HD without his consent (Huggins et al. 1990). In 2002, the Burlington Northern Santa Fe Railway Company (BNSF) was ordered to pay up to $2.2 million to employees whom they had secretly tested for a genetic variation supposed to be associated with carpal tunnel syndrome. Incredibly enough, one of the HGP's own labs—the DOE's Lawrence Berkeley Laboratory—was discovered to be testing the sickle cell status of African American employees without their consent using blood samples submitted during their annual physicals. The problem of genetic discrimination has received extensive attention from ELSI scholars, activists, politicians, and the public alike. Finally, in May 2008, the Genetic Information Nondiscrimination Act (GINA) was passed which prohibits U.S. insurance companies and employers from discriminating on the basis of information derived from genetic tests.[52] There have been criticisms of this legislative approach, however, and not only from insurers and employers. The criticisms are directed at misconceptions underlying the singling out of genetic information for protection: “These misconceptions include the presumption that a clear distinction exists between genetic and nongenetic information, tests, and diseases and the genetic essentialist belief that genetic information is more definitive, has greater predictive value, and is a greater threat to our privacy than is nongenetic medical information” (Beckwith and Alper 1998, p. 208; see also Rothstein 2005). This issue of “genetic exceptionalism” is taken up in the next section (“Genetic Privacy”). 2.2.4 Genetic Privacy Ethical debates over genetic privacy have been closely tied to worries about genetic discrimination by insurers and employers who have third-party access to genetic information from tests and research. Although patient confidentiality and genetic privacy are protected within the physician-patient relationship by professional codes of ethics, this protection is threatened by new information technologies and the cost-containment provisions of managed care (Orentlicher 1997). Employers have access to extensive health information about their employees, with limited legal protection against divulgence of this information (Rothstein 1997). Someone who undergoes genetic testing as a research subject may be forced to divulge this when applying for insurance (Clayton 1997a). Watson and other HGP proponents seemed aware from the outset that failure to address the problem of genetic discrimination would threaten the success of the project. Like many people, Watson advanced the legal protection of genetic privacy as the solution: “I think that somehow we have to get it into the laws that anyone's DNA—the message it gives—is confidential and that the only one who has a right to look at it is the person herself or himself” (1992, p. 172).[53] The strategy adopted in the U.S. to counter genetic discrimination through legislation has been to treat genetic information as inherently private and different from other medical information.[54] The Genetic Privacy Act, proposed by George Annas, Leonard Glantz, and Patricia Roche and designed to prevent the unauthorized disclosure of genetic information, considers genetic information to be “uniquely powerful and uniquely personal,” and therefore deserving of “unique privacy protection” under the law (Annas et al. 1995). Various reasons why genetic information should be regarded as unique or special are offered: genetic information is considered fundamentally important to personal identity (Andrews 1997); people may opt to have genetic testing without considering the effects of such knowledge (Andrews 1997); access to and control of genetic information make it possible for others to have power over a person's life (Annas 1994); genetic information provides information about an individual's family members (Annas 1994; Andrews 1997); genetic information serves as a “future diary” (Annas 1993); the ease with which DNA testing can be carried out makes possible its surreptitious use—to ascertain paternity, for example (Richards 2001). The approach is challenged by bioethicist Thomas Murray, who dubs it “genetic exceptionalism” (1997, p. 61). Murray argues that genetic information is not unique or even distinctive for a number of reasons: many diseases cannot be classified as genetic or nongenetic; genetic information can be gleaned from sources other than DNA; it is difficult to separate genetic and nongenetic information on medical records; genetic exceptionalism follows from, and in turn fosters, myths of genetic determinism and genetic reductionism.[55] Quite aside from this conceptual debate, there are also practical problems. If genetic privacy laws allow people the freedom to divulge their status “voluntarily” as they choose, insurers can offer low premiums as an incentive to applicants who agree to submit to testing and are found to be at low risk, and employers could hire only those applicants willing to undergo testing, just as pre-employment medical exams and drug screens are carried out now. Legal philosopher Anita Allen (1997) identifies four dimensions of genetic privacy: informational privacy, decisional privacy, physical privacy, and proprietary privacy. These provide a suitable framework for discussing various ethical concerns about genetic privacy. The worries about genetic discrimination and third-party access to genetic information already discussed are included in informational privacy (also associated with confidentiality, secrecy, and anonymity). So are other problems of third-party access which arise closer to home. A person's test results often have implications for family members who have not undergone testing, and despite the sacrifice of confidentiality involved, some geneticists call for family members to be informed when they are at risk of a serious disease which is preventable or treatable (see Rhodes 2000). Genetic counselors confront dilemmas about keeping or revealing secrets, for example, involving the diagnosis of a child's condition or misattributed paternity that modifies genetic risk information within families (Biesecker 1997). Decisional privacy protects people's ability to make autonomous choices. Decisional privacy is implicated when people who do not wish to know if they are at risk for a certain disease face insurance- or employment-based testing or population-wide screens, or perhaps feel pressured by relatives to be tested in order to locate a familial mutation. For example, in the case of HD, given that no treatment at all exists, many at-risk individuals choose not to avail themselves of testing (World Federation 1993). Bioethical debate over whether individuals have a right to remain ignorant about their genetic make-up centers on the requirements of autonomy: it is alternatively argued that autonomy is advanced when a person's refusal to be tested is respected (Takala 1999, Takala and Hayry 2000); that there is a duty to know genetic information that could affect decision-making, but given possible social consequences, the decision not to know rests with individuals (Rhodes 1998, 2000); and that the failure to consider all information relevant to making future choices undermines our ability to be rational, self-governing agents (Harris and Keywood 2001). Decisional privacy and the requirements of autonomy are also at issue in the debate over the legitimacy of parental consent for the genetic testing of minors for late-onset conditions (Clayton 1997b; Cohen 1998; Sevick et al. 2005). Physical (as well as decisional) privacy becomes an issue when genetic tests or screens are performed without a person's voluntary, informed consent. This occurs, for example, when researchers use stored DNA samples for purposes that go beyond those to which their donors provided consent. For example, many countries have decades of stored “Guthrie cards” with newborn blood samples provided for metabolic screening. In the U.S., concern has been raised about the paucity of regulations regarding third-party access to these as DNA “banks” for research (McEwen and Reilly 1994); in Australia, interest in correlating health records with 30 years of blood test results provided by newborn screening has resulted in consultation over the use and storage of genetic samples and information (Ankeny 2003). Proprietary privacy is involved should a person's genetic material or information be appropriated for economic purposes. Moore v. Regents of the University of California is a well-known litigation case involving proprietary (as well as decisional and physical) privacy. In the course of his treatment for leukemia at the UCLA Medical Center, John Moore's spleen was removed, and then, without his knowledge or consent, used to develop a commercially lucrative cell line. The court ruled that Moore's right to informed consent was breached, but that he had no property rights over the cell line. Philosophers have approached this question about property rights over DNA from Lockean and Rawlsian perspectives on justice (Farrelly 2002; Moore 2000). As DNA banking and DNA data banking efforts proliferate with few laws to restrict the future use of samples or prevent the transfer of data, and given that any blood or tissue collection is a potential DNA bank, people may lose any hope of keeping their genetic profile private. The U.S. Department of Defense has the world's largest DNA bank: all military personnel are required to provide blood and saliva samples so that the remains of missing soldiers can be identified (McEwen 1997). The U.K.'s forensic DNA bank will eventually include a substantial portion of its population: the 2003 Criminal Justice Act requires anyone arrested for—not necessarily charged with, much less convicted of—a recordable offense to submit a DNA sample[56]; in the U.S. such requirements vary state-by-state (McEwen 1997). As the technology becomes cheaper, the proposal to create a national DNA data bank by sampling every newborn child could arise (Bereano 1992). This repository of data would serve a number of purposes in addition to forensic ones: provide an identity tag (like social security numbers); enable economic planning based on genetic risk factors; support research into the genetic basis of medical and behavioral traits; identify remains; identify children recovered after being kidnapped; monitor individuals presumed to be predisposed to criminal violence; identify potential transplant donors; etc. A number of countries have already created population-level DNA banks and data banks to aid in research into complex diseases (Kaiser 2002), with deCODE Genetics’ initiative in Iceland, which combines genetic, medical, and genealogical data and operates on the basis of presumed consent, the most well-known. On a smaller scale, DNA is also banked by researchers in academic institutions, commercial labs, and hospitals. Additional privacy concerns arise in research contexts where DNA profiles are linked to phenotypic features (clinical data and lifestyle information) and not used just for forensic identification. Bioethicists looking at “biobanks” have focused on the problems prospective research poses for consent, and the adequacy of such provisions as opting in or opting out, waiving or giving blanket consent, withdrawing or renewing consent, etc. (Helgesson and Johnsson 2005; Shickle 2006). Onora O’Neill (2001) questions whether informed consent even remains a realistic goal given the amount of genetic data being banked. She argues that consent to specific propositions cannot be extended to closely related ones, and that the lack of explicit dissent cannot be taken as tacit or implied consent. O’Neill reminds us that consent to the collection, storage, and use of genetic and other data is not sought within the confines of the doctor-patient relationship where privacy is governed by professional standards; rather, the relationship is many-many and privacy is dependent on methods of data protection for which public rather than individual consent becomes appropriate. However, as genetic sequencing becomes a routine part of health care, with all the information researchers need contained in patient records, similar concerns about privacy will arise in the health care sector (Hansson 2004). 2.2.5 Genetic Modification (GM) The HGP will assist scientists in the identification of genes and their functions. As scientists gain knowledge about how various genes contribute to phenotype, genetic modification will become possible. Such interventions could involve knocking out a gene with a detrimental role, or inserting a gene with a beneficial role, though because of gene-gene and gene-environment interactions (as discussed in section 2.1.4.), it will be difficult to predict the effects of many such interventions. The first approved GM procedure was performed in 1990 when NIH scientists genetically modified white blood cells in Ashanti DeSilva, a four-year-old girl with adenosine deaminase deficiency (ADA). Over 1000 clinical trials have since been performed (Rasko et al. 2006, Gene Therapy Clinical Trials Worldwide Database), with studies carried out on numerous diseases, including CF, Franconi's anemia, muscular dystrophy, and hemophilia. A serious setback occurred when 18-year-old Jesse Gelsinger died in a University of Pennsylvania gene therapy trial for ornithine transcarbamylase (OTC) deficiency in 1999. Shortly after, a French research team successfully restored the immune systems of young children with severe combined immunodeficiency (SCID) through gene therapy; this success was tempered, however, when several of the children later developed leukemia as a result of the procedure. Given these problems with safety and efficacy, gene therapy trials have shifted from “single-gene” diseases to conditions like cancer, cardiovascular disease, and rheumatoid arthritis, where vector applications are more local and transient (Branca 2005). Procedures carried out to date involve the GM of somatic cells, but GM could potentially be applied to gametes or IVF embryos, resulting in the genetic changes being passed on to future generations: this is referred to as germ-line GM, or more recently (Rasko et al. 2006), “inheritable genetic modification.” Some bioethicists argue that the distinction between somatic cell and germ-line GM has moral significance, and that somatic cell interventions alone are permissible. A number of arguments are offered against germ-line GM: the long-term effects are uncertain; germ-line interventions are eugenic given that future generations are not patients; only existing persons can provide informed consent (Lappé 1991); humans have a basic right to be born with an unmodified genome (Mauron and Thevoz 1991); some modifications alter personal identity (Zohar 1991); and human trials face daunting ethical and practical challenges (Dresser 2004). Bioethicists who support germ-line GM counter these arguments with others: parents have reproductive autonomy (Zimmerman 1991); the fetus is being treated as a patient in prenatal surgeries, so why not the embryo (Nolan 1991); we constantly modify the conditions future generations inherit, and usually not with altruistic purposes (e.g., degraded environments) (Peters 1995); members of future generations would presumably consent to an augmentation of Rawlsian primary goods (Allhoff 2005); modifications could be reversed if harmful effects arise (Moseley 1991); the germ line might be the only or most effective level for intervention in some diseases (Munson and Davis 1992; Zimmerman 1991); germ-line interventions would avoid subsequent generations of a family having to undergo somatic cell GM or worry about carrier status (Munson and Davis 1992). The distinction between therapy and enhancement, as it applies to both somatic cell and germ-line GM, is also considered by some bioethicists to have moral significance: interventions that restore normal function from a state of disease or disability (negative GM) are considered permissible whereas interventions that improve on normal function (positive GM) are not.[57] Arguments against genetic enhancement include: clinical risks outweigh possible benefits (Anderson 1990); it violates parents’ fiduciary responsibilities (Schonfeld 2003); genetic diversity will be diminished (Parens 1995); the socially acceptable range of traits will be narrowed (Agar 1995); the principle of equality of opportunity (Daniels 1994) and the Rawlsian imperative to help the worst off (Fleck 1994) support negative but not positive GM; appreciation of shared human “fragility” and the accomplishments of individuals may diminish (Parens 1995). A further distinction can be drawn between enhancements that go beyond species typicality to improve health (e.g. disease-resistant genes) and enhancements of social value (e.g. eye color), and it is argued that there is no moral difference between using GM to augment function and using GM to restore function if the aim is disease prevention (Harris 1993; Juengst 1997). Some bioethicists who support GM for enhancement as well as therapeutic purposes counter these arguments by asserting the reproductive autonomy of parents (Stock 2003). Others point to the lack of an objective and/or morally justified distinction between therapy and enhancement, health and disease (Goering 2000; Resnik 2000).[58] Indeed, with GM, we humans ourselves determine what counts as “normal” genetic variation, no longer needing to look to a pre-existing species-typical functional norm (Resnik 1997b). The moral permissibility of particular genetic interventions, therapies and enhancements alike, might instead be assessed according to Rawlsian principles—with equality of opportunity and concern for the worst off supporting the prohibition of some enhancements and the guarantee of a minimal level of genetic health (Resnik 1997b), or a permissible intervention one which, when contemplated in the original position behind a veil of ignorance, can be seen to prevent a condition that would be detrimental in all societies or bring about a condition that would never be detrimental in any society (Goering 2000). GM of the germ line for either therapeutic or enhancement purposes and GM of somatic cells for enhancement purposes have not yet been carried out in humans.[59] Of the four classes of interventions delineated by somatic-cell/germ-line and therapy/enhancement distinctions, only germ-line GM for enhancement purposes remains particularly contentious among bioethicists (Allhoff 2005). Of course, GM of the germ line of plants and nonhuman animals has become commonplace,[60] and some commentators argue that the use of GM for enhancement purposes in humans is inevitable (Baylis and Robert 2004; Stock 2003). After all, look at parents who seek every advantage for their child on route to an Ivy League education, athletes who use illegal performance-enhancing drugs, increased use of plastic surgery, recreational use of Viagra, etc. Genetic modification of the germ line might be even more likely to be embraced for positive/enhancement than negative/therapeutic purposes, since if IVF is being used, concerns about a particular disease can be allayed with PGD and selective implantation. However, as more and more conditions—whether diseases or not—can be screened for in embryos, distinctions between therapy and enhancement and enhancement for health vs. social purposes—already problematic conceptually, a situation compounded by the newfound plasticity of the species adaptive norm—may break down even further in practice. Already in 1969, Sinsheimer foresaw the promise of molecular biology to remake human nature: “For the first time in all time, a living creature understands its origin and can undertake to design its future” (in Kevles 1992, p. 18). Remaking human nature is likely to begin with genetic modifications that convey the possibility of resistance to a serious disease, like HIV/AIDS, or minimize the effects of aging to extend lifespan, but transhumanists who view human nature as “a work-in-progress, a half-baked beginning that we can learn to remold in desirable ways” welcome improvements in memory, intelligence, and emotional capacities as well (Bostrom 2003, p. 493). Widespread inheritable GM could lead to a “radical rupture” with evolutionary processes as we understand them today—undermining the distinction between acquired and inheritable traits, challenging our abilities to reconstruct genetic genealogies, and even creating a new hominid species (Baylis and Robert 2006). Words chosen for titles of recent books—for example, “the future of human nature” (Habermas 2003), “our posthuman future” (Fukuyama 2002), and “redesigning humans” (Stock 2003)—reveal that a serious debate is underway about the prospect of a genetically engineered human nature and what this means for all of us. Some are practically giddy about this “journey to destinations of new imagination” (Stock 2003, p. 1); others warn that it will undo the “intimate connection between human nature and human notions of rights, justice, and morality” (Fukuyama 2002, p. 101). 2.2.6 Justice and the genome The HGP was initially criticized on the basis that the government's support of such a large-scale project would take resources away from other programs arguably more important for health and wellbeing, such as, in the U.S., instituting a system of universal health care or funding prenatal care for poor women. Cures for disease arising from the HGP, after all, are likely to benefit future not existing people (Murphy 1994). These concerns were compounded by the worry that increased scientific attention paid to genetic factors implicated in health and disease would promote interventions at the level of the genome—genetic tests, genetic modifications, drug development based on gene sequences, etc.—and that the need to address the impacts of harmful physical and social environments due to toxins, poverty, racism, etc. would be ignored (Hubbard and Wald 1993). The problem of racial profiling in the legal system was raised: if members of particular racial and ethnic groups are more likely to be arrested, charged, or convicted of a criminal offense, they are more likely to be required to provide DNA samples to forensic databases, and therefore more likely to come back into the system with future offenses. One proposed solution is for a national database which includes everyone (Kitcher 1996), though it is also argued that this compromises the autonomy and dignity of individual citizens (Boylan 2002). And given the history of using biological explanations to provide ideological justification for social inequalities associated with oppressive power structures, the prospective use of molecular genetics to explain race and sex differences has also met with caution (Hubbard 1994). With completion of the HGP, concerns of justice increasingly focus on the distribution of benefits and burdens of genetic technologies, especially given that the HGP was funded by public monies. One of the “majestic horizons” identified by Clinton in June 2000 represented a call for justice—specifically, ethical respect for “our oldest and most cherished human values” to ensure that genome science benefits “all citizens of the world” and prevents discrimination. And yet, another of Clinton's “majestic horizons”—biotechnological development in the private sector based on the identification of all human genes and their functions—raises significant challenges for justice. While the HGP's vision of “individualized preventive medicine” will no doubt promote biotechnological development in the private sector, is this development likely to benefit “all citizens of the world”? Globally, in the developing world, infectious diseases are more of a priority than genetic diseases; in the U.S., millions of citizens and residents are without or lack adequate health care coverage for the routine preventive care and treatment of acute illnesses and injuries proponents of genomic medicine take for granted. Profits direct pharmaceutical research and development: efforts are expended on conditions that are relatively frequent and sometimes minor (e.g. male-pattern baldness [Ellis and Sinclair 2008]); conditions prevalent in affluent countries receive more attention than those prevalent in developing countries (Flory and Kitcher 2004); and populations in the developing world are vulnerable to exploitation (Dickenson 2004). “All citizens of the world” will not have access to the expensive new drugs: in the U.S., beneficiaries will be the wealthy and those with generous prescription drug plans; in the developing world, it remains a struggle to make drugs to treat serious diseases like HIV/AIDS affordable and accessible to all who need them. Concerns have arisen, not just that there will be unequal access to the benefits of genetic technologies, but that the use of these technologies will exacerbate existing patterns of social and economic inequality. As mentioned in section 2.2.3, genetic discrimination by insurers and employers could lead to creation of a “genetic underclass”; similarly, should socioeconomically advantaged families have greater access to prenatal genetic testing, an increased relative frequency of conditions like CF, sickle cell anemia, or Down syndrome would occur among less privileged families. To the extent that race and ethnicity correlate with socioeconomic status, some groups—already affected by disparities in health outcomes unrelated to genetic differences—will be disproportionately represented among this “genetic underclass.” If genetic enhancement becomes available and is used disproportionately by those belonging to advantaged groups, there is the potential for the socioeconomic gap to widen further—think of the diminished range of opportunities available to the genetically unenhanced Vincent in the film Gattaca. An entirely new class structure based on genetic makeup as remolded by culture is foreseen: some welcome humanity's fragmentation into “independent breeds—future human Saint Bernard and dachshund analogs” corresponding to the enhanced and the unenhanced, or family lineages where similar enhancement choices are made in successive generations (Stock 2003); others fear this imperils the continuation of liberal democratic society (Mehlman 2003; Mehlman and Botkin 1998). There are also those who see possibilities for increased equality. One reason is that environments could be tailored to the genetic predispositions of individuals based on better knowledge of gene-environment interactions (Buchanan et al. 2000). Another reason is that the genetic technologies on the horizon challenge the natural-social distinction by making the natural lottery, like the social lottery, alterable: the state could redistribute or narrow the range of differences in natural abilities as is often done for social resources in order to promote equality of opportunity or benefit the worst off (Buchanan et al. 2000; Resnik 1997b).[61] These concerns about differential access to the benefits of genetic technologies and the likely social consequences of such differential access have been addressed by bioethicists and moral philosophers from perspectives informed by competing theories of distributive justice. Libertarians support the “genetic supermarket” famously envisioned by Nozick, where prospective parents and resulting market forces determine which genetic modifications are offered—not centralized decisions made by biologists or government. It has been noted, however, that what is harmless or beneficial in individual cases (e.g. sex selection, increased longevity, competitive goods like beauty or intelligence) may be neutral or even harmful for society taken collectively (Kavka 1994). And yet, given that the state does not restrict parental autonomy when it comes to environmental boosts to their children's abilities (e.g. private schools, music lessons), it is argued that there is no principled distinction between genes and environments which would override a similar right to genetic enhancements (Buchanan et al. 2000). Liberals have used Rawlsian principles to urge a variety of state-imposed limits on a free market in genetic technologies (Brown 2001; Buchanan et al. 2000; Daniels 1994; Farrelly 2002; Resnik 1997b). In their 2000 book From Chance to Choice: Genetics and Justice, in order to protect equal opportunity, Allen Buchanan, Dan Brock, Norman Daniels, and Daniel Wikler argue that the state should ensure access to a “genetic decent minimum” which includes interventions directed to natural primary goods—whether treatments or enhancements—insofar as these are consistent with the neutrality of liberal democracies concerning conceptions of the good.[62] Parental choices would be limited only where these are inconsistent with a child's right to an open future, carry a risk to public good (e.g. sex selection resulting in societal imbalance), diminish equal opportunity if available only to the comparatively well off, or involve enhancements with “positional advantages” (e.g. height) that would be self-defeating if widely available and unfair—and contrary to Rawls’ difference principle which requires permissible inequalities to work to the advantage of all, especially the worst off—if not, and where enhancements would lead to the disabling of others by raising the bar for what is required to participate in society's “dominant cooperative scheme.”[63] Of particular interest in current philosophical debates about justice and the genome is the consensus which has emerged among participants that our usual ways of thinking about justice are themselves challenged by the new genetic technologies (Buchanan et al. 2000).[64] Theories of justice are typically based on conceptions of human nature, but with the capacity to remake human nature, this foundation disappears. Approaches to distributive justice seek to compensate for natural inequalities existing at birth among individuals of fixed identities, but considerations of justice are now implicated earlier, in decisions about the distribution of genetic characteristics, decisions which affect the identities of those born. The potential for genetic stratification places at risk the assumption central to all theorizing about justice—that we share a moral community.\", metadata={'source': 'https://plato.stanford.edu/entries/human-genome/', 'page_title': 'The Human Genome Project'}),\n",
       " Document(page_content='doi: 10.1096/fasebj.5.1.1991595. Affiliations Affiliation * 1 National Center for Human Genome Research, National Institutes of Health, Bethesda, Maryland 20892., * PMID: 1991595, * DOI: 10.1096/fasebj.5.1.1991595, J D Watson et al. FASEB J. 1991 Jan. doi: 10.1096/fasebj.5.1.1991595. Affiliation * 1 National Center for Human Genome Research, National Institutes of Health, Bethesda, Maryland 20892., * PMID: 1991595, * DOI: 10.1096/fasebj.5.1.1991595, Abstract The Human Genome Project has become a reality. Building on a debate that dates back to 1985, several genome projects are now in full stride around the world, and more are likely to form in the next several years. Italy began its genome program in 1987, and the United Kingdom and U.S.S.R. in 1988. The European communities mounted several genome projects on yeast, bacteria, Drosophila, and Arabidospis thaliana (a rapidly growing plant with a small genome) in 1988, and in 1990 commenced a new 2-year program on the human genome. In the United States, we have completed the first year of operation of the National Center for Human Genome Research at the National Institutes of Health (NIH), now the largest single funding source for genome research in the world. There have been dedicated budgets focused on genome-scale research at NIH, the U.S. Department of Energy, and the Howard Hughes Medical Institute for several years, and results are beginning to accumulate. There were three annual meetings on genome mapping and sequencing at Cold Spring Harbor, New York, in the spring of 1988, 1989, and 1990; the talks have shifted from a discussion about how to approach problems to presenting results from experiments already performed. We have finally begun to work rather than merely talk. The purpose of genome projects is to assemble data on the structure of DNA in human chromosomes and those of other organisms. A second goal is to develop new technologies to perform mapping and sequencing. There have been impressive technical advances in the past 5 years since the debate about the human genome project began. We are on the verge of beginning pilot projects to test several approaches to sequencing long stretches of DNA, using both automation and manual methods. Ordered sets of yeast artificial chromosome and cosmid clones have been assembled to span more than 2 million base pairs of several human chromosomes, and a region of 10 million base pairs has been assembled for Caenorhabditis elegans by a collaboration between Washington University and the Medical Research Council laboratory in Cambridge, U.K. This project is now turning to sequencing C. elegans DNA as a logical extension of this work. These are but the first fruits of the genome project. There is much more to come. Similar articles * The Bermuda Triangle: The Pragmatics, Policies, and Principles for Data Sharing in the History of the Human Genome Project. Maxson Jones K, Ankeny RA, Cook-Deegan R. Maxson Jones K, et al. J Hist Biol. 2018 Dec;51(4):693-805. doi: 10.1007/s10739-018-9538-7. J Hist Biol. 2018. PMID: 30390178 Free PMC article., * Major events in the U.S. Human Genome Project and related projects. U.S. Department of Energy Office of Science, Office of Biological and Environmental Research. U.S. Department of Energy Office of Science, Office of Biological and Environmental Research. Md Med. 2000 Autumn;1(4):16-9. Md Med. 2000. PMID: 11147452 No abstract available., * Collins takes helm of NIH genome project. Blume E. Blume E. J Natl Cancer Inst. 1993 May 5;85(9):694-6. doi: 10.1093/jnci/85.9.694. J Natl Cancer Inst. 1993. PMID: 8478950 No abstract available., * The Human Genome Project. History, goals, and progress to date. Engel LW. Engel LW. Arch Pathol Lab Med. 1993 May;117(5):459-65. Arch Pathol Lab Med. 1993. PMID: 8489331 Review., * The origins of the National Cancer Institute. Yaremchuk WA. Yaremchuk WA. J Natl Cancer Inst. 1977 Aug;59(2 Suppl):551-8. J Natl Cancer Inst. 1977. PMID: 328902 Review. No abstract available., Cited by * The International Human Genome Project. Birney E. Birney E. Hum Mol Genet. 2021 Oct 1;30(R2):R161-R163. doi: 10.1093/hmg/ddab198. Hum Mol Genet. 2021. PMID: 34264324 Free PMC article. Review., * Integrated Analysis of Whole Genome and Epigenome Data Using Machine Learning Technology: Toward the Establishment of Precision Oncology. Asada K, Kaneko S, Takasawa K, Machino H, Takahashi S, Shinkai N, Shimoyama R, Komatsu M, Hamamoto R. Asada K, et al. Front Oncol. 2021 May 12;11:666937. doi: 10.3389/fonc.2021.666937. eCollection 2021. Front Oncol. 2021. PMID: 34055633 Free PMC article. Review., * Multisite de novo mutations in human offspring after paternal exposure to ionizing radiation. Holtgrewe M, Knaus A, Hildebrand G, Pantel JT, Santos MRL, Neveling K, Goldmann J, Schubach M, Jäger M, Coutelier M, Mundlos S, Beule D, Sperling K, Krawitz PM. Holtgrewe M, et al. Sci Rep. 2018 Oct 2;8(1):14611. doi: 10.1038/s41598-018-33066-x. Sci Rep. 2018. PMID: 30279461 Free PMC article., * Utilization of multiple \"omics\" studies in microbial pathogeny for microbiology insights. Wiwanitkit V. Wiwanitkit V. Asian Pac J Trop Biomed. 2013 Apr;3(4):330-3. doi: 10.1016/S2221-1691(13)60073-8. Asian Pac J Trop Biomed. 2013. PMID: 23620861 Free PMC article. Review., * Conditional gene manipulation: Cre-ating a new biological era. Zhang J, Zhao J, Jiang WJ, Shan XW, Yang XM, Gao JG. Zhang J, et al. J Zhejiang Univ Sci B. 2012 Jul;13(7):511-24. doi: 10.1631/jzus.B1200042. J Zhejiang Univ Sci B. 2012. PMID: 22761243 Free PMC article. Review.,', metadata={'source': 'https://pubmed.ncbi.nlm.nih.gov/1991595/', 'page_title': 'Origins of the Human Genome Project - PubMed'}),\n",
       " Document(page_content='The Human Genome Project (HGP) refers to the international 13-year effort, formally begun in October 1990 and completed in 2003, to discover all the estimated 20,000-25,000 human genes and make them accessible for further biological study. Another project goal was to determine the complete sequence of the 3 billion DNA subunits (bases in the human genome). As part of the HGP, parallel studies were carried out on selected model organisms such as the bacterium E. coli and the mouse to help develop the technology and interpret human gene function. The DOE Human Genome Program and the NIH National Human Genome Research Institute (NHGRI) together sponsored the U.S. Human Genome Project. History of the Project * HGP Goals and Corresponding Completion Dates, * Budget History of the U.S. Human Genome Project, * Timeline: Major Events in the Human Genome Project, Publications Summarizing Various Aspects of the Project * Special issue of Nature Human Genome Collection (2006), * Special issue of Science: Building on the DNA Revolution (April 11, 2003) \"The Human Genome Project: Lessons from Large-Scale Biology,\" Francis S. Collins, Michael Morgan, Aristides Patrinos, Science 300, 286 (2003) \"Realizing the Potential of the Genome Revolution: The Genomes to Life Program,\" Marvin E. Frazier, Gary M. Johnson, David G. Thomassen, Carl E. Oliver, Aristides Patrinos, Science 300, 290 (2003), * Nature Genetics: A 10-Year Retrospective 1992-2002 (vol. 33, March 2003), * Controversial From the Start —Science article summarizing the history of the HGP (February 2001), * Genomes: 15 Years Later—A perspective from Charles DeLisi, HGP Pioneer (July 2001), * 1997 Human Genome Program Report contains history of the Project, * Bermuda Conference Data Release Policies (1997, 1996). See also, NHGRI Policy Regarding Intellectual Property of Human Genomic Sequence (April 1996)., * NCHGR-DOE Guidance on Human Subjects Issues in Large-Scale DNA Sequencing (1996), * Special Anniversary Issue of Human Genome News (7(3-4); Sept.-Dec. 1995) Summarizing the History and Progress of the Project, * Evolution of a Vision (Part I) by David Smith, then Director of the DOE HGP (December 1995), * Evolution of a Vision (Part II) by Francis S. Collins, then Director of NIH NCHGR (December 1995), * \"Origins of the Human Genome Project,\" by Robert Cook-Deegan (1994; Risk Journal), * Mapping the Genome: The Vision, the Science, the Implementation; What is the Genome Project? [Article from Los Alamos Science. A round table discussion with David Baltimore, David Botstein, David R. Cox, David J. Galas, Leroy Hood, Robert K. Moyzis, Maynard V. Olson, Nancy S. Wexler, and Norton D. Ziner] Los Alamos (National Laboratory) Science 20, 1992., * History of the Department of Energy Human Genome Program adapted from the U.S. DOE 1991-92 Human Genome Program Report (published June 1992), * Data Sharing Policy: (1992) A U.S. Department of Energy and National Institutes of Health Coordinated Effort, * Understanding our Genetic Inheritance. The U.S. Human Genome Project: The First Five Years FY 1991-1995. Report DOE/ER-0452P. (published April 1990), * Mapping Our Genes: Genome Projects —How Big? How Fast? 1988 report from the U.S. Congress Office of Technology Assessment, * \"Orchestrating the Human Genome Project,\" by Charles Cantor, Science 248, April 1990, * \"The Human Genome Project: Past, Present, and Future\" by J.D. Watson, Science 248, April 1990, * \"The Department of Energy (DOE) Human Genome Initiative,\" Benjamin J. Barnhart, Genomics 5, 657 (1989)., * \"The (May 1985) Santa Cruz Workshop,\" R.L. Sinsheimer, Genomics 5, 954 (1989)., * Mapping Our Genes: Genome Projects —How Big? How Fast? 1988 report from the U.S. Congress Office of Technology Assessment, * Mapping and Sequencing the Human Genome, report from the National Research Council Commission on Life Sciences, National Academy Press, Washington, DC, 1988, * The Genome Project, Dec. 13, 1987, NYT Magazine article [], * Report on the Human Genome Initiative for the Office of Health and Environmental Research: April 1987 report that officially outlined the Department of Energy\\'s strategies for the Human Genome Project, * Summary Report of the 1986 Santa Fe Workshop, \"Sequencing the Human Genome\", Bitensky, M.; Los Alamos National Laboratory, Los Alamos, NM. See also Nature: Meetings that changed the world: Santa Fe 1986, (Oct. 16, 2008.), * \"The Alta Summit, December 1984,\" by Robert Cook-Deegan, Genomics 5, 661-663 (published October 1989): The beginning of the Human Genome Project., * Human Genome News (HGN) newsletter on the Human Genome Project. All published issues (since 1989) are available. See also the archive of HGN History and Project Management articles., * DOE HGP Reports and Workshop Abstracts report on program research., * More publications, Project Enabling Legislation * The Atomic Energy Act of 1946 (P.L. 79-585) provided the initial charter for a comprehensive program of research and development related to the utilization of fissionable and radioactive materials for medical, biological, and health purposes., * The Atomic Energy Act of 1954 (P.L. 83-706) further authorized the AEC \"to conduct research on the biologic effects of ionizing radiation.\", * The Energy Reorganization Act of 1974 (P.L. 93-438) provided that responsibilities of the Energy Research and Development Administration (ERDA) shall include \"engaging in and supporting environmental, biomedical, physical, and safety research related to the development of energy resources and utilization technologies.\", * The Federal Non-nuclear Energy Research and Development Act of 1974 (P.L. 93-577) authorized ERDA to conduct a comprehensive non-nuclear energy research, development, and demonstration program to include the environmental and social consequences of the various technologies., * The DOE Organization Act of 1977 (P.L. 95-91) mandated the Department \"to assure incorporation of national environmental protection goals in the formulation and implementation of energy programs; and to advance the goal of restoring, protecting, and enhancing environmental quality, and assuring public health and safety,\" and to conduct \"a comprehensive program of research and development on the environmental effects of energy technology and program.\", Project Sponsors * The U.S. Department of Energy funded its Human Genome Program through their Office of Biological and Environmental Research. (genome@science.doe.gov)., * The U.S. National Institutes of Health funded its program through the National Human Genome Research Institute (NHGRI)., Participating Institutions * List of institutions where much of the U.S.-funded research was done.,', metadata={'source': 'https://web.ornl.gov/sci/techresources/Human_Genome/project/hgp.shtml', 'page_title': 'History of the Human Genome Project'}),\n",
       " Document(page_content=\"Human Genome Project (HGP), an international collaboration that successfully determined, stored, and rendered publicly available the sequences of almost all the genetic content of the chromosomes of the human organism, otherwise known as the human genome.The Human Genome Project (HGP), which operated from 1990 to 2003, provided researchers with basic information about the sequences of the three billion chemical base pairs (i.e., adenine [A], thymine [T], guanine [G], and cytosine [C]) that make up human genomic DNA (deoxyribonucleic acid). The HGP was further intended to improve the technologies needed to interpret and analyze genomic sequences, to identify all the genes encoded in human DNA, and to address the ethical, legal, and social implications that might arise from defining the entire human genomic sequence. Timeline of the HGP Learn about Watson and Crick's double-helix DNA structure, composed of two intertwined chains of nucleotides resembling a spiral ladderSee all videos for this articlePrior to the HGP, the base sequences of numerous human genes had been determined through contributions made by many individual scientists. However, the vast majority of the human genome remained unexplored, and researchers, having recognized the necessity and value of having at hand the basic information of the human genomic sequence, were beginning to search for ways to uncover this information more quickly. Because the HGP required billions of dollars that would inevitably be taken away from traditional biomedical research, many scientists, politicians, and ethicists became involved in vigorous debates over the merits, risks, and relative costs of sequencing the entire human genome in one concerted undertaking. Despite the controversy, the HGP was initiated in 1990 under the leadership of American geneticist Francis Collins, with support from the U.S. Department of Energy and the National Institutes of Health (NIH). The effort was soon joined by scientists from around the world. Moreover, a series of technical advances in the sequencing process itself and in the computer hardware and software used to track and analyze the resulting data enabled rapid progress of the project. Technological advance, however, was only one of the forces driving the pace of discovery of the HGP. In 1998 a private-sector enterprise, Celera Genomics, headed by American biochemist and former NIH scientist J. Craig Venter, began to compete with and potentially undermine the publicly funded HGP. At the heart of the competition was the prospect of gaining control over potential patents on the genome sequence, which was considered a pharmaceutical treasure trove. Although the legal and financial reasons remain unclear, the rivalry between Celera and the NIH ended when they joined forces, thus speeding completion of the rough draft sequence of the human genome. The completion of the rough draft was announced in June 2000 by Collins and Venter. For the next three years, the rough draft sequence was refined, extended, and further analyzed, and in April 2003, coinciding with the 50th anniversary of the publication that described the double-helical structure of DNA, written by British biophysicist Francis Crick and American geneticist and biophysicist James D. Watson, the HGP was declared complete. Britannica Quiz Genetics Quiz Science behind the HGP To appreciate the magnitude, challenge, and implications of the HGP, it is important first to consider the foundation of science upon which it was based—the fields of classical, molecular, and human genetics. Classical genetics is considered to have begun in the mid-1800s with the work of Austrian botanist, teacher, and Augustinian prelate Gregor Mendel, who defined the basic laws of genetics in his studies of the garden pea (Pisum sativum). Mendel succeeded in explaining that, for any given gene, offspring inherit from each parent one form, or allele, of a gene. In addition, the allele that an offspring inherits from a parent for one gene is independent of the allele inherited from that parent for another gene. Mendel’s basic laws of genetics were expanded upon in the early 20th century when molecular geneticists began conducting research using model organisms such as Drosophila melanogaster (also called the vinegar fly or fruit fly) that provided a more comprehensive view of the complexities of genetic transmission. For example, molecular genetics studies demonstrated that two alleles can be codominant (characteristics of both alleles of a gene are expressed) and that not all traits are defined by single genes; in fact, many traits reflect the combined influences of numerous genes. The field of molecular genetics emerged from the realization that DNA and RNA (ribonucleic acid) constitute the genetic material in all living things. In physical terms, a gene is a discrete stretch of nucleotides within a DNA molecule, with each nucleotide containing an A, G, T, or C base unit. It is the specific sequence of these bases that encodes the information contained in the gene and that is ultimately translated into a final product, a molecule of protein or in some cases a molecule of RNA. The protein or RNA product may have a structural role or a regulatory role, or it may serve as an enzyme to promote the formation or metabolism of other molecules, including carbohydrates and lipids. All these molecules work in concert to maintain the processes required for life.Get a Britannica Premium subscription and gain access to exclusive content. Subscribe Now Studies in molecular genetics led to studies in human genetics and the consideration of the ways in which traits in humans are inherited. For example, most traits in humans and other species result from a combination of genetic and environmental influences. In addition, some genes, such as those encoded at neighbouring spots on a single chromosome, tend to be inherited together, rather than independently, whereas other genes, namely those encoded on the mitochondrial genome, are inherited only from the mother, and yet other genes, encoded on the Y chromosome, are passed only from fathers to sons. Using data from the HGP, scientists have estimated that the human genome contains anywhere from 20,000 to 25,000 genes.\", metadata={'source': 'https://www.britannica.com/event/Human-Genome-Project', 'page_title': 'Human Genome Project (HGP) | History, Timeline, & Facts'}),\n",
       " Document(page_content='What was the Human Genome Project? The Human Genome Project was a large, well-organized, and highly collaborative international effort that generated the first sequence of the human genome and that of several additional well-studied organisms. Carried out from 1990–2003, it was one of the most ambitious and important scientific endeavors in human history. Photo: Robert Waterston, M.D., Ph.D., at the 2001 press conference announcing the publication describing the draft sequence of the human genome generated by the Human Genome Project. Dr. Waterston was an instrumental planner, prominent leader, and major participant of the Human Genome Project. (NHGRI Photo Archive) What were the goals of the Human Genome Project? A special committee of the U.S. National Academy of Sciences outlined the original goals for the Human Genome Project in 1988, which included sequencing the entire human genome in addition to the genomes of several carefully selected non-human organisms. Eventually the list of organisms came to include the bacterium E. coli, baker’s yeast, fruit fly, nematode and mouse. The project’s architects and participants hoped the resulting information would usher in a new era for biomedical research, and its goals and related strategic plans were updated periodically throughout the project. In part due to a deliberate focus on technology development, the Human Genome Project ultimately exceeded its initial set of goals, doing so by 2003, two years ahead of its originally projected 2005 completion. Many of the project’s achievements were beyond what scientists thought possible in 1988. President Bill Clinton and Francis Collins, M.D., Ph.D., (NHGRI Director) at a June 2000 event at the White House celebrating the draft human genome sequence generated by the Human Genome Project. Dr. Collins served as the de facto leader of the International Human Genome Sequencing Consortium, the group that sequenced the human genome during the Human Genome Project. (NHGRI Photo Archive) What is DNA sequencing? How was it performed during the Human Genome Project? DNA sequencing involves determining the exact order of the bases in DNA — the As, Cs, Gs and Ts that make up segments of DNA. Because the Human Genome Project aimed to sequence all of the DNA (i.e., the genome) of a set of organisms, significant effort was made to improve the methods for DNA sequencing. Ultimately, the project used one particular method for DNA sequencing, called Sanger DNA sequencing, but first greatly advanced this basic method through a series of major technical innovations. Photo: Dr. Collins analyzing an autoradiogram displaying the results of a Sanger DNA sequencing experiment, such as that used in the early years of the Human Genome Project. (NHGRI Photo Archive) Whose DNA was sequenced by the Human Genome Project? How was it collected? The sequence of the human genome generated by the Human Genome Project was not from a single person. Rather, it reflects a patchwork from multiple people whose identities were intentionally made anonymous to protect their privacy. The project researchers used a thoughtful process to recruit volunteers, acquire their informed consent, and collect their blood samples. Most of the human genome sequence generated by the Human Genome Project came from blood donors in Buffalo, New York; specifically, 93% from 11 donors, and 70% from one donor. Photo: A 1997 Buffalo, New York newspaper advertisement recruiting volunteers to provide blood samples and DNA for the Human Genome Project. (NHGRI History of Genomics Program Archive) Who carried out the Human Genome Project? The Human Genome Project could not have been completed as quickly and effectively without the dedicated participation of an international consortium of thousands of researchers. In the United States, the researchers were funded by the Department of Energy and the National Institutes of Health, which created the Office for Human Genome Research in 1988 (later renamed the National Center for Human Genome Research in 1990 and then the National Human Genome Research Institute in 1997). The sequencing of the human genome involved researchers from 20 separate universities and research centers across the United States, United Kingdom, France, Germany, Japan and China. The groups in these countries became known as the International Human Genome Sequencing Consortium. Photo: Researcher at Washington University in St. Louis handling frozen clones containing human DNA being studied by Human Genome Project researchers. (NHGRI Photo Archive) How much did the Human Genome Project cost? The initially projected cost for the Human Genome Project was $3 billion, based on its envisioned length of 15 years. While precise cost-accounting was difficult to carry out, especially across the set of international funders, most agree that this rough amount is close to the accurate number. The cost of the Human Genome Project, while in the billions of dollars, has been greatly offset by the positive economic benefits that genomics has yielded in the ensuing decades. Such economic gains reflect direct links between resulting products and advances in the pharmaceutical and biotechnology industries, among others. Photo: A subset of the participants of a 1996 meeting held in Bermuda that helped to establish the “Bermuda Principles” for sharing genome sequence data during the Human Genome Project. Shown (from left to right) are: David Bentley, John Sulston, and Eric Green in the front row and Richard McCombie, Richard Gibbs, Richard Wilson, and Elson Chen in the back row. (Photo by Eric Green) Did the Human Genome Project produce a perfectly complete genome sequence? No. Throughout the Human Genome Project, researchers continually improved the methods for DNA sequencing. However, they were limited in their abilities to determine the sequence of some stretches of human DNA (e.g., particularly complex or highly repetitive DNA). In June 2000, the International Human Genome Sequencing Consortium announced that it had produced a draft human genome sequence that accounted for 90% of the human genome. The draft sequence contained more than 150,000 areas where the DNA sequence was unknown because it could not be determined accurately (known as gaps). In April 2003, the consortium announced that it had generated an essentially complete human genome sequence, which was significantly improved from the draft sequence. Specifically, it accounted for 92% of the human genome and less than 400 gaps; it was also more accurate. On March 31, 2022, the Telomere-to-Telomere (T2T) consortium announced that had filled in the remaining gaps and produced the first truly complete human genome sequence. Photo: A Human Genome Project researcher pipetting a DNA sample into an agarose gel to perform gel electrophoresis. (NHGRI Photo Archive) How did the Human Genome Project change practices around data sharing in the scientific research community? Human Genome Project scientists made every part of the draft human genome sequence publicly available shortly after production. This routine came from two meetings in Bermuda in which project researchers agreed to the “Bermuda Principles,” which set out the rules for the rapid release of sequence data. This landmark agreement has been credited with establishing a greater awareness and openness to the sharing of data in biomedical research, making it one of the most important legacies of the Human Genome Project. The first International Strategy Meeting on Human Genome Sequencing in Bermuda 1996. Nobelist James Watson, Ph.D., (front left; in hat) played an instrumental role in launching the Human Genome Project and promoting the project’s sharing of genomic data; however, Dr. Watson is also known for his offensive and scientifically incorrect comments about a number of societal topics that are incongruent with the values of NHGRI. (NHGRI Photo Archive) How did the Human Genome Project foster ethics in biological research? The leaders of the Human Genome Project recognized the need to be proactive in addressing a wide range of ethical and social issues related to the acquisition and use of genomic information. They were especially aware of the potential risks and benefits of incorporating new genomic knowledge into research and medicine. Similarly, they were aware of the potential misuse of genomic information when it came to insurance and employment, among others. To help understand and address these issues, NHGRI established the Ethical, Legal, and Social Implications (ELSI) Research Program in 1990. The early appreciation of the value of this program later led the U.S. Congress to mandate that NHGRI dedicate at least 5% of its research budget to studying the ethical, legal and social implications of genomic advances. The NHGRI ELSI Research Program has become a model for bioethics research worldwide. Former NHGRI Acting Director Michael Gottesman, M.D., during the Human Genome Project. Dr. Gottesman played an important interim role in leading NHGRI between its first Director (Dr. Watson) and its second Director (Dr. Collins). (NHGRI Photo Archive) How did the Human Genome Project affect biological research in general? The Human Genome Project demonstrated that production-oriented, discovery-driven scientific inquiry — which did not involve the investigation of a specific hypothesis or the direct answering of preformed questions — could be remarkably valuable and beneficial to the broader scientific community. The project was also a successful example of “big science” in biomedical research. The magnitude of the technological challenges prompted the Human Genome Project to assemble interdisciplinary groups from across the world, involving experts in engineering, biology, and computer science, among other areas. It also required the work to be concentrated in a modest number of major centers to maximize economies of scale. Before the Human Genome Project, the biomedical research community viewed projects of such scale with deep skepticism. These kinds of massive scientific undertakings have become more commonplace and well-accepted based in part on the success of the Human Genome Project. Photo: Participants of a 2002 meeting (held at Cold Spring Harbor Laboratory) of the International Human Genome Sequencing Consortium, the group involved in generating the first human genome sequence during the Human Genome Project. (Photo by Eric Green)', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'}),\n",
       " Document(page_content='2000 February 2000: President Clinton signs Executive Order to prevent genetic discrimination in any federal workplace. March 2000: A public consortium of scientists and a private company release a substantially complete genome sequence of the fruitfly (Drosophila melanogaster). The findings are reported in the March 24 issue of Science. April 3-6, 2000: The National Human Genome Research Institute (NHGRI), the National Institutes of Health (NIH) Office of Rare Disease Research, and the Don and Linda Carter Foundation sponsor the first NIH Conference on Holoprosencephaly. May 2000: Scientists in Japan and Germany report in the May 18 issue of Nature [nature.com] that they have unraveled the genetic code of human chromosome 21, already known to be involved with Down syndrome, Alzheimer\\'s disease, Usher syndrome and Lou Gehrig\\'s disease. June 2000: The Human Genome Project consortium announces a major milestone: It has assembled 85 percent of the sequence of the human genome - the genetic blueprint for a human being. See White House Announcement August 2000: Scientists discover a genetic \"signature\" that may help explain how malignant melanoma - a deadly form of skin cancer - can spread to other parts of the body. Findings are reported in the August 3 issue of Nature. October 2000: The National Institutes of Health, the Wellcome Trust, and three private companies collaborate to form the Mouse Genome Sequencing Consortium (MGSC) to accelerate the determination of the DNA sequence of the mouse genome. October 2000: The Human Genome Project is the recipient of the American Society of Human Genetics\\' Allan Award to honor the hundreds of scientists involved in deciphering the human genetic code. November 9, 2000: The National Human Genome Research Institute (NHGRI) hosts the second annual \"Consumer Day. 2001 January 16-18, 2001: The Ethical, Legal and Social Implications (ELSI) Research Programs of The National Human Genome Research Institute and the Department of Energy sponsor a conference to celebrate a decade of research and consider its impact on genetic research, health and policy. February 2001: The International Human Genome Sequencing Consortium publishes a series of scientific papers in the February 15 issue of Nature [nature.com], providing the first analysis of the human genome sequence that describes how it is organized and how it evolved. One significant discovery reveals that there are only 30,000 to 40,000 genes, not 100,000 as previously thought. February 2001: National Human Genome Research Institute scientists use microarray technology to develop a gene test that differentiates hereditary from sporadic types of breast cancer. Findings are reported in the February 22 issue of The New England Journal of Medicine [nejm.org]. March 2001: National Human Genome Research Institute and Human Genome Project-funded scientists find a new tumor suppressor gene involved in breast, prostate and other cancers on human chromosome 7. A single post-doc, using the \"working draft\" sequence data, is able to pin down the gene within weeks; before, the same work took several years and the work of many scientists. Findings are reported in the April issue of Nature Genetics [nature.com]. May 2001: The Mouse Genome Sequencing Consortium announces it has achieved three-fold coverage of the mouse DNA sequence. The data, representing 95 percent of the mouse sequence, are publicly available [ncbi.nlm.nih.gov] and will be an important tool for discovering human genes when comparing the genomes of the mouse and human. May 2001: The National Human Genome Research Institute and scientists at Sweden\\'s Lund University develop a method of accurately diagnosing four complex, hard-to-distinguish childhood cancers using microarray technology and artificial neural networks (ANN). Findings are published in the June issue of Nature [nature.com]. September 2001: The National Human Genome Research Institute announces the first Centers of Excellence in Genomic Science Award, a research program that supports multi-investigator, interdisciplinary teams who develop innovative genomic approaches that address important biological and biomedical research problems and seek to change the way genomics is done and used in biomedicine. November 9-11, 2001: The National Human Genome Research Institute co-sponsors The Human Genome Project Conference: The Challenges and Impact of Human Genome Research for Minority Communities, with the Zeta Phi Beta Sorority National Education Foundation, the National Human Genome Center at Howard University, and the Family Life Center Foundation at Shiloh Baptist Church. December 12-14, 2001: The National Human Genome Research Institute holds the planning conference, Beyond the Beginning: The Future of Genomics at the Airlie Conference Center in Warrenton, Virginia, to develop a broad vision of the future of genomics research that will lay the foundation for a bold new plan for NHGRI. 2002 January 2002: The National Human Genome Research Institute and other scientists find a gene on chromosome 1 associated with an inherited form of prostate cancer in some families. The findings are published in the February issue of Nature Genetics [nature.com]. February 2002: The National Human Genome Research Institute and the NIH Office of Rare Diseases launch the Genetics and Rare Diseases Information Center, delivering free and accurate information to patients and their families about genetic and rare diseases. May 2002: The Mouse Genome Sequencing Consortium announces a 96 percent complete working draft of the mouse genome freely available in public databases [ensembl.org]. The methods to sequence the mouse genome set a new standard for speed and accuracy. June 2002: The National Human Genome Research Institute launches a new Web site, www.genome.gov, that provides improved usability and easy access to new content for a wide range of users. September 2002: Alan Edward Guttmacher, M.D. is named as the second deputy director of the National Human Genome Research Institute (NHGRI). Vence L. Bonham, Jr., J.D., is appointed as NHGRI\\'s Senior Consultant to the Director on Health Disparities. September 2002: Gene discovery by an international team of researchers led by the National Human Genome Research Institute, reveals the cause for a rare form of microcephaly, a devastating brain disorder. October 2002: The National Human Genome Research Institute launches the International HapMap Project, this new venture is aimed at speeding the discovery of genes related to common illnesses such as asthma, cancer, diabetes and heart disease. October 2002: The National Human Genome Research Institute, in cooperation with five other institutes and centers at the National Institutes of Health, awards a three-year, $15-million grant to combine three of the world\\'s current protein sequence databases into a single global resource. November 2002: The National Human Genome Research Institute selects Eric D. Green, M.D., Ph.D., as the new scientific director for the NHGRI. November 2002: The National Human Genome Research Institute names William A. Gahl, M.D., Ph.D., as its new intramural clinical director. December 2002: The international Mouse Genome Sequencing Consortium announces the publication of a high-quality draft sequence of the mouse genome - the genetic blueprint of a mouse. 2003 February 2003: The National Human Genome Research Institute and the Department of Energy announce April 2003 events to celebrate the 50th anniversary of the discovery of the DNA double helix, the completion of the human genome sequence and the publication of the vision plan for NHGRI. March 2003: The National Human Genome Research Institute announces a new project - the ENCODE Project - with the long-term goal of creating a comprehensive encyclopedia of functional elements encoded in the human DNA. April 2003: The National Human Genome Research Institute celebrates the completion of the human genome sequence, the 50th anniversary of the description of the DNA double helix and the publication of the vision document for the future of genomics research. April 2003: National Human Genome Research Institute researchers identify the gene that causes the premature aging disorder progeria. The findings were released online in the journal Nature [nature.com]. June 2003: National Human Genome Research Institute study findings, reported in the June 13 issue of Science [scienceonline.org], will aid researchers in discovering safer methods of gene therapy. June 2003: A detailed analysis of the just-completed sequence of the human Y chromosome - a study published in the June issue of Nature [nature.com], and funded in large part by the National Human Genome Research Institute (NHGRI) - shows the Y chromosome appears to exchange genes between the two copies of repeated sequences that lie near to each other as mirror images. July 2003: A detailed analysis of the reference sequence of chromosome 7, carried out by a multinational team of scientists led by the Washington University School of Medicine, uncovers structural features that appear to promote genetic changes that can cause disease. The findings were reported in the July 10 issue of the journal Nature [nature.com]. National Human Genome Research Institute (NHGRI) Scientific Director Eric Green co-authored the study. August 2003: A team of researchers led by the National Human Genome Research Institute reports findings on the comparison of 13 vertebrate genomes. Results, published in the August 14 issue of Nature [nature.com], suggested that comparison of a wide variety of species\\' genomes will not only illuminate genomic evolution but help identify functional elements in the human genome. October 2003: The National Human Genome Research Institute announced the first grants in a three-year, $36 million scientific reconnaissance mission - called ENCODE - aimed at discovering all parts of the human genome that are crucial to biological function. October 14, 2003: The U.S. Senate passes the Genetic Information Nondiscrimination Act of 2003 (S. 1053) [thomas.loc.gov] by a vote of 95-0, the first time the Senate has passed a bipartisan, genetic nondiscrimination bill. The bill prevents health insurers and employers from using genetic information to determine eligibility, set premiums, or hire and fire people. The bill faces approval in the House of Representatives and from the President. November 7, 2003: The National Human Genome Research Institute announces the selection of five centers to carry out a new generation of large-scale sequencing projects designed to maximize the promise of the Human Genome Project and dramatically expand our understanding of human health and disease. December 5, 2003: The National Human Genome Research Institute announces the formation of a new branch - the Social and Behavioral Research Branch - within its Division of Intramural Research. December 10, 2003: The National Human Genome Research Institute announces the first draft version of the chimpanzee genome sequence and its alignment with the human genome. December 17, 2003: The International HapMap Consortium publishes a paper that sets forth the scientific rationale and strategybehind its effort to create a map of human genetic variation. 2004 January 7, 2004: The National Human Genome Research Institute announces that the first draft version of the honey bee genome sequence has been deposited into free public databases. January 26, 2004: The National Human Genome Research Institute and other scientists successfully create transgenic animals using sperm genetically modified and grown in a laboratory dish, an achievement with implications for wide ranging research, from developmental biology to gene therapy. The study [pnas.org] was published in the online edition of the Proceedings of the National Academy of Sciences. February 3, 2004: The Genetic and Rare Diseases Information Center established by the National Human Genome Research Institute and NIH Office of Rare Diseases, announces it has expanded its efforts to enable healthcare workers, patients and families who speak Spanish to take advantage of its free services. February 25, 2004: The National Human Genome Research Institute\\'s Large-Scale Sequencing Research Network announces it will begin sequencing the genome of the first marsupial, the gray short-tailed South American opossum, and more than a dozen other model organisms to further advance our understanding of the human genome. March 1, 2004: The National Human Genome Research Institute announces that the first draft version of the chicken genome sequence has been deposited into free public databases. March 11, 2004: National Human Genome Research Institute and other scientists find variants in a gene that may predispose people to type 2 diabetes, the most common form of the disease. March 24, 2004: The National Human Genome Research Institute announces that the International Sequencing Consortium has launched a free online resource, where scientists and the public can view the latest information on sequencing projects for animal, plant and eukaryotic genomes. March 31, 2004: The International Rat Genome Sequencing Project Consortium announces the publication of a high-quality draft sequence of the rat genome. June 8, 2004: The National Human Genome Research Institute and the Melbourne-based Australian Genome Research Facility, Ltd. announce a partnership to sequence the genome of the tammar wallaby, a member of the kangaroo family. June 28, 2004: The National Human Genome Research Institute announces it has established two new Centers of Excellence in Genomic Science (CEGS) at Harvard Medical School in Boston and the Johns Hopkins University School of Medicine in Baltimore. July 14, 2004: The National Human Genome Research Institute announces that the first draft version of the dog genome sequence has been deposited into free public databases. July 19, 2004: The National Human Genome Research Institute launches the NHGRI Policy and Legislative Database, an online resource that will enable researchers, health professionals and the general public to more easily locate information on laws and policies related to a wide array of genetic issues July 26, 2004: National Human Genome Research Institute scientists and an interdisciplinary consortium of researchers from 11 universities and institutions discover a possible inherited component for lung cancer, a disease normally associated with external causes, such as cigarette smoking. August 4, 2004: The National Human Genome Research Institute\\'s Large-Scale Sequencing Research Network announces a comprehensive strategic plan to sequence 18 additional organisms, including the African savannah elephant, domestic cat and orangutan, to help interpret the human genome. August 31, 2004: The National Human Genome Research Institute launches four interdisciplinary Centers for Excellence in Ethical, Legal and Social Implications Research to address some of the most pressing ethical, legal and social questions raised by recent advances in genetic and genomic research. October 6, 2004: The National Human Genome Research Institute announces that the first draft version of the bovine genome sequence has been deposited into free public databases. October 14, 2004: The National Human Genome Research Institute awards more than $38 million in grants to develop new sequencing technologies to accomplish the near-term goal of sequencing a mammalian-sized genome for $100,000 and the longer-term challenge of sequencing an individual human genome for $1,000 or less. October 18, 2004: The National Human Genome Research Institute announces that two of its medical geneticists, Dr. Alan Guttmacher and Dr. Robert Nussbaum, are elected to the Institute of Medicine of the National Academies. October 21, 2004: The International Human Genome Sequencing Consortium, led in the United States by the National Human Genome Research Institute and the Department of Energy (DOE) publishes its scientific description of the finished human genome sequence, reducing the estimated number of human protein-coding genes from 35,000 to only 20,000-25,000, a surprisingly low number for our species. October 22, 2004: The ENCODE Consortium publishes a paper in the October 22 issue of Science that sets forth the scientific rationale and strategy behind its quest to produce a comprehensive catalog of all parts of the human genome crucial to biological function. November 8, 2004: The National Human Genome Research Institute partners with the Department of Health and Human Services and the Office of the Surgeon General to launch a free computer program, My Family Health Portrait, which the public can use to record important family health information that may identify common diseases that run in families. December 8, 2004: The National Human Genome Research Institute and the International Chicken Genome Sequencing Consortium announces the publication of an analysis comparing the chicken and human genomes. It is the first bird to have its genome sequenced and analyzed. 2005 February 11, 2005: NHGRI establishes Office of Ethics, appointing Barbara Fuller, J.D., as Deputy Ethics Counselor March 16, 2005: As part of the Human Genome Project, the National Institutes of Health hails the first comprehensive analysis of the sequence of the human X chromosome, saying that this provides sweeping new insights into the evolution of sex chromosomes and the biological differences between males and females. August 8, 2005: NHGRI announces it has awards grants totaling more than $32 million to advance the development of innovative sequencing technologies intended to reduce the cost of DNA sequencing and expand the use of genomics in biomedical research and health care. August 29, 2005: In a surprising development, a research team led by NHGRI finds that a class of experimental anti-cancer drugs shows promise in laboratory studies for treating the fatal genetic disorder that causes premature aging. August 31, 2005: The first comprehensive comparison of the genetic blueprints of humans and chimpanzees shows our closest living relatives share perfect identity with 96 percent of our DNA sequence. October 5, 2005: The National Institutes of Health announces contracts that will give researchers unprecedented access to two private collections of knockout mice, providing valuable models for the study of human disease and laying the groundwork for a public, genome-wide library of knockout mice. October 26, 2005: The International HapMap Consortium publishes a comprehensive catalog of human genetic variation, a landmark achievement that is already accelerating the search for genes involved in common diseases, such as asthma, diabetes, cancer and heart disease. November 15, 2005: As part of the U.S. Surgeon General\\'s Family Health Initiative, an updated version of the computerized tool designed to help families gather their health information is unveiled. December 13, 2005: The National Cancer Institute and the National Human Genome Research Institute launches a comprehensive effort to accelerate understanding of the molecular basis of cancer through the application of genome analysis technologies, especially large-scale genome sequencing. 2006 March 29, 2006: A multi-institution team of experts, coordinated by geneticists from the National Human Genome Research Institute supports efforts to identify more than 70 bodies still unidentified in the aftermath of Hurricane Katrina. July 24, 2006: Researchers at the National Institutes of Health Chemical Genomics Center (NCGC) - an NHGRI Affiliated Center - develop a new screening approach that can profile compounds in large chemical libraries more accurately and precisely than standard methods, speeding the production of data that can be used to probe biological activities and identify leads for drug discovery. August 21, 2006: The National Human Genome Research Institute announces grants totaling $54 million over five years to establish one new Center of Excellence in Genomic Science and continue support for two existing centers. September 13, 2006: The National Cancer Institute and the National Human Genome Research Institute announce the first three cancers that will be studied in the pilot phase of The Cancer Genome Atlas. The cancers to be studied in the TCGA Pilot Project are lung, brain (glioblastoma) and ovarian. October 4, 2006: The National Human Genome Research Institute announces the latest round of grant awards totaling more than $13 million to speed the development of innovative sequencing technologies that reduce the cost of DNA sequencing and expand the use of genomics in medical research and health care. October 16, 2006: The National Cancer Institute and the National Human Genome Research Institute announce another two of the components of The Cancer Genome Atlas Pilot Project, a three-year, $100 million collaboration to test the feasibility of using large-scale genome analysis technologies to identify important genetic changes involved in cancer. 2007 January 14, 2007: The National Human Genome Research Institute celebrated its 10th anniversary as an institute of the National Institutes of Health (NIH), marking a decade that saw genomics emerge as a powerful research tool and looking ahead to an era in which genomics will transform medical care. April 26, 2007: National Human Genome Research Institute researchers identify new genetic risk factors for type 2 diabetes. In the most comprehensive look at genetic risk factors for type 2 diabetes to date, a U.S.- Finnish team, working in close collaboration with two other groups, has identified at least four new genetic variants associated with increased risk of diabetes and confirmed existence of another six. May 4, 2007: The National Human Genome Research Institute and the National Cancer Institute, have teamed with Group Health Cooperative in Seattle and Henry Ford Health System in Detroit to launch the Multiplex Initiative, a study to investigate the interest level of healthy, young adults in receiving genetic testing for eight common conditions. July 2, 2007: As part of The Cancer Genome Atlas pilot project, the National Institutes of Health, National Cancer Institute and NHGRI award eight two-year grants totaling $3.4 million to support the development of innovative technologies for exploring the genomic underpinnings of cancer. August 1, 2007: Looking ahead to a future in which each person\\'s genome can be sequenced as a routine part of medical research and health care, the National Human Genome Research Institute awards more than $15 million in grants to support development of innovative technologies with the potential to dramatically reduce the cost of DNA sequencing. August 2, 2007: The National Human Genome Research Institute establishes Genomic Healthcare Branch, headed by William Gregory Feero, M.D., Ph.D., to promote the effective integration of genomic discoveries into healthcare. August 20, 2007: The National Human Genome Research Institute announces grants of $30 million to establish one, new Center of Excellence in Genomic Science at the Dana Farber-Cancer Institute and continue its support of the center at Stanford University. August 22, 2007: The National Human Genome Research Institute establishes the Office of Population Genomics headed by Teri Manolio, M.D., Ph.D. October 9, 2007: The National Human Genome Research Institute announces grants totaling more than $80 million over the next four years to expand the ENCyclopedia Of DNA Elements (ENCODE) project, which in its pilot phase yielded provocative new insights into the organization and function of the human genome. October 10, 2007: The National Human Genome Research Institute announces the establishment of two new centers to address the most critical ethical, legal and social questions faced by researchers and patients involved in genetic and genomic research. October 16, 2007: The NIH Intramural Sequencing Center (NISC), an National Human Genome Research Institutes affiliated center, celebrates 10th anniversary with a day-long symposium. November 4, 2007: An international team of scientists, supported in part by the National Human Genome Research Institute, announces that its systematic effort to map the genomic changes underlying lung cancer has uncovered a critical gene alteration not previously linked to any form of cancer. November 5, 2007: In a White House Ceremony, NHGRI Director Francis S. Collins, M.D., Ph.D. is awarded the Presidential Medal of Freedom. December 19, 2007: To better understand the role the astounding assortment of bacteria, fungi and other microbes play in human health and disease, the National Institutes of Health (NIH) announces the official launch of the Human Microbiome Project. The human microbiome is the collective genomes of all microorganisms present in or on the human body. 2008 January 22, 2008: An international research consortium announces the 1000 Genomes Project, an ambitious effort that will involve sequencing the genomes of at least a thousand people from around the world to create the most detailed and medically useful picture to date of human genetic variation. February 14, 2008: Testing the safety of chemicals ranging from pesticides to household cleaners will benefit from new technologies and a plan for collaboration, according to federal scientists from the National Institutes of Health (NIH) and the U.S. Environmental Protection Agency (EPA), who announced a new toxicity testing agreement. March 17, 2008: The National Institutes of Health (NIH) announces the establishment of the NIH Intramural Center for Genomics and Health Disparities (NICGHD), a new venue for research about the way populations are impacted by diseases, including obesity, diabetes and hypertension. NICGHD will employ a genomics approach, collecting and analyzing genetic, clinical, lifestyle and socio-economic data to study a range of clinical conditions that have puzzled and troubled public health experts for decades. April 29, 2008: Research organizations from around the world announce they are launching the International Cancer Genome Consortium (ICGC), a collaboration designed to generate high-quality genomic data on up to 50 types of cancer through efforts projected to take up to a decade. May 7, 2008: The first analysis of the genome sequence of the duck-billed platypus is published in Nature by an international team of scientists, revealing clues about how genomes were organized during the early evolution of mammals. The research was supported in part by the National Human Genome Research Institute. May 21, 2008: President Bush signs into law the Genetic Information Nondiscrimination Act (GINA) that will protect Americans against discrimination based on their genetic information when it comes to health insurance and employment. The bill had passed the Senate unanimously and the House by a vote of 414 to 1. May 28, 2008: Francis S. Collins, M.D., Ph.D., the director of the National Human Genome Research Institute, part of the National Institutes of Health, announces his intention to step down on August 1 to explore writing projects and other professional opportunities. August 2, 2008: Alan Edward Guttmacher, M.D., becomes Acting Director of NHGRI. He replaces Francis S. Collins, M.D., Ph.D., who stepped down after 15 years at the helm of NHGRI to pursue other professional opportunities. 2009 March 12, 2009: A team that includes researchers from the National Institutes of Health (NIH) has found a new way of detecting functional regions in the human genome. The novel approach involves looking at the three-dimensional shape of the genome\\'s DNA and not just reading the sequence of the four-letter alphabet of its DNA bases. March 29, 2009: National Institutes of Health (NIH) researchers have identified a gene that suppresses tumor growth in melanoma, the deadliest form of skin cancer. The finding is reported today in the journal Nature Genetics as part of a systematic genetic analysis of a group of enzymes implicated in skin cancer and many other types of cancer. April 20, 2009: The National Human Genome Research Institute announces the release of the first version of a free online toolkit aimed at standardizing measurements of research subjects\\' physical characteristics and environmental exposures. The toolkit, is the first product of the Consensus Measures for Phenotypes and eXposures (PhenX) initiative. May 20, 2009: The National Institutes of Health launches a $24 million program to jumpstart a trans-NIH initiative called the Therapeutics for Rare and Neglected Diseases program, or TRND. The NIH Office of Rare Diseases Research (ORDR) will oversee the program, and TRND\\'s laboratory operations will be administered by the National Human Genome Research Institute (NHGRI), which also operates the NIH Chemical Genomics Center (NCGC), a principal collaborator in TRND. Other NIH components will also participate in the initiative. August 7, 2009: The U.S. Senate confirms that former NHGRI Director Francis S. Collins, M.D., Ph.D., will be the director of the National Institutes of Health (NIH). President Barack Obama announced the nomination of Dr. Collins on July 8th. Dr. Collins, a physician-geneticist who led NHGRI from 1993 to 2008, managed the NIH component of the international Human Genome Project. Statement from HHS Secretary Kathleen Sebelius November 17, 2009: Francis S. Collins, M.D., Ph.D., director of the National Institutes of Health (NIH), announces the appointment of Eric D. Green, M.D., Ph.D., to be director of the National Human Genome Research Institute (NHGRI). This marks the first time an institute director has risen to lead the entire NIH and subsequently picked his own successor.', metadata={'source': 'https://www.genome.gov/about-nhgri/Brief-History-Timeline', 'page_title': 'NHGRI History and Timeline of Events'}),\n",
       " Document(page_content='The Human Genome Project is one of the greatest scientific feats in history. The project was a voyage of biological discovery led by an international group of researchers looking to comprehensively study all of the DNA (known as a genome) of a select set of organisms. Launched in October 1990 and completed in April 2003, the Human Genome Project’s signature accomplishment – generating the first sequence of the human genome – provided fundamental information about the human blueprint, which has since accelerated the study of human biology and improved the practice of medicine. Learn more about the Human Genome Project below. Last updated: September 2, 2022', metadata={'source': 'https://www.genome.gov/human-genome-project', 'page_title': 'The Human Genome Project'}),\n",
       " Document(page_content=\"NHGRI's Oral History Collection: Interview with Robert Waterston and Jane Rogers Drs. Waterston and Rogers discuss their respective roles in the Human Genome Project, the critical events leading up to the completion of the “draft sequence” and the publication of the “draft sequence” paper. 20-year anniversary of publications reporting the draft human genome sequence — Eric Green In this interview, Eric Green, M.D., Ph.D., director of the National Human Genome Research Institute (NHGRI), talks with Christopher Donohue, Ph.D., NHGRI historian, about the significance of the “initial sequencing and analyses of the human genome” paper published in Nature in February 2001.\", metadata={'source': 'https://www.genome.gov/human-genome-project/20th-anniversary-of-landmark-human-genome-project-publications', 'page_title': '20th anniversary of landmark Human Genome Project publications'}),\n",
       " Document(page_content='The History of Genomics Program is a unique history and science communications effort at the National Human Genome Research Institute, part of the National Institutes of Health. This program uses the history of genomics and the Human Genome Project to promote equity, combat the dehumanization of marginalized communities and build trust in medical advances by confronting past and present-day wrongs, including stigma, eugenics, scientific racism, ableism and heteronormativity. The program develops scholarship, sponsors lectures and symposia, records oral histories and provides access to an archive of documents about NHGRI and the field of genomics. The program supports a vision of genomics as a science for all.', metadata={'source': 'https://www.genome.gov/leadership-initiatives/History-of-Genomics-Program', 'page_title': 'History of Genomics Program'}),\n",
       " Document(page_content='Page Loading Error* We are unable to process your request at this time. If this error continues, please contact JSTOR support. Please include the time the error occurred and what you were requesting. Go back to JSTOR ©2000- ITHAKA. All Rights Reserved. JSTOR®, the JSTOR logo, JPASS®, and ITHAKA® are registered trademarks of ITHAKA.', metadata={'source': 'https://www.jstor.org/stable/3083473', 'page_title': 'JSTOR: Page Loading Error'}),\n",
       " Document(page_content='Your access to PubMed Central has been blocked because you are using an automated process to retrieve content from PMC, in violation of the terms of the PMC Copyright Notice.', metadata={'source': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1592577/', 'page_title': 'Page not available - PMC'})]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "documents"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Splitting documents into chunks\n",
    "\n",
    "As most webpages (documents) are too long to feed directly to our language model, we must split the data into smaller pieces or chunks. In this case, we have divided the text into blocks of 1,000 characters each, with a 200 character overlap between each stride. For instance, chunk 1 spans from 0 to 1000, while chunk 2 ranges from 800 to 1800, and so on."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.text_splitter import CharacterTextSplitter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "text_splitter = CharacterTextSplitter(separator=' ', chunk_size=1000, chunk_overlap=200)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "texts = text_splitter.split_documents(documents)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "400"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(texts)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Computing embeddings of chunks and storage in a vector store\n",
    "\n",
    "We can use a language model to compute embeddings for our input text (chunks). If you're unfamiliar with the concept of embeddings, imagine a pile of colored M&Ms in the middle of your table. Large foundational models have learned the general properties of M&Ms and will move them to different locations on the table based on their color. So you will end up with green M&Ms in one corner, blue in another, and so on. Now imagine we are given a cyan M&M. We've never seen that color before. But we know that cyan is a mix of green and blue, so it can be placed somewhere close to those colors on the table. We can then measure how far away the new cyan M&M is from each other M&M on the table. The closest one will be the most similar and the furthest one will be the most dissimilar.\n",
    "\n",
    "Instead of M&Ms, we have chunks of documents. And instead of 2 dimensions, like our flat table surface, we have 1,000+ dimensions."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.embeddings.openai import OpenAIEmbeddings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get your own APU key at https://platform.openai.com/account/api-keys\n",
    "OPENAI_API_KEY = 'REPLACE_WITH_YOUR_KEY' # Replace with your own OpenAI API key"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "embeddings = OpenAIEmbeddings(openai_api_key=OPENAI_API_KEY)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.vectorstores import Chroma"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Using embedded DuckDB without persistence: data will be transient\n"
     ]
    }
   ],
   "source": [
    "docsearch = Chroma.from_documents(texts, embeddings)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Configuring what model we use and ask questions\n",
    "\n",
    "We can now pick which model to use and start asking questions!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.llms import OpenAIChat # Required to create a Language Model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/mdrk/miniconda3/envs/python311/lib/python3.11/site-packages/langchain/llms/openai.py:667: UserWarning: You are trying to use a chat model. This way of initializing it is no longer supported. Instead, please use: `from langchain.chat_models import ChatOpenAI`\n",
      "  warnings.warn(\n"
     ]
    }
   ],
   "source": [
    "# Pick an OpenAI model\n",
    "llm = OpenAIChat(model_name='gpt-3.5-turbo', openai_api_key=OPENAI_API_KEY)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain import  VectorDBQA # Required to create a Question-Answer object using a vector"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pprint # Required to pretty print the results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/mdrk/miniconda3/envs/python311/lib/python3.11/site-packages/langchain/chains/retrieval_qa/base.py:185: UserWarning: `VectorDBQA` is deprecated - please use `from langchain.chains import RetrievalQA`\n",
      "  warnings.warn(\n"
     ]
    }
   ],
   "source": [
    "# Stuff all the information into a single prompt (see https://docs.langchain.com/docs/components/chains/index_related_chains#stuffing)\n",
    "qa = VectorDBQA.from_chain_type(llm=llm, chain_type=\"stuff\", vectorstore=docsearch, return_source_documents=True)\n",
    "query = \"Who were the main players in the race to complete the human genome? And what were their approaches? Give as much detail as possible.\"\n",
    "result = qa({\"query\": query})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'query': 'Who were the main players in the race to complete the human genome? '\n",
      "          'And what were their approaches? Give as much detail as possible.',\n",
      " 'result': 'The main players in the race to complete the human genome were the '\n",
      "           'publicly funded Human Genome Project (HGP) and the privately '\n",
      "           'funded Celera Corporation, led by J. Craig Venter. Their '\n",
      "           'approaches differed in that the HGP was a large, collaborative '\n",
      "           'international effort, while Celera focused on creating a '\n",
      "           'proprietary database using advanced sequencing technology. The '\n",
      "           'competition arose from the prospect of gaining control over '\n",
      "           'potential patents on the genome sequence, which was considered '\n",
      "           'valuable. However, the rivalry ended when Celera and the HGP '\n",
      "           'joined forces, thus speeding completion of the rough draft '\n",
      "           'sequence of the human genome. Collaborative efforts continued for '\n",
      "           'the next three years to refine and extend the rough draft, leading '\n",
      "           'to the declaration of completion of the HGP in April 2003.',\n",
      " 'source_documents': [Document(page_content='What was the Human Genome Project? The Human Genome Project was a large, well-organized, and highly collaborative international effort that generated the first sequence of the human genome and that of several additional well-studied organisms. Carried out from 1990–2003, it was one of the most ambitious and important scientific endeavors in human history. Photo: Robert Waterston, M.D., Ph.D., at the 2001 press conference announcing the publication describing the draft sequence of the human genome generated by the Human Genome Project. Dr. Waterston was an instrumental planner, prominent leader, and major participant of the Human Genome Project. (NHGRI Photo Archive) What were the goals of the Human Genome Project? A special committee of the U.S. National Academy of Sciences outlined the original goals for the Human Genome Project in 1988, which included sequencing the entire human genome in addition to the genomes of several carefully selected non-human organisms. Eventually the list', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'}),\n",
      "                      Document(page_content='NIH scientist J. Craig Venter, began to compete with and potentially undermine the publicly funded HGP. At the heart of the competition was the prospect of gaining control over potential patents on the genome sequence, which was considered a pharmaceutical treasure trove. Although the legal and financial reasons remain unclear, the rivalry between Celera and the NIH ended when they joined forces, thus speeding completion of the rough draft sequence of the human genome. The completion of the rough draft was announced in June 2000 by Collins and Venter. For the next three years, the rough draft sequence was refined, extended, and further analyzed, and in April 2003, coinciding with the 50th anniversary of the publication that described the double-helical structure of DNA, written by British biophysicist Francis Crick and American geneticist and biophysicist James D. Watson, the HGP was declared complete. Britannica Quiz Genetics Quiz Science behind the HGP To appreciate the magnitude,', metadata={'source': 'https://www.britannica.com/event/Human-Genome-Project', 'page_title': 'Human Genome Project (HGP) | History, Timeline, & Facts'}),\n",
      "                      Document(page_content='The Human Genome Project (HGP) refers to the international 13-year effort, formally begun in October 1990 and completed in 2003, to discover all the estimated 20,000-25,000 human genes and make them accessible for further biological study. Another project goal was to determine the complete sequence of the 3 billion DNA subunits (bases in the human genome). As part of the HGP, parallel studies were carried out on selected model organisms such as the bacterium E. coli and the mouse to help develop the technology and interpret human gene function. The DOE Human Genome Program and the NIH National Human Genome Research Institute (NHGRI) together sponsored the U.S. Human Genome Project. History of the Project * HGP Goals and Corresponding Completion Dates, * Budget History of the U.S. Human Genome Project, * Timeline: Major Events in the Human Genome Project, Publications Summarizing Various Aspects of the Project * Special issue of Nature Human Genome Collection (2006), * Special issue of', metadata={'source': 'https://web.ornl.gov/sci/techresources/Human_Genome/project/hgp.shtml', 'page_title': 'History of the Human Genome Project'}),\n",
      "                      Document(page_content='The magnitude of the technological challenges prompted the Human Genome Project to assemble interdisciplinary groups from across the world, involving experts in engineering, biology, and computer science, among other areas. It also required the work to be concentrated in a modest number of major centers to maximize economies of scale. Before the Human Genome Project, the biomedical research community viewed projects of such scale with deep skepticism. These kinds of massive scientific undertakings have become more commonplace and well-accepted based in part on the success of the Human Genome Project. Photo: Participants of a 2002 meeting (held at Cold Spring Harbor Laboratory) of the International Human Genome Sequencing Consortium, the group involved in generating the first human genome sequence during the Human Genome Project. (Photo by Eric Green)', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'})]}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project',\n",
       " 'https://www.britannica.com/event/Human-Genome-Project',\n",
       " 'https://web.ornl.gov/sci/techresources/Human_Genome/project/hgp.shtml',\n",
       " 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project']"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[a.metadata['source'] for a in result['source_documents']] # Print the source documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "qa = VectorDBQA.from_chain_type(llm=llm, chain_type=\"stuff\", vectorstore=docsearch, return_source_documents=True)\n",
    "query = \"How were the donor participants recruited for the human genome project? Summarize in three sentences.\"\n",
    "result = qa({\"query\": query})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'query': 'How were the donor participants recruited for the human genome '\n",
      "          'project? Summarize in three sentences.',\n",
      " 'result': 'The International Human Genome Sequencing Consortium collected '\n",
      "           'blood or sperm samples from many donors, with their identities '\n",
      "           'protected to maintain anonymity. Only a few samples were used for '\n",
      "           'DNA resources, and most of the sequence generated by the public '\n",
      "           'HGP came from a single anonymous male donor from Buffalo, New '\n",
      "           'York. Volunteers were recruited through a process of informed '\n",
      "           'consent, with a 1997 newspaper advertisement from Buffalo seeking '\n",
      "           'participants.',\n",
      " 'source_documents': [Document(page_content='of the joint publications, press releases announced that the project had been completed by both groups. Improved drafts were announced in 2003 and 2005, filling in to approximately 92% of the sequence currently. Genome donors[edit] In the International Human Genome Sequencing Consortium (IHGSC) public-sector HGP, researchers collected blood (female) or sperm (male) samples from a large number of donors. Only a few of many collected samples were processed as DNA resources. Thus the donor identities were protected so neither donors nor scientists could know whose DNA was sequenced. DNA clones taken from many different libraries were used in the overall project, with most of those libraries being created by Dr. Pieter J. de Jong. Much of the sequence (>70%) of the reference genome produced by the public HGP came from a single anonymous male donor from Buffalo, New York, (code name RP11; the \"RP\" refers to Roswell Park Comprehensive Cancer Center).[70][71] HGP scientists used white blood', metadata={'source': 'https://en.wikipedia.org/wiki/Human_Genome_Project', 'page_title': 'Human Genome Project - Wikipedia'}),\n",
      "                      Document(page_content='their blood samples. Most of the human genome sequence generated by the Human Genome Project came from blood donors in Buffalo, New York; specifically, 93% from 11 donors, and 70% from one donor. Photo: A 1997 Buffalo, New York newspaper advertisement recruiting volunteers to provide blood samples and DNA for the Human Genome Project. (NHGRI History of Genomics Program Archive) Who carried out the Human Genome Project? The Human Genome Project could not have been completed as quickly and effectively without the dedicated participation of an international consortium of thousands of researchers. In the United States, the researchers were funded by the Department of Energy and the National Institutes of Health, which created the Office for Human Genome Research in 1988 (later renamed the National Center for Human Genome Research in 1990 and then the National Human Genome Research Institute in 1997). The sequencing of the human genome involved researchers from 20 separate universities and', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'}),\n",
      "                      Document(page_content='the project used one particular method for DNA sequencing, called Sanger DNA sequencing, but first greatly advanced this basic method through a series of major technical innovations. Photo: Dr. Collins analyzing an autoradiogram displaying the results of a Sanger DNA sequencing experiment, such as that used in the early years of the Human Genome Project. (NHGRI Photo Archive) Whose DNA was sequenced by the Human Genome Project? How was it collected? The sequence of the human genome generated by the Human Genome Project was not from a single person. Rather, it reflects a patchwork from multiple people whose identities were intentionally made anonymous to protect their privacy. The project researchers used a thoughtful process to recruit volunteers, acquire their informed consent, and collect their blood samples. Most of the human genome sequence generated by the Human Genome Project came from blood donors in Buffalo, New York; specifically, 93% from 11 donors, and 70% from one donor.', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'}),\n",
      "                      Document(page_content='The magnitude of the technological challenges prompted the Human Genome Project to assemble interdisciplinary groups from across the world, involving experts in engineering, biology, and computer science, among other areas. It also required the work to be concentrated in a modest number of major centers to maximize economies of scale. Before the Human Genome Project, the biomedical research community viewed projects of such scale with deep skepticism. These kinds of massive scientific undertakings have become more commonplace and well-accepted based in part on the success of the Human Genome Project. Photo: Participants of a 2002 meeting (held at Cold Spring Harbor Laboratory) of the International Human Genome Sequencing Consortium, the group involved in generating the first human genome sequence during the Human Genome Project. (Photo by Eric Green)', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'})]}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "qa = VectorDBQA.from_chain_type(llm=llm, chain_type=\"stuff\", vectorstore=docsearch, return_source_documents=True)\n",
    "query = \"What happened to Craig Venter's company following the completion of the human genome project? Give as much detail as possible.\"\n",
    "result = qa({\"query\": query})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'query': \"What happened to Craig Venter's company following the completion of \"\n",
      "          'the human genome project? Give as much detail as possible.',\n",
      " 'result': 'Following the completion of the human genome project, Craig '\n",
      "           \"Venter's company, Celera Genomics, faced a decision on what type \"\n",
      "           'of company it would become. It added sequences from three '\n",
      "           'different mouse strains to its database and briefly ventured into '\n",
      "           'proteomics. However, Venter resigned as CEO in January 2002, and '\n",
      "           'the company decided to focus on drug discovery rather than '\n",
      "           'information. Despite being timed to coincide with the celebrations '\n",
      "           'of the 50th anniversary of the Watson-Crick discovery of the '\n",
      "           'double-helical structure of DNA, there was less fanfare '\n",
      "           'surrounding the official date of completion of the HGP in April '\n",
      "           '2003. Celera remained a threat, as the validity of the WGS '\n",
      "           'sequencing approach was demonstrated in March 2000 when Celera and '\n",
      "           'the Berkeley Drosophila Genome Project published the sequence of '\n",
      "           'D. melanogaster. However, the rivalry between Celera and the HGP '\n",
      "           'ended when they joined forces, thus speeding up the completion of '\n",
      "           'the rough draft sequence of the human genome.',\n",
      " 'source_documents': [Document(page_content='NIH scientist J. Craig Venter, began to compete with and potentially undermine the publicly funded HGP. At the heart of the competition was the prospect of gaining control over potential patents on the genome sequence, which was considered a pharmaceutical treasure trove. Although the legal and financial reasons remain unclear, the rivalry between Celera and the NIH ended when they joined forces, thus speeding completion of the rough draft sequence of the human genome. The completion of the rough draft was announced in June 2000 by Collins and Venter. For the next three years, the rough draft sequence was refined, extended, and further analyzed, and in April 2003, coinciding with the 50th anniversary of the publication that described the double-helical structure of DNA, written by British biophysicist Francis Crick and American geneticist and biophysicist James D. Watson, the HGP was declared complete. Britannica Quiz Genetics Quiz Science behind the HGP To appreciate the magnitude,', metadata={'source': 'https://www.britannica.com/event/Human-Genome-Project', 'page_title': 'Human Genome Project (HGP) | History, Timeline, & Facts'}),\n",
      "                      Document(page_content=\"to fill in. Celera was faced with deciding what sort of company it would be: sequences from three different mouse strains were added to help attract subscribers to its database, and a brief foray was made into proteomics, but Venter resigned as CEO in January 2002 with the company's decision to focus on drug discovery rather than information (Davies 2002). Despite being timed to coincide with celebrations of the 50th anniversary of the Watson-Crick discovery of the double-helical structure of DNA, there was less fanfare surrounding the official date of completion of the HGP in April 2003, two years earlier than had been anticipated at the time of its official launch in October 1990, and several months earlier than called for in the most recent five-year plan. Americans had terrorism and war on their minds. In the end, sequencing—the third phase of the publicly-funded project—was carried out at 16 centers in six countries by divvying up among them sections of chromosomes for sequencing.\", metadata={'source': 'https://plato.stanford.edu/entries/human-genome/', 'page_title': 'The Human Genome Project'}),\n",
      "                      Document(page_content='1990, but the early years were not very successful. In 1998, the american biochemist Craig Venter questioned the operation of the public project. It was expensive, costly, and they used an inefficient technology. And his company, Celera Genomics, joined the race by applying ‘Shotgun’ technology to sequence multiple DNA fragments simultaneously. “Celera was a very open-minded company in the sense that they immediately had the participation of experts in the field, and they contacted us to use our software to visualize the genome of the fruit fly“, says Guigó. This first work meant that a few months before publishing the sequence of the human genome, Celera contacted them again to visualize the sequence of the human genome. “When we were developing the software, here in Barcelona, I thought I was working on something that was important, but just as many scientists consider that their research is important. However, when I went to Celera Genomics with my PhD student Josep Abril, to work', metadata={'source': 'https://ellipse.prbb.org/the-human-genome-20-years-of-history/', 'page_title': 'The human genome: 20 years of history - El·lipse'}),\n",
      "                      Document(page_content=\"was also evidence that Celera did remain a threat: the validity of the WGS sequencing approach was demonstrated in March 2000 when Celera and the (publicly-funded) Berkeley Drosophila Genome Project published the sequence of D. melanogaster of about 180 Mb (Adams et al. 2000). In June 2000, the contest ended in what appeared to be a tie for the prize, but was more an arranged truce. On 26 June 2000, Collins, Venter, and the DOE's Ari Patrinos joined U.S. President Bill Clinton (and British Prime Minister Tony Blair by satellite link) at a White House press conference to announce that the human genome had been sequenced. That Collins and Venter even shared the limelight on that day was itself a tremendous feat. The negotiated draw at the finish line permitted HGP scientists to save face and their upstart competitor to minimize the risk of alienating university-based researchers and losing their business. The agreement between parties included eventual simultaneous publication of their\", metadata={'source': 'https://plato.stanford.edu/entries/human-genome/', 'page_title': 'The Human Genome Project'})]}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "qa = VectorDBQA.from_chain_type(llm=llm, chain_type=\"stuff\", vectorstore=docsearch, return_source_documents=True)\n",
    "query = \"How come the project finished earlier then expected? Give as much detail as possible.\"\n",
    "result = qa({\"query\": query})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'query': 'How come the project finished earlier then expected? Give as much '\n",
      "          'detail as possible.',\n",
      " 'result': 'The project finished earlier than expected due to a deliberate '\n",
      "           'focus on technology development, improved sequencing technologies, '\n",
      "           'and a change in approach to the finishing process. The original '\n",
      "           \"completion date was set for 2005, but the project's goals and \"\n",
      "           'related strategic plans were updated periodically throughout the '\n",
      "           'project. The final completion date was moved forward to 2003 with '\n",
      "           'a plan for a \"working draft\" of the human genome sequence by '\n",
      "           'December 2001. The project ended up costing less than expected, at '\n",
      "           \"around $2.7 billion. Many of the project's achievements were \"\n",
      "           'beyond what scientists thought possible in 1988.',\n",
      " 'source_documents': [Document(page_content='original goals for the Human Genome Project in 1988, which included sequencing the entire human genome in addition to the genomes of several carefully selected non-human organisms. Eventually the list of organisms came to include the bacterium E. coli, baker’s yeast, fruit fly, nematode and mouse. The project’s architects and participants hoped the resulting information would usher in a new era for biomedical research, and its goals and related strategic plans were updated periodically throughout the project. In part due to a deliberate focus on technology development, the Human Genome Project ultimately exceeded its initial set of goals, doing so by 2003, two years ahead of its originally projected 2005 completion. Many of the project’s achievements were beyond what scientists thought possible in 1988. President Bill Clinton and Francis Collins, M.D., Ph.D., (NHGRI Director) at a June 2000 event at the White House celebrating the draft human genome sequence generated by the Human', metadata={'source': 'https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genome-project', 'page_title': 'Human Genome Project Fact Sheet'}),\n",
      "                      Document(page_content='complete draft was published in 2003, and genome \"finishing\" work continued for more than a decade after that.The $3 billion project was formally founded in 1990 by the US Department of Energy and the National Institutes of Health, and was expected to take 15 years.[24] In addition to the United States, the international consortium comprised geneticists in the United Kingdom, France, Australia, China, and myriad other spontaneous relationships.[25] The project ended up costing less than expected, at about $2.7 billion (equivalent to about $5 billion in 2021).[7][26][27]Two technologies enabled the project: gene mapping and DNA sequencing. The gene mapping technique of restriction fragment length polymorphism (RFLP) arose from the search for the location of the breast cancer gene by Dr. Mark Skolnick of the University of Utah,[28] which began in 1974.[29] Seeing a linkage marker for the gene, collaboration with David Botstein, Ray White and Ron Davies conceived of a way to construct a', metadata={'source': 'https://en.wikipedia.org/wiki/Human_Genome_Project', 'page_title': 'Human Genome Project - Wikipedia'}),\n",
      "                      Document(page_content='Project, * Timeline: Major Events in the Human Genome Project, Publications Summarizing Various Aspects of the Project * Special issue of Nature Human Genome Collection (2006), * Special issue of Science: Building on the DNA Revolution (April 11, 2003) \"The Human Genome Project: Lessons from Large-Scale Biology,\" Francis S. Collins, Michael Morgan, Aristides Patrinos, Science 300, 286 (2003) \"Realizing the Potential of the Genome Revolution: The Genomes to Life Program,\" Marvin E. Frazier, Gary M. Johnson, David G. Thomassen, Carl E. Oliver, Aristides Patrinos, Science 300, 290 (2003), * Nature Genetics: A 10-Year Retrospective 1992-2002 (vol. 33, March 2003), * Controversial From the Start —Science article summarizing the history of the HGP (February 2001), * Genomes: 15 Years Later—A perspective from Charles DeLisi, HGP Pioneer (July 2001), * 1997 Human Genome Program Report contains history of the Project, * Bermuda Conference Data Release Policies (1997, 1996). See also, NHGRI', metadata={'source': 'https://web.ornl.gov/sci/techresources/Human_Genome/project/hgp.shtml', 'page_title': 'History of the Human Genome Project'}),\n",
      "                      Document(page_content='1998-2003 (Collins et al. 1998). The plan moved the final completion date forward from 2005 to 2003 and aimed for a “working draft” of the human genome sequence to be completed by December 2001. This would be achieved by delaying the finishing process, no longer going clone-by-clone to shotgun, reassemble, and finish the sequence of one clone before proceeding to the next. A physical map of 41,664 STS markers was soon published (Deloukas et al. 1998), and so the physical mapping goal was met, but with only six percent of the human genome sequence completed, the plan called for new and improved sequencing technologies which could increase the sequencing capacity from 90 Mb per year at about $.50 per base to 500 Mb per year at no more than $.25 per base. Goals for completing the sequencing of the remaining model organisms were also set: December 1998 for C. elegans which was 80 percent complete, 2002 for D. melanogaster which was nine percent complete, and 2005 for M. musculus which was', metadata={'source': 'https://plato.stanford.edu/entries/human-genome/', 'page_title': 'The Human Genome Project'})]}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<details>\n",
       "<summary>Click to view session information</summary>\n",
       "<pre>\n",
       "-----\n",
       "bs4                 4.12.2\n",
       "langchain           0.0.141\n",
       "numpy               1.23.5\n",
       "readabilipy         NA\n",
       "requests            2.28.2\n",
       "session_info        1.0.0\n",
       "-----\n",
       "</pre>\n",
       "<details>\n",
       "<summary>Click to view modules imported as dependencies</summary>\n",
       "<pre>\n",
       "aiohttp                     3.8.4\n",
       "aiosignal                   1.3.1\n",
       "appnope                     0.1.3\n",
       "asttokens                   NA\n",
       "async_timeout               4.0.2\n",
       "attr                        23.1.0\n",
       "backcall                    0.2.0\n",
       "backoff                     2.2.1\n",
       "certifi                     2022.12.07\n",
       "charset_normalizer          3.1.0\n",
       "chromadb                    0.3.21\n",
       "clickhouse_connect          NA\n",
       "cython_runtime              NA\n",
       "dateutil                    2.8.2\n",
       "debugpy                     1.5.1\n",
       "decorator                   5.1.1\n",
       "duckdb                      0.7.1\n",
       "executing                   1.2.0\n",
       "frozenlist                  1.3.3\n",
       "greenlet                    2.0.1\n",
       "hnswlib                     NA\n",
       "html5lib                    1.1\n",
       "idna                        3.4\n",
       "ipykernel                   6.15.0\n",
       "jedi                        0.18.2\n",
       "lxml                        4.9.2\n",
       "lz4                         4.3.2\n",
       "monotonic                   NA\n",
       "mpl_toolkits                NA\n",
       "multidict                   6.0.4\n",
       "openai                      NA\n",
       "openapi_schema_pydantic     NA\n",
       "packaging                   23.1\n",
       "pandas                      1.5.3\n",
       "parso                       0.8.3\n",
       "pexpect                     4.8.0\n",
       "pickleshare                 0.7.5\n",
       "pkg_resources               NA\n",
       "platformdirs                3.2.0\n",
       "posthog                     2.5.0\n",
       "prompt_toolkit              3.0.38\n",
       "psutil                      5.9.0\n",
       "ptyprocess                  0.7.0\n",
       "pure_eval                   0.2.2\n",
       "pyarrow                     11.0.0\n",
       "pydantic                    1.10.7\n",
       "pydev_ipython               NA\n",
       "pydevconsole                NA\n",
       "pydevd                      2.6.0\n",
       "pydevd_concurrency_analyser NA\n",
       "pydevd_file_utils           NA\n",
       "pydevd_plugins              NA\n",
       "pydevd_tracing              NA\n",
       "pygments                    2.15.0\n",
       "pytz                        2023.3\n",
       "regex                       2.5.125\n",
       "six                         1.16.0\n",
       "socks                       1.7.1\n",
       "soupsieve                   2.4.1\n",
       "sqlalchemy                  1.4.47\n",
       "stack_data                  0.6.2\n",
       "tenacity                    NA\n",
       "tiktoken                    NA\n",
       "tiktoken_ext                NA\n",
       "tornado                     6.2\n",
       "traitlets                   5.9.0\n",
       "typing_extensions           NA\n",
       "urllib3                     1.26.15\n",
       "wcwidth                     0.2.6\n",
       "webencodings                0.5.1\n",
       "yaml                        6.0\n",
       "yarl                        1.8.2\n",
       "zmq                         23.2.0\n",
       "zoneinfo                    NA\n",
       "zstandard                   0.21.0\n",
       "</pre>\n",
       "</details> <!-- seems like this ends pre, so might as well be explicit -->\n",
       "<pre>\n",
       "-----\n",
       "IPython             8.12.0\n",
       "jupyter_client      8.2.0\n",
       "jupyter_core        5.3.0\n",
       "-----\n",
       "Python 3.11.2 (main, Mar 27 2023, 18:38:21) [Clang 14.0.6 ]\n",
       "macOS-13.0-arm64-arm-64bit\n",
       "-----\n",
       "Session information updated at 2023-04-17 03:34\n",
       "</pre>\n",
       "</details>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import session_info\n",
    "session_info.show()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "python311",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.2"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
